{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "8a988125-f9cb-4eea-b23d-1547b60c44f7",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_dir<-(\"/research/labs/bme/weiz/m238739/GWAS/data2/\")\n",
    "fig_dir<-(\"/research/labs/bme/weiz/m238739/GWAS/fig/\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "e05dca1e-78b2-4b66-8fa4-dec7b3aff11b",
   "metadata": {},
   "outputs": [],
   "source": [
    "#install.packages(\"gwasrapidd\")\n",
    "library(gwasrapidd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "eaa2cc43-85c1-4428-a47a-fbdf456ea880",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[1] ‘0.99.17’"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "packageVersion(\"gwasrapidd\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "d83f1400-61ce-41d5-a078-b1fbb459e940",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "  downloading [===============>--------------------------------]  33% eta:  4s\n",
      "\n",
      "  downloading [===================>----------------------------]  42% eta:  4s\n",
      "\n",
      "  downloading [=======================>------------------------]  50% eta:  3s\n",
      "\n",
      "  downloading [===========================>--------------------]  58% eta:  3s\n",
      "\n",
      "  downloading [===============================>----------------]  67% eta:  2s\n",
      "\n",
      "  downloading [===================================>------------]  75% eta:  2s\n",
      "\n",
      "  downloading [=======================================>--------]  83% eta:  1s\n",
      "\n",
      "  downloading [===========================================>----]  92% eta:  1s\n",
      "\n",
      "  downloading [================================================] 100% eta:  0s\n",
      "                                                                              \n",
      "\n"
     ]
    }
   ],
   "source": [
    "my_studies <- get_studies(efo_trait = 'type 2 diabetes mellitus')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "0fb6cf2a-ba1e-4a72-b597-e93600e2b6e5",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "229"
      ],
      "text/latex": [
       "229"
      ],
      "text/markdown": [
       "229"
      ],
      "text/plain": [
       "[1] 229"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<style>\n",
       ".list-inline {list-style: none; margin:0; padding: 0}\n",
       ".list-inline>li {display: inline-block}\n",
       ".list-inline>li:not(:last-child)::after {content: \"\\00b7\"; padding: 0 .5ex}\n",
       "</style>\n",
       "<ol class=list-inline><li>'GCST000073'</li><li>'GCST000074'</li><li>'GCST000075'</li><li>'GCST000076'</li><li>'GCST000049'</li><li>'GCST000277'</li><li>'GCST000024'</li><li>'GCST000025'</li><li>'GCST000012'</li><li>'GCST000167'</li><li>'GCST000219'</li><li>'GCST000221'</li><li>'GCST000383'</li><li>'GCST000665'</li><li>'GCST000712'</li><li>'GCST000787'</li><li>'GCST000796'</li><li>'GCST001033'</li><li>'GCST001070'</li><li>'GCST001092'</li><li>'GCST001759'</li><li>'GCST001461'</li><li>'GCST001809'</li><li>'GCST001173'</li><li>'GCST001843'</li><li>'GCST001369'</li><li>'GCST001550'</li><li>'GCST001213'</li><li>'GCST001397'</li><li>'GCST002123'</li><li>'GCST001919'</li><li>'GCST002128'</li><li>'GCST001351'</li><li>'GCST001965'</li><li>'GCST002560'</li><li>'GCST002224'</li><li>'GCST000027'</li><li>'GCST002339'</li><li>'GCST002025'</li><li>'GCST002317'</li><li>'GCST001666'</li><li>'GCST002352'</li><li>'GCST000976'</li><li>'GCST002718'</li><li>'GCST002806'</li><li>'GCST003400'</li><li>'GCST008434'</li><li>'GCST007120'</li><li>'GCST003457'</li><li>'GCST004758'</li><li>'GCST004305'</li><li>'GCST004880'</li><li>'GCST004894'</li><li>'GCST003619'</li><li>'GCST004125'</li><li>'GCST004304'</li><li>'GCST004773'</li><li>'GCST004774'</li><li>'GCST004782'</li><li>'GCST005047'</li><li>'GCST006868'</li><li>'GCST006494'</li><li>'GCST006495'</li><li>'GCST007246'</li><li>'GCST007220'</li><li>'GCST007221'</li><li>'GCST007222'</li><li>'GCST007223'</li><li>'GCST007224'</li><li>'GCST007303'</li><li>'GCST007620'</li><li>'GCST007949'</li><li>'GCST007950'</li><li>'GCST008048'</li><li>'GCST005414'</li><li>'GCST006867'</li><li>'GCST008976'</li><li>'GCST006861'</li><li>'GCST008776'</li><li>'GCST008464'</li><li>'GCST008463'</li><li>'GCST005413'</li><li>'GCST005899'</li><li>'GCST005898'</li><li>'GCST005912'</li><li>'GCST006484'</li><li>'GCST007515'</li><li>'GCST007517'</li><li>'GCST007516'</li><li>'GCST008114'</li><li>'GCST005716'</li><li>'GCST006145'</li><li>'GCST007359'</li><li>'GCST006801'</li><li>'GCST007077'</li><li>'GCST007847'</li><li>'GCST007518'</li><li>'GCST008827'</li><li>'GCST008833'</li><li>'GCST010557'</li><li>'GCST010556'</li><li>'GCST010555'</li><li>'GCST010554'</li><li>'GCST010553'</li><li>'GCST010551'</li><li>'GCST010548'</li><li>'GCST009859'</li><li>'GCST009857'</li><li>'GCST009732'</li><li>'GCST009875'</li><li>'GCST010552'</li><li>'GCST010549'</li><li>'GCST010550'</li><li>'GCST009379'</li><li>'GCST009380'</li><li>'GCST010766'</li><li>'GCST009372'</li><li>'GCST010272'</li><li>'GCST010118'</li><li>'GCST010547'</li><li>'GCST010436'</li><li>'GCST90012181'</li><li>'GCST90013693'</li><li>'GCST90013735'</li><li>'GCST90013770'</li><li>'GCST90013841'</li><li>'GCST011321'</li><li>'GCST011487'</li><li>'GCST011539'</li><li>'GCST011337'</li><li>'GCST011329'</li><li>'GCST011320'</li><li>'GCST90029024'</li><li>'GCST011990'</li><li>'GCST011991'</li><li>'GCST012043'</li><li>'GCST012042'</li><li>'GCST90038634'</li><li>'GCST90053824'</li><li>'GCST90053830'</li><li>'GCST90053826'</li><li>'GCST90053827'</li><li>'GCST90053828'</li><li>'GCST90053829'</li><li>'GCST90296697'</li><li>'GCST90296698'</li><li>'GCST90296699'</li><li>'GCST90296700'</li><li>'GCST90086068'</li><li>'GCST90086069'</li><li>'GCST90086070'</li><li>'GCST90086071'</li><li>'GCST90086072'</li><li>'GCST90026417'</li><li>'GCST90026415'</li><li>'GCST90026414'</li><li>'GCST90026413'</li><li>'GCST90428469'</li><li>'GCST90100587'</li><li>'GCST90013892'</li><li>'GCST90013942'</li><li>'GCST90093109'</li><li>'GCST90093110'</li><li>'GCST90101836'</li><li>'GCST90101837'</li><li>'GCST90006934'</li><li>'GCST90026416'</li><li>'GCST90026412'</li><li>'GCST90043636'</li><li>'GCST90043637'</li><li>'GCST90043638'</li><li>'GCST90018706'</li><li>'GCST90018926'</li><li>'GCST90083728'</li><li>'GCST90083725'</li><li>'GCST90083721'</li><li>'GCST90083720'</li><li>'GCST90083724'</li><li>'GCST90083723'</li><li>'GCST90083722'</li><li>'GCST90083726'</li><li>'GCST90083727'</li><li>'GCST90079740'</li><li>'GCST90079738'</li><li>'GCST90079737'</li><li>'GCST90079736'</li><li>'GCST90081516'</li><li>'GCST90079734'</li><li>'GCST90079741'</li><li>'GCST90079735'</li><li>'GCST90274723'</li><li>'GCST90310232'</li><li>'GCST90081526'</li><li>'GCST90081524'</li><li>'GCST90132183'</li><li>'GCST90132184'</li><li>'GCST90132185'</li><li>'GCST90132186'</li><li>'GCST90132187'</li><li>'GCST90132188'</li><li>'GCST90255648'</li><li>'GCST90246053'</li><li>'GCST90020177'</li><li>'GCST90079739'</li><li>'GCST90079742'</li><li>'GCST90244660'</li><li>'GCST90244661'</li><li>'GCST90103391'</li><li>'GCST90081517'</li><li>'GCST90053852'</li><li>'GCST90179113'</li><li>'GCST90161239'</li><li>'GCST90244709'</li><li>'GCST90245849'</li><li>'GCST90134404'</li><li>'GCST90134405'</li><li>'GCST90225532'</li><li>'GCST90272881'</li><li>'GCST90134620'</li><li>'GCST90137502'</li><li>'GCST90319695'</li><li>'GCST90077724'</li><li>'GCST90077725'</li><li>'GCST90081710'</li><li>'GCST90081711'</li><li>'GCST90085502'</li><li>'GCST90085503'</li><li>'GCST90085510'</li><li>'GCST90085512'</li></ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 'GCST000073'\n",
       "\\item 'GCST000074'\n",
       "\\item 'GCST000075'\n",
       "\\item 'GCST000076'\n",
       "\\item 'GCST000049'\n",
       "\\item 'GCST000277'\n",
       "\\item 'GCST000024'\n",
       "\\item 'GCST000025'\n",
       "\\item 'GCST000012'\n",
       "\\item 'GCST000167'\n",
       "\\item 'GCST000219'\n",
       "\\item 'GCST000221'\n",
       "\\item 'GCST000383'\n",
       "\\item 'GCST000665'\n",
       "\\item 'GCST000712'\n",
       "\\item 'GCST000787'\n",
       "\\item 'GCST000796'\n",
       "\\item 'GCST001033'\n",
       "\\item 'GCST001070'\n",
       "\\item 'GCST001092'\n",
       "\\item 'GCST001759'\n",
       "\\item 'GCST001461'\n",
       "\\item 'GCST001809'\n",
       "\\item 'GCST001173'\n",
       "\\item 'GCST001843'\n",
       "\\item 'GCST001369'\n",
       "\\item 'GCST001550'\n",
       "\\item 'GCST001213'\n",
       "\\item 'GCST001397'\n",
       "\\item 'GCST002123'\n",
       "\\item 'GCST001919'\n",
       "\\item 'GCST002128'\n",
       "\\item 'GCST001351'\n",
       "\\item 'GCST001965'\n",
       "\\item 'GCST002560'\n",
       "\\item 'GCST002224'\n",
       "\\item 'GCST000027'\n",
       "\\item 'GCST002339'\n",
       "\\item 'GCST002025'\n",
       "\\item 'GCST002317'\n",
       "\\item 'GCST001666'\n",
       "\\item 'GCST002352'\n",
       "\\item 'GCST000976'\n",
       "\\item 'GCST002718'\n",
       "\\item 'GCST002806'\n",
       "\\item 'GCST003400'\n",
       "\\item 'GCST008434'\n",
       "\\item 'GCST007120'\n",
       "\\item 'GCST003457'\n",
       "\\item 'GCST004758'\n",
       "\\item 'GCST004305'\n",
       "\\item 'GCST004880'\n",
       "\\item 'GCST004894'\n",
       "\\item 'GCST003619'\n",
       "\\item 'GCST004125'\n",
       "\\item 'GCST004304'\n",
       "\\item 'GCST004773'\n",
       "\\item 'GCST004774'\n",
       "\\item 'GCST004782'\n",
       "\\item 'GCST005047'\n",
       "\\item 'GCST006868'\n",
       "\\item 'GCST006494'\n",
       "\\item 'GCST006495'\n",
       "\\item 'GCST007246'\n",
       "\\item 'GCST007220'\n",
       "\\item 'GCST007221'\n",
       "\\item 'GCST007222'\n",
       "\\item 'GCST007223'\n",
       "\\item 'GCST007224'\n",
       "\\item 'GCST007303'\n",
       "\\item 'GCST007620'\n",
       "\\item 'GCST007949'\n",
       "\\item 'GCST007950'\n",
       "\\item 'GCST008048'\n",
       "\\item 'GCST005414'\n",
       "\\item 'GCST006867'\n",
       "\\item 'GCST008976'\n",
       "\\item 'GCST006861'\n",
       "\\item 'GCST008776'\n",
       "\\item 'GCST008464'\n",
       "\\item 'GCST008463'\n",
       "\\item 'GCST005413'\n",
       "\\item 'GCST005899'\n",
       "\\item 'GCST005898'\n",
       "\\item 'GCST005912'\n",
       "\\item 'GCST006484'\n",
       "\\item 'GCST007515'\n",
       "\\item 'GCST007517'\n",
       "\\item 'GCST007516'\n",
       "\\item 'GCST008114'\n",
       "\\item 'GCST005716'\n",
       "\\item 'GCST006145'\n",
       "\\item 'GCST007359'\n",
       "\\item 'GCST006801'\n",
       "\\item 'GCST007077'\n",
       "\\item 'GCST007847'\n",
       "\\item 'GCST007518'\n",
       "\\item 'GCST008827'\n",
       "\\item 'GCST008833'\n",
       "\\item 'GCST010557'\n",
       "\\item 'GCST010556'\n",
       "\\item 'GCST010555'\n",
       "\\item 'GCST010554'\n",
       "\\item 'GCST010553'\n",
       "\\item 'GCST010551'\n",
       "\\item 'GCST010548'\n",
       "\\item 'GCST009859'\n",
       "\\item 'GCST009857'\n",
       "\\item 'GCST009732'\n",
       "\\item 'GCST009875'\n",
       "\\item 'GCST010552'\n",
       "\\item 'GCST010549'\n",
       "\\item 'GCST010550'\n",
       "\\item 'GCST009379'\n",
       "\\item 'GCST009380'\n",
       "\\item 'GCST010766'\n",
       "\\item 'GCST009372'\n",
       "\\item 'GCST010272'\n",
       "\\item 'GCST010118'\n",
       "\\item 'GCST010547'\n",
       "\\item 'GCST010436'\n",
       "\\item 'GCST90012181'\n",
       "\\item 'GCST90013693'\n",
       "\\item 'GCST90013735'\n",
       "\\item 'GCST90013770'\n",
       "\\item 'GCST90013841'\n",
       "\\item 'GCST011321'\n",
       "\\item 'GCST011487'\n",
       "\\item 'GCST011539'\n",
       "\\item 'GCST011337'\n",
       "\\item 'GCST011329'\n",
       "\\item 'GCST011320'\n",
       "\\item 'GCST90029024'\n",
       "\\item 'GCST011990'\n",
       "\\item 'GCST011991'\n",
       "\\item 'GCST012043'\n",
       "\\item 'GCST012042'\n",
       "\\item 'GCST90038634'\n",
       "\\item 'GCST90053824'\n",
       "\\item 'GCST90053830'\n",
       "\\item 'GCST90053826'\n",
       "\\item 'GCST90053827'\n",
       "\\item 'GCST90053828'\n",
       "\\item 'GCST90053829'\n",
       "\\item 'GCST90296697'\n",
       "\\item 'GCST90296698'\n",
       "\\item 'GCST90296699'\n",
       "\\item 'GCST90296700'\n",
       "\\item 'GCST90086068'\n",
       "\\item 'GCST90086069'\n",
       "\\item 'GCST90086070'\n",
       "\\item 'GCST90086071'\n",
       "\\item 'GCST90086072'\n",
       "\\item 'GCST90026417'\n",
       "\\item 'GCST90026415'\n",
       "\\item 'GCST90026414'\n",
       "\\item 'GCST90026413'\n",
       "\\item 'GCST90428469'\n",
       "\\item 'GCST90100587'\n",
       "\\item 'GCST90013892'\n",
       "\\item 'GCST90013942'\n",
       "\\item 'GCST90093109'\n",
       "\\item 'GCST90093110'\n",
       "\\item 'GCST90101836'\n",
       "\\item 'GCST90101837'\n",
       "\\item 'GCST90006934'\n",
       "\\item 'GCST90026416'\n",
       "\\item 'GCST90026412'\n",
       "\\item 'GCST90043636'\n",
       "\\item 'GCST90043637'\n",
       "\\item 'GCST90043638'\n",
       "\\item 'GCST90018706'\n",
       "\\item 'GCST90018926'\n",
       "\\item 'GCST90083728'\n",
       "\\item 'GCST90083725'\n",
       "\\item 'GCST90083721'\n",
       "\\item 'GCST90083720'\n",
       "\\item 'GCST90083724'\n",
       "\\item 'GCST90083723'\n",
       "\\item 'GCST90083722'\n",
       "\\item 'GCST90083726'\n",
       "\\item 'GCST90083727'\n",
       "\\item 'GCST90079740'\n",
       "\\item 'GCST90079738'\n",
       "\\item 'GCST90079737'\n",
       "\\item 'GCST90079736'\n",
       "\\item 'GCST90081516'\n",
       "\\item 'GCST90079734'\n",
       "\\item 'GCST90079741'\n",
       "\\item 'GCST90079735'\n",
       "\\item 'GCST90274723'\n",
       "\\item 'GCST90310232'\n",
       "\\item 'GCST90081526'\n",
       "\\item 'GCST90081524'\n",
       "\\item 'GCST90132183'\n",
       "\\item 'GCST90132184'\n",
       "\\item 'GCST90132185'\n",
       "\\item 'GCST90132186'\n",
       "\\item 'GCST90132187'\n",
       "\\item 'GCST90132188'\n",
       "\\item 'GCST90255648'\n",
       "\\item 'GCST90246053'\n",
       "\\item 'GCST90020177'\n",
       "\\item 'GCST90079739'\n",
       "\\item 'GCST90079742'\n",
       "\\item 'GCST90244660'\n",
       "\\item 'GCST90244661'\n",
       "\\item 'GCST90103391'\n",
       "\\item 'GCST90081517'\n",
       "\\item 'GCST90053852'\n",
       "\\item 'GCST90179113'\n",
       "\\item 'GCST90161239'\n",
       "\\item 'GCST90244709'\n",
       "\\item 'GCST90245849'\n",
       "\\item 'GCST90134404'\n",
       "\\item 'GCST90134405'\n",
       "\\item 'GCST90225532'\n",
       "\\item 'GCST90272881'\n",
       "\\item 'GCST90134620'\n",
       "\\item 'GCST90137502'\n",
       "\\item 'GCST90319695'\n",
       "\\item 'GCST90077724'\n",
       "\\item 'GCST90077725'\n",
       "\\item 'GCST90081710'\n",
       "\\item 'GCST90081711'\n",
       "\\item 'GCST90085502'\n",
       "\\item 'GCST90085503'\n",
       "\\item 'GCST90085510'\n",
       "\\item 'GCST90085512'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 'GCST000073'\n",
       "2. 'GCST000074'\n",
       "3. 'GCST000075'\n",
       "4. 'GCST000076'\n",
       "5. 'GCST000049'\n",
       "6. 'GCST000277'\n",
       "7. 'GCST000024'\n",
       "8. 'GCST000025'\n",
       "9. 'GCST000012'\n",
       "10. 'GCST000167'\n",
       "11. 'GCST000219'\n",
       "12. 'GCST000221'\n",
       "13. 'GCST000383'\n",
       "14. 'GCST000665'\n",
       "15. 'GCST000712'\n",
       "16. 'GCST000787'\n",
       "17. 'GCST000796'\n",
       "18. 'GCST001033'\n",
       "19. 'GCST001070'\n",
       "20. 'GCST001092'\n",
       "21. 'GCST001759'\n",
       "22. 'GCST001461'\n",
       "23. 'GCST001809'\n",
       "24. 'GCST001173'\n",
       "25. 'GCST001843'\n",
       "26. 'GCST001369'\n",
       "27. 'GCST001550'\n",
       "28. 'GCST001213'\n",
       "29. 'GCST001397'\n",
       "30. 'GCST002123'\n",
       "31. 'GCST001919'\n",
       "32. 'GCST002128'\n",
       "33. 'GCST001351'\n",
       "34. 'GCST001965'\n",
       "35. 'GCST002560'\n",
       "36. 'GCST002224'\n",
       "37. 'GCST000027'\n",
       "38. 'GCST002339'\n",
       "39. 'GCST002025'\n",
       "40. 'GCST002317'\n",
       "41. 'GCST001666'\n",
       "42. 'GCST002352'\n",
       "43. 'GCST000976'\n",
       "44. 'GCST002718'\n",
       "45. 'GCST002806'\n",
       "46. 'GCST003400'\n",
       "47. 'GCST008434'\n",
       "48. 'GCST007120'\n",
       "49. 'GCST003457'\n",
       "50. 'GCST004758'\n",
       "51. 'GCST004305'\n",
       "52. 'GCST004880'\n",
       "53. 'GCST004894'\n",
       "54. 'GCST003619'\n",
       "55. 'GCST004125'\n",
       "56. 'GCST004304'\n",
       "57. 'GCST004773'\n",
       "58. 'GCST004774'\n",
       "59. 'GCST004782'\n",
       "60. 'GCST005047'\n",
       "61. 'GCST006868'\n",
       "62. 'GCST006494'\n",
       "63. 'GCST006495'\n",
       "64. 'GCST007246'\n",
       "65. 'GCST007220'\n",
       "66. 'GCST007221'\n",
       "67. 'GCST007222'\n",
       "68. 'GCST007223'\n",
       "69. 'GCST007224'\n",
       "70. 'GCST007303'\n",
       "71. 'GCST007620'\n",
       "72. 'GCST007949'\n",
       "73. 'GCST007950'\n",
       "74. 'GCST008048'\n",
       "75. 'GCST005414'\n",
       "76. 'GCST006867'\n",
       "77. 'GCST008976'\n",
       "78. 'GCST006861'\n",
       "79. 'GCST008776'\n",
       "80. 'GCST008464'\n",
       "81. 'GCST008463'\n",
       "82. 'GCST005413'\n",
       "83. 'GCST005899'\n",
       "84. 'GCST005898'\n",
       "85. 'GCST005912'\n",
       "86. 'GCST006484'\n",
       "87. 'GCST007515'\n",
       "88. 'GCST007517'\n",
       "89. 'GCST007516'\n",
       "90. 'GCST008114'\n",
       "91. 'GCST005716'\n",
       "92. 'GCST006145'\n",
       "93. 'GCST007359'\n",
       "94. 'GCST006801'\n",
       "95. 'GCST007077'\n",
       "96. 'GCST007847'\n",
       "97. 'GCST007518'\n",
       "98. 'GCST008827'\n",
       "99. 'GCST008833'\n",
       "100. 'GCST010557'\n",
       "101. 'GCST010556'\n",
       "102. 'GCST010555'\n",
       "103. 'GCST010554'\n",
       "104. 'GCST010553'\n",
       "105. 'GCST010551'\n",
       "106. 'GCST010548'\n",
       "107. 'GCST009859'\n",
       "108. 'GCST009857'\n",
       "109. 'GCST009732'\n",
       "110. 'GCST009875'\n",
       "111. 'GCST010552'\n",
       "112. 'GCST010549'\n",
       "113. 'GCST010550'\n",
       "114. 'GCST009379'\n",
       "115. 'GCST009380'\n",
       "116. 'GCST010766'\n",
       "117. 'GCST009372'\n",
       "118. 'GCST010272'\n",
       "119. 'GCST010118'\n",
       "120. 'GCST010547'\n",
       "121. 'GCST010436'\n",
       "122. 'GCST90012181'\n",
       "123. 'GCST90013693'\n",
       "124. 'GCST90013735'\n",
       "125. 'GCST90013770'\n",
       "126. 'GCST90013841'\n",
       "127. 'GCST011321'\n",
       "128. 'GCST011487'\n",
       "129. 'GCST011539'\n",
       "130. 'GCST011337'\n",
       "131. 'GCST011329'\n",
       "132. 'GCST011320'\n",
       "133. 'GCST90029024'\n",
       "134. 'GCST011990'\n",
       "135. 'GCST011991'\n",
       "136. 'GCST012043'\n",
       "137. 'GCST012042'\n",
       "138. 'GCST90038634'\n",
       "139. 'GCST90053824'\n",
       "140. 'GCST90053830'\n",
       "141. 'GCST90053826'\n",
       "142. 'GCST90053827'\n",
       "143. 'GCST90053828'\n",
       "144. 'GCST90053829'\n",
       "145. 'GCST90296697'\n",
       "146. 'GCST90296698'\n",
       "147. 'GCST90296699'\n",
       "148. 'GCST90296700'\n",
       "149. 'GCST90086068'\n",
       "150. 'GCST90086069'\n",
       "151. 'GCST90086070'\n",
       "152. 'GCST90086071'\n",
       "153. 'GCST90086072'\n",
       "154. 'GCST90026417'\n",
       "155. 'GCST90026415'\n",
       "156. 'GCST90026414'\n",
       "157. 'GCST90026413'\n",
       "158. 'GCST90428469'\n",
       "159. 'GCST90100587'\n",
       "160. 'GCST90013892'\n",
       "161. 'GCST90013942'\n",
       "162. 'GCST90093109'\n",
       "163. 'GCST90093110'\n",
       "164. 'GCST90101836'\n",
       "165. 'GCST90101837'\n",
       "166. 'GCST90006934'\n",
       "167. 'GCST90026416'\n",
       "168. 'GCST90026412'\n",
       "169. 'GCST90043636'\n",
       "170. 'GCST90043637'\n",
       "171. 'GCST90043638'\n",
       "172. 'GCST90018706'\n",
       "173. 'GCST90018926'\n",
       "174. 'GCST90083728'\n",
       "175. 'GCST90083725'\n",
       "176. 'GCST90083721'\n",
       "177. 'GCST90083720'\n",
       "178. 'GCST90083724'\n",
       "179. 'GCST90083723'\n",
       "180. 'GCST90083722'\n",
       "181. 'GCST90083726'\n",
       "182. 'GCST90083727'\n",
       "183. 'GCST90079740'\n",
       "184. 'GCST90079738'\n",
       "185. 'GCST90079737'\n",
       "186. 'GCST90079736'\n",
       "187. 'GCST90081516'\n",
       "188. 'GCST90079734'\n",
       "189. 'GCST90079741'\n",
       "190. 'GCST90079735'\n",
       "191. 'GCST90274723'\n",
       "192. 'GCST90310232'\n",
       "193. 'GCST90081526'\n",
       "194. 'GCST90081524'\n",
       "195. 'GCST90132183'\n",
       "196. 'GCST90132184'\n",
       "197. 'GCST90132185'\n",
       "198. 'GCST90132186'\n",
       "199. 'GCST90132187'\n",
       "200. 'GCST90132188'\n",
       "201. 'GCST90255648'\n",
       "202. 'GCST90246053'\n",
       "203. 'GCST90020177'\n",
       "204. 'GCST90079739'\n",
       "205. 'GCST90079742'\n",
       "206. 'GCST90244660'\n",
       "207. 'GCST90244661'\n",
       "208. 'GCST90103391'\n",
       "209. 'GCST90081517'\n",
       "210. 'GCST90053852'\n",
       "211. 'GCST90179113'\n",
       "212. 'GCST90161239'\n",
       "213. 'GCST90244709'\n",
       "214. 'GCST90245849'\n",
       "215. 'GCST90134404'\n",
       "216. 'GCST90134405'\n",
       "217. 'GCST90225532'\n",
       "218. 'GCST90272881'\n",
       "219. 'GCST90134620'\n",
       "220. 'GCST90137502'\n",
       "221. 'GCST90319695'\n",
       "222. 'GCST90077724'\n",
       "223. 'GCST90077725'\n",
       "224. 'GCST90081710'\n",
       "225. 'GCST90081711'\n",
       "226. 'GCST90085502'\n",
       "227. 'GCST90085503'\n",
       "228. 'GCST90085510'\n",
       "229. 'GCST90085512'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  [1] \"GCST000073\"   \"GCST000074\"   \"GCST000075\"   \"GCST000076\"   \"GCST000049\"  \n",
       "  [6] \"GCST000277\"   \"GCST000024\"   \"GCST000025\"   \"GCST000012\"   \"GCST000167\"  \n",
       " [11] \"GCST000219\"   \"GCST000221\"   \"GCST000383\"   \"GCST000665\"   \"GCST000712\"  \n",
       " [16] \"GCST000787\"   \"GCST000796\"   \"GCST001033\"   \"GCST001070\"   \"GCST001092\"  \n",
       " [21] \"GCST001759\"   \"GCST001461\"   \"GCST001809\"   \"GCST001173\"   \"GCST001843\"  \n",
       " [26] \"GCST001369\"   \"GCST001550\"   \"GCST001213\"   \"GCST001397\"   \"GCST002123\"  \n",
       " [31] \"GCST001919\"   \"GCST002128\"   \"GCST001351\"   \"GCST001965\"   \"GCST002560\"  \n",
       " [36] \"GCST002224\"   \"GCST000027\"   \"GCST002339\"   \"GCST002025\"   \"GCST002317\"  \n",
       " [41] \"GCST001666\"   \"GCST002352\"   \"GCST000976\"   \"GCST002718\"   \"GCST002806\"  \n",
       " [46] \"GCST003400\"   \"GCST008434\"   \"GCST007120\"   \"GCST003457\"   \"GCST004758\"  \n",
       " [51] \"GCST004305\"   \"GCST004880\"   \"GCST004894\"   \"GCST003619\"   \"GCST004125\"  \n",
       " [56] \"GCST004304\"   \"GCST004773\"   \"GCST004774\"   \"GCST004782\"   \"GCST005047\"  \n",
       " [61] \"GCST006868\"   \"GCST006494\"   \"GCST006495\"   \"GCST007246\"   \"GCST007220\"  \n",
       " [66] \"GCST007221\"   \"GCST007222\"   \"GCST007223\"   \"GCST007224\"   \"GCST007303\"  \n",
       " [71] \"GCST007620\"   \"GCST007949\"   \"GCST007950\"   \"GCST008048\"   \"GCST005414\"  \n",
       " [76] \"GCST006867\"   \"GCST008976\"   \"GCST006861\"   \"GCST008776\"   \"GCST008464\"  \n",
       " [81] \"GCST008463\"   \"GCST005413\"   \"GCST005899\"   \"GCST005898\"   \"GCST005912\"  \n",
       " [86] \"GCST006484\"   \"GCST007515\"   \"GCST007517\"   \"GCST007516\"   \"GCST008114\"  \n",
       " [91] \"GCST005716\"   \"GCST006145\"   \"GCST007359\"   \"GCST006801\"   \"GCST007077\"  \n",
       " [96] \"GCST007847\"   \"GCST007518\"   \"GCST008827\"   \"GCST008833\"   \"GCST010557\"  \n",
       "[101] \"GCST010556\"   \"GCST010555\"   \"GCST010554\"   \"GCST010553\"   \"GCST010551\"  \n",
       "[106] \"GCST010548\"   \"GCST009859\"   \"GCST009857\"   \"GCST009732\"   \"GCST009875\"  \n",
       "[111] \"GCST010552\"   \"GCST010549\"   \"GCST010550\"   \"GCST009379\"   \"GCST009380\"  \n",
       "[116] \"GCST010766\"   \"GCST009372\"   \"GCST010272\"   \"GCST010118\"   \"GCST010547\"  \n",
       "[121] \"GCST010436\"   \"GCST90012181\" \"GCST90013693\" \"GCST90013735\" \"GCST90013770\"\n",
       "[126] \"GCST90013841\" \"GCST011321\"   \"GCST011487\"   \"GCST011539\"   \"GCST011337\"  \n",
       "[131] \"GCST011329\"   \"GCST011320\"   \"GCST90029024\" \"GCST011990\"   \"GCST011991\"  \n",
       "[136] \"GCST012043\"   \"GCST012042\"   \"GCST90038634\" \"GCST90053824\" \"GCST90053830\"\n",
       "[141] \"GCST90053826\" \"GCST90053827\" \"GCST90053828\" \"GCST90053829\" \"GCST90296697\"\n",
       "[146] \"GCST90296698\" \"GCST90296699\" \"GCST90296700\" \"GCST90086068\" \"GCST90086069\"\n",
       "[151] \"GCST90086070\" \"GCST90086071\" \"GCST90086072\" \"GCST90026417\" \"GCST90026415\"\n",
       "[156] \"GCST90026414\" \"GCST90026413\" \"GCST90428469\" \"GCST90100587\" \"GCST90013892\"\n",
       "[161] \"GCST90013942\" \"GCST90093109\" \"GCST90093110\" \"GCST90101836\" \"GCST90101837\"\n",
       "[166] \"GCST90006934\" \"GCST90026416\" \"GCST90026412\" \"GCST90043636\" \"GCST90043637\"\n",
       "[171] \"GCST90043638\" \"GCST90018706\" \"GCST90018926\" \"GCST90083728\" \"GCST90083725\"\n",
       "[176] \"GCST90083721\" \"GCST90083720\" \"GCST90083724\" \"GCST90083723\" \"GCST90083722\"\n",
       "[181] \"GCST90083726\" \"GCST90083727\" \"GCST90079740\" \"GCST90079738\" \"GCST90079737\"\n",
       "[186] \"GCST90079736\" \"GCST90081516\" \"GCST90079734\" \"GCST90079741\" \"GCST90079735\"\n",
       "[191] \"GCST90274723\" \"GCST90310232\" \"GCST90081526\" \"GCST90081524\" \"GCST90132183\"\n",
       "[196] \"GCST90132184\" \"GCST90132185\" \"GCST90132186\" \"GCST90132187\" \"GCST90132188\"\n",
       "[201] \"GCST90255648\" \"GCST90246053\" \"GCST90020177\" \"GCST90079739\" \"GCST90079742\"\n",
       "[206] \"GCST90244660\" \"GCST90244661\" \"GCST90103391\" \"GCST90081517\" \"GCST90053852\"\n",
       "[211] \"GCST90179113\" \"GCST90161239\" \"GCST90244709\" \"GCST90245849\" \"GCST90134404\"\n",
       "[216] \"GCST90134405\" \"GCST90225532\" \"GCST90272881\" \"GCST90134620\" \"GCST90137502\"\n",
       "[221] \"GCST90319695\" \"GCST90077724\" \"GCST90077725\" \"GCST90081710\" \"GCST90081711\"\n",
       "[226] \"GCST90085502\" \"GCST90085503\" \"GCST90085510\" \"GCST90085512\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "gwasrapidd::n(my_studies)\n",
    "my_studies@studies$study_id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "77720273-d01f-48ff-9501-ef6781bca991",
   "metadata": {},
   "outputs": [],
   "source": [
    "open_in_pubmed(my_studies@publications$pubmed_id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "fd903064-3db1-4bbb-823d-0f71dae18484",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<style>\n",
       ".list-inline {list-style: none; margin:0; padding: 0}\n",
       ".list-inline>li {display: inline-block}\n",
       ".list-inline>li:not(:last-child)::after {content: \"\\00b7\"; padding: 0 .5ex}\n",
       "</style>\n",
       "<ol class=list-inline><li>'A 100K genome-wide association scan for diabetes and related traits in the Framingham Heart Study: replication and integration with other genome-wide datasets.'</li><li>'A search for variants associated with young-onset type 2 diabetes in American Indians in a 100K genotyping array.'</li><li>'Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies.'</li><li>'Identification of novel candidate genes for type 2 diabetes from a genome-wide association scan in the Old Order Amish: evidence for replication from diabetes-related quantitative traits and from independent populations.'</li><li>'Type 2 diabetes whole-genome association study in four populations: the DiaGen consortium.'</li><li>'Adiposity-related heterogeneity in patterns of type 2 diabetes susceptibility observed in genome-wide association data.'</li><li>'A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants.'</li><li>'Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes.'</li><li>'A genome-wide association study identifies novel risk loci for type 2 diabetes.'</li><li>'Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes.'</li><li>'Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus.'</li><li>'SNPs in KCNQ1 are associated with susceptibility to type 2 diabetes in East Asian and European populations.'</li><li>'Confirmation of multiple risk Loci and genetic impacts by a genome-wide association study of type 2 diabetes in the Japanese population.'</li><li>'Genetic variants at 2q24 are associated with susceptibility to type 2 diabetes.'</li><li>'Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis.'</li><li>'A genome-wide association study in the Japanese population identifies susceptibility loci for type 2 diabetes at UBE2E2 and C2CD4A-C2CD4B.'</li><li>'Identification of new genetic risk variants for type 2 diabetes.'</li><li>'Transferability of type 2 diabetes implicated loci in multi-ethnic cohorts from Southeast Asia.'</li><li>'Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas.'</li><li>'Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals.'</li><li>'Genome-wide association study for type 2 diabetes in Indians identifies a new susceptibility locus at 2q21.'</li><li>'A single-nucleotide polymorphism in ANK1 is associated with susceptibility to type 2 diabetes in Japanese populations.'</li><li>'Genome-wide association study identifies a novel locus contributing to type 2 diabetes susceptibility in Sikhs of Punjabi origin from India.'</li><li>'A genome-wide association study confirms previously reported loci for type 2 diabetes in Han Chinese.'</li><li>'Exploring genome-wide - dietary heme iron intake interactions and the risk of type 2 diabetes.'</li><li>'A genome-wide association search for type 2 diabetes genes in African Americans.'</li><li>'Stratifying type 2 diabetes cases by BMI identifies genetic risk variants in LAMA1 and enrichment for risk variants in lean compared to obese cases.'</li><li>'Genome-wide association study in individuals of South Asian ancestry identifies six new type 2 diabetes susceptibility loci.'</li><li>'1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data.'</li><li>'A genome-wide search for type 2 diabetes susceptibility genes in an extended Arab family.'</li><li>'Genome-wide association study in a Chinese population identifies a susceptibility locus for type 2 diabetes at 7q32 near PAX4.'</li><li>'Genome-wide association study identifies three novel loci for type 2 diabetes.'</li><li>'Meta-analysis of genome-wide association studies identifies eight new loci for type 2 diabetes in east Asians.'</li><li>'New susceptibility loci in MYL2, C12orf51 and OAS1 associated with 1-h plasma glucose as predisposing risk factors for type 2 diabetes in the Korean population.'</li><li>'Meta-analysis of genome-wide association studies in African Americans provides insights into the genetic architecture of type 2 diabetes.'</li><li>'A genome-wide association study in American Indians implicates DNER as a susceptibility locus for type 2 diabetes.'</li><li>'A variant in CDKAL1 influences insulin response and risk of type 2 diabetes.'</li><li>'Familial young-onset diabetes, pre-diabetes and cardiovascular disease are associated with genetic variants of DACH1 in Chinese.'</li><li>'Genetic modifiers of cystic fibrosis-related diabetes.'</li><li>'Sequence variants in SLC16A11 are a common risk factor for type 2 diabetes in Mexico.'</li><li>'A genome-wide association study identifies GRK5 and RASGRP1 as type 2 diabetes loci in Chinese Hans.'</li><li>'Genome-wide trans-ancestry meta-analysis provides insight into the genetic architecture of type 2 diabetes susceptibility.'</li><li>'Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project.'</li><li>'T2DM GWAS in the Lebanese population confirms the role of TCF7L2 and CDKAL1 in disease susceptibility.'</li><li>'First genome-wide association study in an Australian aboriginal population provides insights into genetic risk factors for body mass index and type 2 diabetes.'</li><li>'Genome-wide association studies in the Japanese population identify seven novel loci for type 2 diabetes.'</li><li>'Single nucleotide polymorphisms in type 2 diabetes among Hispanic adults.'</li><li>'Genetic overlap between multiple sclerosis and several cardiovascular disease risk factors.'</li><li>'Genetic analysis of advanced glycation end products in the DHS MIND study.'</li><li>'Genetics of Type 2 Diabetes in U.S. Hispanic/Latino Individuals: Results From the Hispanic Community Health Study/Study of Latinos (HCHS/SOL).'</li><li>'Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.'</li><li>'Linking Alzheimer\\'s disease and type 2 diabetes: Novel shared susceptibility genes detected by cFDR approach.'</li><li>'Identification of new susceptibility loci for type 2 diabetes and shared etiological pathways with coronary heart disease.'</li><li>'Multi-ethnic genome-wide association study identifies novel locus for type 2 diabetes susceptibility.'</li><li>'PROX1 gene CC genotype as a major determinant of early onset of type 2 diabetes in slavic study participants from Action in Diabetes and Vascular Disease: Preterax and Diamicron MR Controlled Evaluation study.'</li><li>'Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.'</li><li>'An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.'</li><li>'An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.'</li><li>'Genome-wide association study identifies novel type II diabetes risk loci in Jordan subpopulations.'</li><li>'Large-scale association analysis provides insights into the genetic architecture and pathophysiology of type 2 diabetes.'</li><li>'Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.'</li><li>'Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.'</li><li>'Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.'</li><li>'First Genome-Wide Association Study of Latent Autoimmune Diabetes in Adults Reveals Novel Insights Linking Immune and Metabolic Diabetes.'</li><li>'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'</li><li>'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'</li><li>'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'</li><li>'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'</li><li>'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'</li><li>'Dissecting the genetic relationship between cardiovascular risk factors and Alzheimer\\'s disease.'</li><li>'Common and rare exonic MUC5B variants associated with type 2 diabetes in Han Chinese.'</li><li>'Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.'</li><li>'Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.'</li><li>'Genetic analyses of diverse populations improves discovery for complex traits.'</li><li>'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'</li><li>'Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.'</li><li>'Multivariate analysis of genome-wide data to identify potential pleiotropic genes for type 2 diabetes, obesity and coronary artery disease using MetaCCA.'</li><li>'Identification of novel variants associated with osteoporosis, type 2 diabetes and potentially pleiotropic loci using pleiotropic cFDR method.'</li><li>'Genetic overlap between birthweight and adult cardiometabolic diseases has implications for genomic medicine.'</li><li>'ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.'</li><li>'ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.'</li><li>'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'</li><li>'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'</li><li>'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'</li><li>'Identification and functional analysis of glycemic trait loci in the China Health and Nutrition Survey.'</li><li>'Pilot genome-wide association study identifying novel risk loci for type 2 diabetes in a Maya population.'</li><li>'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'</li><li>'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'</li><li>'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'</li><li>'Genome-wide association study of type 2 diabetes in Africa.'</li><li>'A Powerful Procedure for Pathway-Based Meta-analysis Using Summary Statistics Identifies 43 Pathways Associated with Type II Diabetes in European Populations.'</li><li>'A Loss-of-Function Splice Acceptor Variant in IGF2 Is Protective for Type 2 Diabetes.'</li><li>'Additional common variants associated with type 2 diabetes and coronary artery disease detected using a pleiotropic cFDR method.'</li><li>'Variants in the FTO and CDKAL1 loci have recessive effects on risk of obesity and type 2 diabetes, respectively.'</li><li>'Leveraging Polygenic Functional Enrichment to Improve GWAS Power.'</li><li>'Identification of 28 new susceptibility loci for type 2 diabetes in the Japanese population.'</li><li>'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'</li><li>'Structural equation modeling for hypertension and type 2 diabetes based on multiple SNPs and multiple phenotypes.'</li><li>'Exome sequencing of 20,791 cases of type 2 diabetes and 24,440 controls.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.'</li><li>'Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.'</li><li>'Identification of novel high-impact recessively inherited type 2 diabetes risk variants in the Greenlandic population.'</li><li>'Meta-analysis of Immunochip data of four autoimmune diseases reveals novel single-disease and cross-phenotype associations.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.'</li><li>'Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.'</li><li>'A Fast and Accurate Method for Genome-Wide Time-to-Event Data Analysis and Its Application to UK Biobank.'</li><li>'Evidence that genes involved in hedgehog signaling are associated with both bipolar disorder and high BMI.'</li><li>'Shared genetic architecture and casual relationship between leptin levels and type 2 diabetes: large-scale cross-trait meta-analysis and Mendelian randomization analysis.'</li><li>'Identification of type 2 diabetes loci in 433,540 East Asian individuals.'</li><li>'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'</li><li>'Exome Chip Analysis of 14,026 Koreans Reve-als Known and Newly Discovered Genetic Loci Associated with Type 2 Diabetes Mellitus.'</li><li>'Investigation of gene-gene interactions in cardiac traits and serum fatty acid levels in the LURIC Health Study.'</li><li>'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'</li><li>'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'</li><li>'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'</li><li>'Multi-trait association studies discover pleiotropic loci between Alzheimer\\'s disease and cardiometabolic traits.'</li><li>'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'</li><li>'Genetic Predisposition to Coronary Artery Disease in Type 2 Diabetes Mellitus.'</li><li>'Analysis of overlapping genetic association in type 1 and type 2 diabetes.'</li><li>'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'</li><li>'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'</li><li>'A powerful method for pleiotropic analysis under composite null hypothesis identifies novel shared loci between Type 2 Diabetes and Prostate Cancer.'</li><li>'Mixed-model association for biobank-scale datasets.'</li><li>'Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.'</li><li>'Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.'</li><li>'The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.'</li><li>'The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.'</li><li>'Common genetic associations between age-related diseases.'</li><li>'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'</li><li>'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'</li><li>'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'</li><li>'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'</li><li>'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'</li><li>'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'</li><li>'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'</li><li>'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'</li><li>'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'</li><li>'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'</li><li>'The impact of non-additive genetic associations on age-related complex diseases.'</li><li>'The impact of non-additive genetic associations on age-related complex diseases.'</li><li>'The impact of non-additive genetic associations on age-related complex diseases.'</li><li>'The impact of non-additive genetic associations on age-related complex diseases.'</li><li>'The impact of non-additive genetic associations on age-related complex diseases.'</li><li>'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'</li><li>'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'</li><li>'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'</li><li>'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'</li><li>'Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores.'</li><li>'Recessive Genome-wide Meta-analysis Illuminates Genetic Architecture of Type 2 Diabetes.'</li><li>'Computationally efficient whole-genome regression for quantitative and binary traits.'</li><li>'Computationally efficient whole-genome regression for quantitative and binary traits.'</li><li>'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'</li><li>'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'</li><li>'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'</li><li>'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'</li><li>'Genome-wide association analysis of type 2 diabetes in the EPIC-InterAct study.'</li><li>'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'</li><li>'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'</li><li>'A generalized linear mixed model association tool for biobank-scale data.'</li><li>'A generalized linear mixed model association tool for biobank-scale data.'</li><li>'A generalized linear mixed model association tool for biobank-scale data.'</li><li>'A cross-population atlas of genetic associations for 220 human phenotypes.'</li><li>'A cross-population atlas of genetic associations for 220 human phenotypes.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Genome-wide association study of cardiometabolic multimorbidity in the UK Biobank.'</li><li>'A multivariate genome-wide association study of psycho-cardiometabolic multimorbidity.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'</li><li>'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'</li><li>'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'</li><li>'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'</li><li>'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'</li><li>'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'</li><li>'The power of TOPMed imputation for the discovery of Latino-enriched rare variants associated with type 2 diabetes.'</li><li>'Genotyping and population characteristics of the China Kadoorie Biobank.'</li><li>'Shared genetic architecture between neuroticism, coronary artery disease and cardiovascular risk factors.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.'</li><li>'Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.'</li><li>'CNest: A novel copy number association discovery method uncovers 862 new associations from 200,629 whole-exome sequence datasets in the UK Biobank.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Body mass index and type 2 diabetes and breast cancer survival: a Mendelian randomization study.'</li><li>'Genetic evidence for a causal relationship between type 2 diabetes and peripheral artery disease in both Europeans and East Asians.'</li><li>'Phenome-wide analysis of Taiwan Biobank reveals novel glycemia-related loci and genetic risks for diabetes.'</li><li>'Evidence that the pituitary gland connects type 2 diabetes mellitus and schizophrenia based on large-scale trans-ethnic genetic analyses.'</li><li>'Integrating polygenic risk scores in the prediction of type 2 diabetes risk and subtypes in British Pakistanis and Bangladeshis: A population-based cohort study.'</li><li>'A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.'</li><li>'A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.'</li><li>'Identifying shared genetic loci between coronavirus disease 2019 and cardiovascular diseases based on cross-trait meta-analysis.'</li><li>'Identification of shared genetic architecture between non-alcoholic fatty liver disease and type 2 diabetes: A genome-wide analysis.'</li><li>'Susceptibility Loci for Type 2 Diabetes in the Ethnically Endogamous Indian Sindhi Population: A Pooled Blood Genome-Wide Association Study.'</li><li>'Association of Polygenic Variants with Type 2 Diabetes Risk and Their Interaction with Lifestyles in Asians.'</li><li>'Common and rare variants associated with cardiometabolic traits across 98,622 whole-genome sequences in the All of Us research program.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li><li>'Exome sequencing and analysis of 454,787 UK Biobank participants.'</li></ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 'A 100K genome-wide association scan for diabetes and related traits in the Framingham Heart Study: replication and integration with other genome-wide datasets.'\n",
       "\\item 'A search for variants associated with young-onset type 2 diabetes in American Indians in a 100K genotyping array.'\n",
       "\\item 'Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies.'\n",
       "\\item 'Identification of novel candidate genes for type 2 diabetes from a genome-wide association scan in the Old Order Amish: evidence for replication from diabetes-related quantitative traits and from independent populations.'\n",
       "\\item 'Type 2 diabetes whole-genome association study in four populations: the DiaGen consortium.'\n",
       "\\item 'Adiposity-related heterogeneity in patterns of type 2 diabetes susceptibility observed in genome-wide association data.'\n",
       "\\item 'A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants.'\n",
       "\\item 'Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes.'\n",
       "\\item 'A genome-wide association study identifies novel risk loci for type 2 diabetes.'\n",
       "\\item 'Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes.'\n",
       "\\item 'Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus.'\n",
       "\\item 'SNPs in KCNQ1 are associated with susceptibility to type 2 diabetes in East Asian and European populations.'\n",
       "\\item 'Confirmation of multiple risk Loci and genetic impacts by a genome-wide association study of type 2 diabetes in the Japanese population.'\n",
       "\\item 'Genetic variants at 2q24 are associated with susceptibility to type 2 diabetes.'\n",
       "\\item 'Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis.'\n",
       "\\item 'A genome-wide association study in the Japanese population identifies susceptibility loci for type 2 diabetes at UBE2E2 and C2CD4A-C2CD4B.'\n",
       "\\item 'Identification of new genetic risk variants for type 2 diabetes.'\n",
       "\\item 'Transferability of type 2 diabetes implicated loci in multi-ethnic cohorts from Southeast Asia.'\n",
       "\\item 'Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas.'\n",
       "\\item 'Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals.'\n",
       "\\item 'Genome-wide association study for type 2 diabetes in Indians identifies a new susceptibility locus at 2q21.'\n",
       "\\item 'A single-nucleotide polymorphism in ANK1 is associated with susceptibility to type 2 diabetes in Japanese populations.'\n",
       "\\item 'Genome-wide association study identifies a novel locus contributing to type 2 diabetes susceptibility in Sikhs of Punjabi origin from India.'\n",
       "\\item 'A genome-wide association study confirms previously reported loci for type 2 diabetes in Han Chinese.'\n",
       "\\item 'Exploring genome-wide - dietary heme iron intake interactions and the risk of type 2 diabetes.'\n",
       "\\item 'A genome-wide association search for type 2 diabetes genes in African Americans.'\n",
       "\\item 'Stratifying type 2 diabetes cases by BMI identifies genetic risk variants in LAMA1 and enrichment for risk variants in lean compared to obese cases.'\n",
       "\\item 'Genome-wide association study in individuals of South Asian ancestry identifies six new type 2 diabetes susceptibility loci.'\n",
       "\\item '1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data.'\n",
       "\\item 'A genome-wide search for type 2 diabetes susceptibility genes in an extended Arab family.'\n",
       "\\item 'Genome-wide association study in a Chinese population identifies a susceptibility locus for type 2 diabetes at 7q32 near PAX4.'\n",
       "\\item 'Genome-wide association study identifies three novel loci for type 2 diabetes.'\n",
       "\\item 'Meta-analysis of genome-wide association studies identifies eight new loci for type 2 diabetes in east Asians.'\n",
       "\\item 'New susceptibility loci in MYL2, C12orf51 and OAS1 associated with 1-h plasma glucose as predisposing risk factors for type 2 diabetes in the Korean population.'\n",
       "\\item 'Meta-analysis of genome-wide association studies in African Americans provides insights into the genetic architecture of type 2 diabetes.'\n",
       "\\item 'A genome-wide association study in American Indians implicates DNER as a susceptibility locus for type 2 diabetes.'\n",
       "\\item 'A variant in CDKAL1 influences insulin response and risk of type 2 diabetes.'\n",
       "\\item 'Familial young-onset diabetes, pre-diabetes and cardiovascular disease are associated with genetic variants of DACH1 in Chinese.'\n",
       "\\item 'Genetic modifiers of cystic fibrosis-related diabetes.'\n",
       "\\item 'Sequence variants in SLC16A11 are a common risk factor for type 2 diabetes in Mexico.'\n",
       "\\item 'A genome-wide association study identifies GRK5 and RASGRP1 as type 2 diabetes loci in Chinese Hans.'\n",
       "\\item 'Genome-wide trans-ancestry meta-analysis provides insight into the genetic architecture of type 2 diabetes susceptibility.'\n",
       "\\item 'Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project.'\n",
       "\\item 'T2DM GWAS in the Lebanese population confirms the role of TCF7L2 and CDKAL1 in disease susceptibility.'\n",
       "\\item 'First genome-wide association study in an Australian aboriginal population provides insights into genetic risk factors for body mass index and type 2 diabetes.'\n",
       "\\item 'Genome-wide association studies in the Japanese population identify seven novel loci for type 2 diabetes.'\n",
       "\\item 'Single nucleotide polymorphisms in type 2 diabetes among Hispanic adults.'\n",
       "\\item 'Genetic overlap between multiple sclerosis and several cardiovascular disease risk factors.'\n",
       "\\item 'Genetic analysis of advanced glycation end products in the DHS MIND study.'\n",
       "\\item 'Genetics of Type 2 Diabetes in U.S. Hispanic/Latino Individuals: Results From the Hispanic Community Health Study/Study of Latinos (HCHS/SOL).'\n",
       "\\item 'Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.'\n",
       "\\item 'Linking Alzheimer\\textbackslash{}'s disease and type 2 diabetes: Novel shared susceptibility genes detected by cFDR approach.'\n",
       "\\item 'Identification of new susceptibility loci for type 2 diabetes and shared etiological pathways with coronary heart disease.'\n",
       "\\item 'Multi-ethnic genome-wide association study identifies novel locus for type 2 diabetes susceptibility.'\n",
       "\\item 'PROX1 gene CC genotype as a major determinant of early onset of type 2 diabetes in slavic study participants from Action in Diabetes and Vascular Disease: Preterax and Diamicron MR Controlled Evaluation study.'\n",
       "\\item 'Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.'\n",
       "\\item 'An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.'\n",
       "\\item 'An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.'\n",
       "\\item 'Genome-wide association study identifies novel type II diabetes risk loci in Jordan subpopulations.'\n",
       "\\item 'Large-scale association analysis provides insights into the genetic architecture and pathophysiology of type 2 diabetes.'\n",
       "\\item 'Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.'\n",
       "\\item 'Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.'\n",
       "\\item 'Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.'\n",
       "\\item 'First Genome-Wide Association Study of Latent Autoimmune Diabetes in Adults Reveals Novel Insights Linking Immune and Metabolic Diabetes.'\n",
       "\\item 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "\\item 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "\\item 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "\\item 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "\\item 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "\\item 'Dissecting the genetic relationship between cardiovascular risk factors and Alzheimer\\textbackslash{}'s disease.'\n",
       "\\item 'Common and rare exonic MUC5B variants associated with type 2 diabetes in Han Chinese.'\n",
       "\\item 'Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.'\n",
       "\\item 'Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.'\n",
       "\\item 'Genetic analyses of diverse populations improves discovery for complex traits.'\n",
       "\\item 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "\\item 'Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.'\n",
       "\\item 'Multivariate analysis of genome-wide data to identify potential pleiotropic genes for type 2 diabetes, obesity and coronary artery disease using MetaCCA.'\n",
       "\\item 'Identification of novel variants associated with osteoporosis, type 2 diabetes and potentially pleiotropic loci using pleiotropic cFDR method.'\n",
       "\\item 'Genetic overlap between birthweight and adult cardiometabolic diseases has implications for genomic medicine.'\n",
       "\\item 'ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.'\n",
       "\\item 'ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.'\n",
       "\\item 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "\\item 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "\\item 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "\\item 'Identification and functional analysis of glycemic trait loci in the China Health and Nutrition Survey.'\n",
       "\\item 'Pilot genome-wide association study identifying novel risk loci for type 2 diabetes in a Maya population.'\n",
       "\\item 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "\\item 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "\\item 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "\\item 'Genome-wide association study of type 2 diabetes in Africa.'\n",
       "\\item 'A Powerful Procedure for Pathway-Based Meta-analysis Using Summary Statistics Identifies 43 Pathways Associated with Type II Diabetes in European Populations.'\n",
       "\\item 'A Loss-of-Function Splice Acceptor Variant in IGF2 Is Protective for Type 2 Diabetes.'\n",
       "\\item 'Additional common variants associated with type 2 diabetes and coronary artery disease detected using a pleiotropic cFDR method.'\n",
       "\\item 'Variants in the FTO and CDKAL1 loci have recessive effects on risk of obesity and type 2 diabetes, respectively.'\n",
       "\\item 'Leveraging Polygenic Functional Enrichment to Improve GWAS Power.'\n",
       "\\item 'Identification of 28 new susceptibility loci for type 2 diabetes in the Japanese population.'\n",
       "\\item 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "\\item 'Structural equation modeling for hypertension and type 2 diabetes based on multiple SNPs and multiple phenotypes.'\n",
       "\\item 'Exome sequencing of 20,791 cases of type 2 diabetes and 24,440 controls.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.'\n",
       "\\item 'Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.'\n",
       "\\item 'Identification of novel high-impact recessively inherited type 2 diabetes risk variants in the Greenlandic population.'\n",
       "\\item 'Meta-analysis of Immunochip data of four autoimmune diseases reveals novel single-disease and cross-phenotype associations.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.'\n",
       "\\item 'Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.'\n",
       "\\item 'A Fast and Accurate Method for Genome-Wide Time-to-Event Data Analysis and Its Application to UK Biobank.'\n",
       "\\item 'Evidence that genes involved in hedgehog signaling are associated with both bipolar disorder and high BMI.'\n",
       "\\item 'Shared genetic architecture and casual relationship between leptin levels and type 2 diabetes: large-scale cross-trait meta-analysis and Mendelian randomization analysis.'\n",
       "\\item 'Identification of type 2 diabetes loci in 433,540 East Asian individuals.'\n",
       "\\item 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "\\item 'Exome Chip Analysis of 14,026 Koreans Reve-als Known and Newly Discovered Genetic Loci Associated with Type 2 Diabetes Mellitus.'\n",
       "\\item 'Investigation of gene-gene interactions in cardiac traits and serum fatty acid levels in the LURIC Health Study.'\n",
       "\\item 'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'\n",
       "\\item 'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'\n",
       "\\item 'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'\n",
       "\\item 'Multi-trait association studies discover pleiotropic loci between Alzheimer\\textbackslash{}'s disease and cardiometabolic traits.'\n",
       "\\item 'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'\n",
       "\\item 'Genetic Predisposition to Coronary Artery Disease in Type 2 Diabetes Mellitus.'\n",
       "\\item 'Analysis of overlapping genetic association in type 1 and type 2 diabetes.'\n",
       "\\item 'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'\n",
       "\\item 'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'\n",
       "\\item 'A powerful method for pleiotropic analysis under composite null hypothesis identifies novel shared loci between Type 2 Diabetes and Prostate Cancer.'\n",
       "\\item 'Mixed-model association for biobank-scale datasets.'\n",
       "\\item 'Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.'\n",
       "\\item 'Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.'\n",
       "\\item 'The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.'\n",
       "\\item 'The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.'\n",
       "\\item 'Common genetic associations between age-related diseases.'\n",
       "\\item 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "\\item 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "\\item 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "\\item 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "\\item 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "\\item 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "\\item 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "\\item 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "\\item 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "\\item 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "\\item 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "\\item 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "\\item 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "\\item 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "\\item 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "\\item 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "\\item 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "\\item 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "\\item 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "\\item 'Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores.'\n",
       "\\item 'Recessive Genome-wide Meta-analysis Illuminates Genetic Architecture of Type 2 Diabetes.'\n",
       "\\item 'Computationally efficient whole-genome regression for quantitative and binary traits.'\n",
       "\\item 'Computationally efficient whole-genome regression for quantitative and binary traits.'\n",
       "\\item 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "\\item 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "\\item 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "\\item 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "\\item 'Genome-wide association analysis of type 2 diabetes in the EPIC-InterAct study.'\n",
       "\\item 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "\\item 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "\\item 'A generalized linear mixed model association tool for biobank-scale data.'\n",
       "\\item 'A generalized linear mixed model association tool for biobank-scale data.'\n",
       "\\item 'A generalized linear mixed model association tool for biobank-scale data.'\n",
       "\\item 'A cross-population atlas of genetic associations for 220 human phenotypes.'\n",
       "\\item 'A cross-population atlas of genetic associations for 220 human phenotypes.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Genome-wide association study of cardiometabolic multimorbidity in the UK Biobank.'\n",
       "\\item 'A multivariate genome-wide association study of psycho-cardiometabolic multimorbidity.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "\\item 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "\\item 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "\\item 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "\\item 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "\\item 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "\\item 'The power of TOPMed imputation for the discovery of Latino-enriched rare variants associated with type 2 diabetes.'\n",
       "\\item 'Genotyping and population characteristics of the China Kadoorie Biobank.'\n",
       "\\item 'Shared genetic architecture between neuroticism, coronary artery disease and cardiovascular risk factors.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.'\n",
       "\\item 'Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.'\n",
       "\\item 'CNest: A novel copy number association discovery method uncovers 862 new associations from 200,629 whole-exome sequence datasets in the UK Biobank.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Body mass index and type 2 diabetes and breast cancer survival: a Mendelian randomization study.'\n",
       "\\item 'Genetic evidence for a causal relationship between type 2 diabetes and peripheral artery disease in both Europeans and East Asians.'\n",
       "\\item 'Phenome-wide analysis of Taiwan Biobank reveals novel glycemia-related loci and genetic risks for diabetes.'\n",
       "\\item 'Evidence that the pituitary gland connects type 2 diabetes mellitus and schizophrenia based on large-scale trans-ethnic genetic analyses.'\n",
       "\\item 'Integrating polygenic risk scores in the prediction of type 2 diabetes risk and subtypes in British Pakistanis and Bangladeshis: A population-based cohort study.'\n",
       "\\item 'A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.'\n",
       "\\item 'A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.'\n",
       "\\item 'Identifying shared genetic loci between coronavirus disease 2019 and cardiovascular diseases based on cross-trait meta-analysis.'\n",
       "\\item 'Identification of shared genetic architecture between non-alcoholic fatty liver disease and type 2 diabetes: A genome-wide analysis.'\n",
       "\\item 'Susceptibility Loci for Type 2 Diabetes in the Ethnically Endogamous Indian Sindhi Population: A Pooled Blood Genome-Wide Association Study.'\n",
       "\\item 'Association of Polygenic Variants with Type 2 Diabetes Risk and Their Interaction with Lifestyles in Asians.'\n",
       "\\item 'Common and rare variants associated with cardiometabolic traits across 98,622 whole-genome sequences in the All of Us research program.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\item 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 'A 100K genome-wide association scan for diabetes and related traits in the Framingham Heart Study: replication and integration with other genome-wide datasets.'\n",
       "2. 'A search for variants associated with young-onset type 2 diabetes in American Indians in a 100K genotyping array.'\n",
       "3. 'Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies.'\n",
       "4. 'Identification of novel candidate genes for type 2 diabetes from a genome-wide association scan in the Old Order Amish: evidence for replication from diabetes-related quantitative traits and from independent populations.'\n",
       "5. 'Type 2 diabetes whole-genome association study in four populations: the DiaGen consortium.'\n",
       "6. 'Adiposity-related heterogeneity in patterns of type 2 diabetes susceptibility observed in genome-wide association data.'\n",
       "7. 'A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants.'\n",
       "8. 'Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes.'\n",
       "9. 'A genome-wide association study identifies novel risk loci for type 2 diabetes.'\n",
       "10. 'Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes.'\n",
       "11. 'Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus.'\n",
       "12. 'SNPs in KCNQ1 are associated with susceptibility to type 2 diabetes in East Asian and European populations.'\n",
       "13. 'Confirmation of multiple risk Loci and genetic impacts by a genome-wide association study of type 2 diabetes in the Japanese population.'\n",
       "14. 'Genetic variants at 2q24 are associated with susceptibility to type 2 diabetes.'\n",
       "15. 'Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis.'\n",
       "16. 'A genome-wide association study in the Japanese population identifies susceptibility loci for type 2 diabetes at UBE2E2 and C2CD4A-C2CD4B.'\n",
       "17. 'Identification of new genetic risk variants for type 2 diabetes.'\n",
       "18. 'Transferability of type 2 diabetes implicated loci in multi-ethnic cohorts from Southeast Asia.'\n",
       "19. 'Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas.'\n",
       "20. 'Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals.'\n",
       "21. 'Genome-wide association study for type 2 diabetes in Indians identifies a new susceptibility locus at 2q21.'\n",
       "22. 'A single-nucleotide polymorphism in ANK1 is associated with susceptibility to type 2 diabetes in Japanese populations.'\n",
       "23. 'Genome-wide association study identifies a novel locus contributing to type 2 diabetes susceptibility in Sikhs of Punjabi origin from India.'\n",
       "24. 'A genome-wide association study confirms previously reported loci for type 2 diabetes in Han Chinese.'\n",
       "25. 'Exploring genome-wide - dietary heme iron intake interactions and the risk of type 2 diabetes.'\n",
       "26. 'A genome-wide association search for type 2 diabetes genes in African Americans.'\n",
       "27. 'Stratifying type 2 diabetes cases by BMI identifies genetic risk variants in LAMA1 and enrichment for risk variants in lean compared to obese cases.'\n",
       "28. 'Genome-wide association study in individuals of South Asian ancestry identifies six new type 2 diabetes susceptibility loci.'\n",
       "29. '1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data.'\n",
       "30. 'A genome-wide search for type 2 diabetes susceptibility genes in an extended Arab family.'\n",
       "31. 'Genome-wide association study in a Chinese population identifies a susceptibility locus for type 2 diabetes at 7q32 near PAX4.'\n",
       "32. 'Genome-wide association study identifies three novel loci for type 2 diabetes.'\n",
       "33. 'Meta-analysis of genome-wide association studies identifies eight new loci for type 2 diabetes in east Asians.'\n",
       "34. 'New susceptibility loci in MYL2, C12orf51 and OAS1 associated with 1-h plasma glucose as predisposing risk factors for type 2 diabetes in the Korean population.'\n",
       "35. 'Meta-analysis of genome-wide association studies in African Americans provides insights into the genetic architecture of type 2 diabetes.'\n",
       "36. 'A genome-wide association study in American Indians implicates DNER as a susceptibility locus for type 2 diabetes.'\n",
       "37. 'A variant in CDKAL1 influences insulin response and risk of type 2 diabetes.'\n",
       "38. 'Familial young-onset diabetes, pre-diabetes and cardiovascular disease are associated with genetic variants of DACH1 in Chinese.'\n",
       "39. 'Genetic modifiers of cystic fibrosis-related diabetes.'\n",
       "40. 'Sequence variants in SLC16A11 are a common risk factor for type 2 diabetes in Mexico.'\n",
       "41. 'A genome-wide association study identifies GRK5 and RASGRP1 as type 2 diabetes loci in Chinese Hans.'\n",
       "42. 'Genome-wide trans-ancestry meta-analysis provides insight into the genetic architecture of type 2 diabetes susceptibility.'\n",
       "43. 'Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project.'\n",
       "44. 'T2DM GWAS in the Lebanese population confirms the role of TCF7L2 and CDKAL1 in disease susceptibility.'\n",
       "45. 'First genome-wide association study in an Australian aboriginal population provides insights into genetic risk factors for body mass index and type 2 diabetes.'\n",
       "46. 'Genome-wide association studies in the Japanese population identify seven novel loci for type 2 diabetes.'\n",
       "47. 'Single nucleotide polymorphisms in type 2 diabetes among Hispanic adults.'\n",
       "48. 'Genetic overlap between multiple sclerosis and several cardiovascular disease risk factors.'\n",
       "49. 'Genetic analysis of advanced glycation end products in the DHS MIND study.'\n",
       "50. 'Genetics of Type 2 Diabetes in U.S. Hispanic/Latino Individuals: Results From the Hispanic Community Health Study/Study of Latinos (HCHS/SOL).'\n",
       "51. 'Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.'\n",
       "52. 'Linking Alzheimer\\'s disease and type 2 diabetes: Novel shared susceptibility genes detected by cFDR approach.'\n",
       "53. 'Identification of new susceptibility loci for type 2 diabetes and shared etiological pathways with coronary heart disease.'\n",
       "54. 'Multi-ethnic genome-wide association study identifies novel locus for type 2 diabetes susceptibility.'\n",
       "55. 'PROX1 gene CC genotype as a major determinant of early onset of type 2 diabetes in slavic study participants from Action in Diabetes and Vascular Disease: Preterax and Diamicron MR Controlled Evaluation study.'\n",
       "56. 'Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.'\n",
       "57. 'An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.'\n",
       "58. 'An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.'\n",
       "59. 'Genome-wide association study identifies novel type II diabetes risk loci in Jordan subpopulations.'\n",
       "60. 'Large-scale association analysis provides insights into the genetic architecture and pathophysiology of type 2 diabetes.'\n",
       "61. 'Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.'\n",
       "62. 'Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.'\n",
       "63. 'Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.'\n",
       "64. 'First Genome-Wide Association Study of Latent Autoimmune Diabetes in Adults Reveals Novel Insights Linking Immune and Metabolic Diabetes.'\n",
       "65. 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "66. 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "67. 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "68. 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "69. 'Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.'\n",
       "70. 'Dissecting the genetic relationship between cardiovascular risk factors and Alzheimer\\'s disease.'\n",
       "71. 'Common and rare exonic MUC5B variants associated with type 2 diabetes in Han Chinese.'\n",
       "72. 'Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.'\n",
       "73. 'Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.'\n",
       "74. 'Genetic analyses of diverse populations improves discovery for complex traits.'\n",
       "75. 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "76. 'Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.'\n",
       "77. 'Multivariate analysis of genome-wide data to identify potential pleiotropic genes for type 2 diabetes, obesity and coronary artery disease using MetaCCA.'\n",
       "78. 'Identification of novel variants associated with osteoporosis, type 2 diabetes and potentially pleiotropic loci using pleiotropic cFDR method.'\n",
       "79. 'Genetic overlap between birthweight and adult cardiometabolic diseases has implications for genomic medicine.'\n",
       "80. 'ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.'\n",
       "81. 'ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.'\n",
       "82. 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "83. 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "84. 'Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.'\n",
       "85. 'Identification and functional analysis of glycemic trait loci in the China Health and Nutrition Survey.'\n",
       "86. 'Pilot genome-wide association study identifying novel risk loci for type 2 diabetes in a Maya population.'\n",
       "87. 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "88. 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "89. 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "90. 'Genome-wide association study of type 2 diabetes in Africa.'\n",
       "91. 'A Powerful Procedure for Pathway-Based Meta-analysis Using Summary Statistics Identifies 43 Pathways Associated with Type II Diabetes in European Populations.'\n",
       "92. 'A Loss-of-Function Splice Acceptor Variant in IGF2 Is Protective for Type 2 Diabetes.'\n",
       "93. 'Additional common variants associated with type 2 diabetes and coronary artery disease detected using a pleiotropic cFDR method.'\n",
       "94. 'Variants in the FTO and CDKAL1 loci have recessive effects on risk of obesity and type 2 diabetes, respectively.'\n",
       "95. 'Leveraging Polygenic Functional Enrichment to Improve GWAS Power.'\n",
       "96. 'Identification of 28 new susceptibility loci for type 2 diabetes in the Japanese population.'\n",
       "97. 'Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.'\n",
       "98. 'Structural equation modeling for hypertension and type 2 diabetes based on multiple SNPs and multiple phenotypes.'\n",
       "99. 'Exome sequencing of 20,791 cases of type 2 diabetes and 24,440 controls.'\n",
       "100. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "101. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "102. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "103. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "104. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "105. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "106. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "107. 'Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.'\n",
       "108. 'Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.'\n",
       "109. 'Identification of novel high-impact recessively inherited type 2 diabetes risk variants in the Greenlandic population.'\n",
       "110. 'Meta-analysis of Immunochip data of four autoimmune diseases reveals novel single-disease and cross-phenotype associations.'\n",
       "111. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "112. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "113. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "114. 'Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.'\n",
       "115. 'Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.'\n",
       "116. 'A Fast and Accurate Method for Genome-Wide Time-to-Event Data Analysis and Its Application to UK Biobank.'\n",
       "117. 'Evidence that genes involved in hedgehog signaling are associated with both bipolar disorder and high BMI.'\n",
       "118. 'Shared genetic architecture and casual relationship between leptin levels and type 2 diabetes: large-scale cross-trait meta-analysis and Mendelian randomization analysis.'\n",
       "119. 'Identification of type 2 diabetes loci in 433,540 East Asian individuals.'\n",
       "120. 'Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.'\n",
       "121. 'Exome Chip Analysis of 14,026 Koreans Reve-als Known and Newly Discovered Genetic Loci Associated with Type 2 Diabetes Mellitus.'\n",
       "122. 'Investigation of gene-gene interactions in cardiac traits and serum fatty acid levels in the LURIC Health Study.'\n",
       "123. 'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'\n",
       "124. 'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'\n",
       "125. 'Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.'\n",
       "126. 'Multi-trait association studies discover pleiotropic loci between Alzheimer\\'s disease and cardiometabolic traits.'\n",
       "127. 'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'\n",
       "128. 'Genetic Predisposition to Coronary Artery Disease in Type 2 Diabetes Mellitus.'\n",
       "129. 'Analysis of overlapping genetic association in type 1 and type 2 diabetes.'\n",
       "130. 'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'\n",
       "131. 'Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.'\n",
       "132. 'A powerful method for pleiotropic analysis under composite null hypothesis identifies novel shared loci between Type 2 Diabetes and Prostate Cancer.'\n",
       "133. 'Mixed-model association for biobank-scale datasets.'\n",
       "134. 'Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.'\n",
       "135. 'Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.'\n",
       "136. 'The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.'\n",
       "137. 'The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.'\n",
       "138. 'Common genetic associations between age-related diseases.'\n",
       "139. 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "140. 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "141. 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "142. 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "143. 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "144. 'Genetic discovery and risk characterization in type 2 diabetes across diverse populations.'\n",
       "145. 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "146. 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "147. 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "148. 'Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.'\n",
       "149. 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "150. 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "151. 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "152. 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "153. 'The impact of non-additive genetic associations on age-related complex diseases.'\n",
       "154. 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "155. 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "156. 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "157. 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "158. 'Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores.'\n",
       "159. 'Recessive Genome-wide Meta-analysis Illuminates Genetic Architecture of Type 2 Diabetes.'\n",
       "160. 'Computationally efficient whole-genome regression for quantitative and binary traits.'\n",
       "161. 'Computationally efficient whole-genome regression for quantitative and binary traits.'\n",
       "162. 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "163. 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "164. 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "165. 'Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.'\n",
       "166. 'Genome-wide association analysis of type 2 diabetes in the EPIC-InterAct study.'\n",
       "167. 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "168. 'Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.'\n",
       "169. 'A generalized linear mixed model association tool for biobank-scale data.'\n",
       "170. 'A generalized linear mixed model association tool for biobank-scale data.'\n",
       "171. 'A generalized linear mixed model association tool for biobank-scale data.'\n",
       "172. 'A cross-population atlas of genetic associations for 220 human phenotypes.'\n",
       "173. 'A cross-population atlas of genetic associations for 220 human phenotypes.'\n",
       "174. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "175. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "176. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "177. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "178. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "179. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "180. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "181. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "182. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "183. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "184. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "185. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "186. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "187. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "188. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "189. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "190. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "191. 'Genome-wide association study of cardiometabolic multimorbidity in the UK Biobank.'\n",
       "192. 'A multivariate genome-wide association study of psycho-cardiometabolic multimorbidity.'\n",
       "193. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "194. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "195. 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "196. 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "197. 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "198. 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "199. 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "200. 'Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.'\n",
       "201. 'The power of TOPMed imputation for the discovery of Latino-enriched rare variants associated with type 2 diabetes.'\n",
       "202. 'Genotyping and population characteristics of the China Kadoorie Biobank.'\n",
       "203. 'Shared genetic architecture between neuroticism, coronary artery disease and cardiovascular risk factors.'\n",
       "204. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "205. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "206. 'Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.'\n",
       "207. 'Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.'\n",
       "208. 'CNest: A novel copy number association discovery method uncovers 862 new associations from 200,629 whole-exome sequence datasets in the UK Biobank.'\n",
       "209. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "210. 'Body mass index and type 2 diabetes and breast cancer survival: a Mendelian randomization study.'\n",
       "211. 'Genetic evidence for a causal relationship between type 2 diabetes and peripheral artery disease in both Europeans and East Asians.'\n",
       "212. 'Phenome-wide analysis of Taiwan Biobank reveals novel glycemia-related loci and genetic risks for diabetes.'\n",
       "213. 'Evidence that the pituitary gland connects type 2 diabetes mellitus and schizophrenia based on large-scale trans-ethnic genetic analyses.'\n",
       "214. 'Integrating polygenic risk scores in the prediction of type 2 diabetes risk and subtypes in British Pakistanis and Bangladeshis: A population-based cohort study.'\n",
       "215. 'A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.'\n",
       "216. 'A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.'\n",
       "217. 'Identifying shared genetic loci between coronavirus disease 2019 and cardiovascular diseases based on cross-trait meta-analysis.'\n",
       "218. 'Identification of shared genetic architecture between non-alcoholic fatty liver disease and type 2 diabetes: A genome-wide analysis.'\n",
       "219. 'Susceptibility Loci for Type 2 Diabetes in the Ethnically Endogamous Indian Sindhi Population: A Pooled Blood Genome-Wide Association Study.'\n",
       "220. 'Association of Polygenic Variants with Type 2 Diabetes Risk and Their Interaction with Lifestyles in Asians.'\n",
       "221. 'Common and rare variants associated with cardiometabolic traits across 98,622 whole-genome sequences in the All of Us research program.'\n",
       "222. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "223. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "224. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "225. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "226. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "227. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "228. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "229. 'Exome sequencing and analysis of 454,787 UK Biobank participants.'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  [1] \"A 100K genome-wide association scan for diabetes and related traits in the Framingham Heart Study: replication and integration with other genome-wide datasets.\"                                                             \n",
       "  [2] \"A search for variants associated with young-onset type 2 diabetes in American Indians in a 100K genotyping array.\"                                                                                                           \n",
       "  [3] \"Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies.\"                                                                                                                       \n",
       "  [4] \"Identification of novel candidate genes for type 2 diabetes from a genome-wide association scan in the Old Order Amish: evidence for replication from diabetes-related quantitative traits and from independent populations.\"\n",
       "  [5] \"Type 2 diabetes whole-genome association study in four populations: the DiaGen consortium.\"                                                                                                                                  \n",
       "  [6] \"Adiposity-related heterogeneity in patterns of type 2 diabetes susceptibility observed in genome-wide association data.\"                                                                                                     \n",
       "  [7] \"A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants.\"                                                                                                                       \n",
       "  [8] \"Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes.\"                                                                                                                         \n",
       "  [9] \"A genome-wide association study identifies novel risk loci for type 2 diabetes.\"                                                                                                                                             \n",
       " [10] \"Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes.\"                                                                                    \n",
       " [11] \"Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus.\"                                                                                                                                           \n",
       " [12] \"SNPs in KCNQ1 are associated with susceptibility to type 2 diabetes in East Asian and European populations.\"                                                                                                                 \n",
       " [13] \"Confirmation of multiple risk Loci and genetic impacts by a genome-wide association study of type 2 diabetes in the Japanese population.\"                                                                                    \n",
       " [14] \"Genetic variants at 2q24 are associated with susceptibility to type 2 diabetes.\"                                                                                                                                             \n",
       " [15] \"Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis.\"                                                                                                                             \n",
       " [16] \"A genome-wide association study in the Japanese population identifies susceptibility loci for type 2 diabetes at UBE2E2 and C2CD4A-C2CD4B.\"                                                                                  \n",
       " [17] \"Identification of new genetic risk variants for type 2 diabetes.\"                                                                                                                                                            \n",
       " [18] \"Transferability of type 2 diabetes implicated loci in multi-ethnic cohorts from Southeast Asia.\"                                                                                                                             \n",
       " [19] \"Genome-wide association study of type 2 diabetes in a sample from Mexico City and a meta-analysis of a Mexican-American sample from Starr County, Texas.\"                                                                    \n",
       " [20] \"Genome-wide association and meta-analysis in populations from Starr County, Texas, and Mexico City identify type 2 diabetes susceptibility loci and enrichment for expression quantitative trait loci in top signals.\"       \n",
       " [21] \"Genome-wide association study for type 2 diabetes in Indians identifies a new susceptibility locus at 2q21.\"                                                                                                                 \n",
       " [22] \"A single-nucleotide polymorphism in ANK1 is associated with susceptibility to type 2 diabetes in Japanese populations.\"                                                                                                      \n",
       " [23] \"Genome-wide association study identifies a novel locus contributing to type 2 diabetes susceptibility in Sikhs of Punjabi origin from India.\"                                                                                \n",
       " [24] \"A genome-wide association study confirms previously reported loci for type 2 diabetes in Han Chinese.\"                                                                                                                       \n",
       " [25] \"Exploring genome-wide - dietary heme iron intake interactions and the risk of type 2 diabetes.\"                                                                                                                              \n",
       " [26] \"A genome-wide association search for type 2 diabetes genes in African Americans.\"                                                                                                                                            \n",
       " [27] \"Stratifying type 2 diabetes cases by BMI identifies genetic risk variants in LAMA1 and enrichment for risk variants in lean compared to obese cases.\"                                                                        \n",
       " [28] \"Genome-wide association study in individuals of South Asian ancestry identifies six new type 2 diabetes susceptibility loci.\"                                                                                                \n",
       " [29] \"1000 Genomes-based imputation identifies novel and refined associations for the Wellcome Trust Case Control Consortium phase 1 Data.\"                                                                                        \n",
       " [30] \"A genome-wide search for type 2 diabetes susceptibility genes in an extended Arab family.\"                                                                                                                                   \n",
       " [31] \"Genome-wide association study in a Chinese population identifies a susceptibility locus for type 2 diabetes at 7q32 near PAX4.\"                                                                                              \n",
       " [32] \"Genome-wide association study identifies three novel loci for type 2 diabetes.\"                                                                                                                                              \n",
       " [33] \"Meta-analysis of genome-wide association studies identifies eight new loci for type 2 diabetes in east Asians.\"                                                                                                              \n",
       " [34] \"New susceptibility loci in MYL2, C12orf51 and OAS1 associated with 1-h plasma glucose as predisposing risk factors for type 2 diabetes in the Korean population.\"                                                            \n",
       " [35] \"Meta-analysis of genome-wide association studies in African Americans provides insights into the genetic architecture of type 2 diabetes.\"                                                                                   \n",
       " [36] \"A genome-wide association study in American Indians implicates DNER as a susceptibility locus for type 2 diabetes.\"                                                                                                          \n",
       " [37] \"A variant in CDKAL1 influences insulin response and risk of type 2 diabetes.\"                                                                                                                                                \n",
       " [38] \"Familial young-onset diabetes, pre-diabetes and cardiovascular disease are associated with genetic variants of DACH1 in Chinese.\"                                                                                            \n",
       " [39] \"Genetic modifiers of cystic fibrosis-related diabetes.\"                                                                                                                                                                      \n",
       " [40] \"Sequence variants in SLC16A11 are a common risk factor for type 2 diabetes in Mexico.\"                                                                                                                                       \n",
       " [41] \"A genome-wide association study identifies GRK5 and RASGRP1 as type 2 diabetes loci in Chinese Hans.\"                                                                                                                        \n",
       " [42] \"Genome-wide trans-ancestry meta-analysis provides insight into the genetic architecture of type 2 diabetes susceptibility.\"                                                                                                  \n",
       " [43] \"Genome-wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project.\"                                                                                            \n",
       " [44] \"T2DM GWAS in the Lebanese population confirms the role of TCF7L2 and CDKAL1 in disease susceptibility.\"                                                                                                                      \n",
       " [45] \"First genome-wide association study in an Australian aboriginal population provides insights into genetic risk factors for body mass index and type 2 diabetes.\"                                                             \n",
       " [46] \"Genome-wide association studies in the Japanese population identify seven novel loci for type 2 diabetes.\"                                                                                                                   \n",
       " [47] \"Single nucleotide polymorphisms in type 2 diabetes among Hispanic adults.\"                                                                                                                                                   \n",
       " [48] \"Genetic overlap between multiple sclerosis and several cardiovascular disease risk factors.\"                                                                                                                                 \n",
       " [49] \"Genetic analysis of advanced glycation end products in the DHS MIND study.\"                                                                                                                                                  \n",
       " [50] \"Genetics of Type 2 Diabetes in U.S. Hispanic/Latino Individuals: Results From the Hispanic Community Health Study/Study of Latinos (HCHS/SOL).\"                                                                              \n",
       " [51] \"Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.\"                                                                                                                            \n",
       " [52] \"Linking Alzheimer's disease and type 2 diabetes: Novel shared susceptibility genes detected by cFDR approach.\"                                                                                                               \n",
       " [53] \"Identification of new susceptibility loci for type 2 diabetes and shared etiological pathways with coronary heart disease.\"                                                                                                  \n",
       " [54] \"Multi-ethnic genome-wide association study identifies novel locus for type 2 diabetes susceptibility.\"                                                                                                                       \n",
       " [55] \"PROX1 gene CC genotype as a major determinant of early onset of type 2 diabetes in slavic study participants from Action in Diabetes and Vascular Disease: Preterax and Diamicron MR Controlled Evaluation study.\"           \n",
       " [56] \"Interaction of iron status with single nucleotide polymorphisms on incidence of type 2 diabetes.\"                                                                                                                            \n",
       " [57] \"An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.\"                                                                                                                                                  \n",
       " [58] \"An Expanded Genome-Wide Association Study of Type 2 Diabetes in Europeans.\"                                                                                                                                                  \n",
       " [59] \"Genome-wide association study identifies novel type II diabetes risk loci in Jordan subpopulations.\"                                                                                                                         \n",
       " [60] \"Large-scale association analysis provides insights into the genetic architecture and pathophysiology of type 2 diabetes.\"                                                                                                    \n",
       " [61] \"Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.\"                                                                                                         \n",
       " [62] \"Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.\"                                                                           \n",
       " [63] \"Genome-wide interaction with the insulin secretion locus MTNR1B reveals CMIP as a novel type 2 diabetes susceptibility gene in African Americans.\"                                                                           \n",
       " [64] \"First Genome-Wide Association Study of Latent Autoimmune Diabetes in Adults Reveals Novel Insights Linking Immune and Metabolic Diabetes.\"                                                                                   \n",
       " [65] \"Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.\"                                                                                                                                \n",
       " [66] \"Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.\"                                                                                                                                \n",
       " [67] \"Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.\"                                                                                                                                \n",
       " [68] \"Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.\"                                                                                                                                \n",
       " [69] \"Evidence for genetic contribution to the increased risk of type 2 diabetes in schizophrenia.\"                                                                                                                                \n",
       " [70] \"Dissecting the genetic relationship between cardiovascular risk factors and Alzheimer's disease.\"                                                                                                                            \n",
       " [71] \"Common and rare exonic MUC5B variants associated with type 2 diabetes in Han Chinese.\"                                                                                                                                       \n",
       " [72] \"Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.\"                                                                                           \n",
       " [73] \"Genome-wide association study identifies novel loci for type 2 diabetes-attributed end-stage kidney disease in African Americans.\"                                                                                           \n",
       " [74] \"Genetic analyses of diverse populations improves discovery for complex traits.\"                                                                                                                                              \n",
       " [75] \"Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.\"                                                                                                                    \n",
       " [76] \"Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes.\"                                                                                                         \n",
       " [77] \"Multivariate analysis of genome-wide data to identify potential pleiotropic genes for type 2 diabetes, obesity and coronary artery disease using MetaCCA.\"                                                                   \n",
       " [78] \"Identification of novel variants associated with osteoporosis, type 2 diabetes and potentially pleiotropic loci using pleiotropic cFDR method.\"                                                                              \n",
       " [79] \"Genetic overlap between birthweight and adult cardiometabolic diseases has implications for genomic medicine.\"                                                                                                               \n",
       " [80] \"ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.\"                                                                                                                    \n",
       " [81] \"ZRANB3 is an African-specific type 2 diabetes locus associated with beta-cell mass and insulin response.\"                                                                                                                    \n",
       " [82] \"Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.\"                                                                                                                    \n",
       " [83] \"Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.\"                                                                                                                    \n",
       " [84] \"Re-analysis of public genetic data reveals a rare X-chromosomal variant associated with type 2 diabetes.\"                                                                                                                    \n",
       " [85] \"Identification and functional analysis of glycemic trait loci in the China Health and Nutrition Survey.\"                                                                                                                     \n",
       " [86] \"Pilot genome-wide association study identifying novel risk loci for type 2 diabetes in a Maya population.\"                                                                                                                   \n",
       " [87] \"Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.\"                                                                                                            \n",
       " [88] \"Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.\"                                                                                                            \n",
       " [89] \"Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.\"                                                                                                            \n",
       " [90] \"Genome-wide association study of type 2 diabetes in Africa.\"                                                                                                                                                                 \n",
       " [91] \"A Powerful Procedure for Pathway-Based Meta-analysis Using Summary Statistics Identifies 43 Pathways Associated with Type II Diabetes in European Populations.\"                                                              \n",
       " [92] \"A Loss-of-Function Splice Acceptor Variant in IGF2 Is Protective for Type 2 Diabetes.\"                                                                                                                                       \n",
       " [93] \"Additional common variants associated with type 2 diabetes and coronary artery disease detected using a pleiotropic cFDR method.\"                                                                                            \n",
       " [94] \"Variants in the FTO and CDKAL1 loci have recessive effects on risk of obesity and type 2 diabetes, respectively.\"                                                                                                            \n",
       " [95] \"Leveraging Polygenic Functional Enrichment to Improve GWAS Power.\"                                                                                                                                                           \n",
       " [96] \"Identification of 28 new susceptibility loci for type 2 diabetes in the Japanese population.\"                                                                                                                                \n",
       " [97] \"Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes.\"                                                                                                            \n",
       " [98] \"Structural equation modeling for hypertension and type 2 diabetes based on multiple SNPs and multiple phenotypes.\"                                                                                                           \n",
       " [99] \"Exome sequencing of 20,791 cases of type 2 diabetes and 24,440 controls.\"                                                                                                                                                    \n",
       "[100] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[101] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[102] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[103] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[104] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[105] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[106] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[107] \"Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.\"                                                                                                                     \n",
       "[108] \"Shared Genetic and Experimental Links between Obesity-Related Traits and Asthma Subtypes in UK Biobank.\"                                                                                                                     \n",
       "[109] \"Identification of novel high-impact recessively inherited type 2 diabetes risk variants in the Greenlandic population.\"                                                                                                      \n",
       "[110] \"Meta-analysis of Immunochip data of four autoimmune diseases reveals novel single-disease and cross-phenotype associations.\"                                                                                                 \n",
       "[111] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[112] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[113] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[114] \"Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.\"                                                                                             \n",
       "[115] \"Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps.\"                                                                                             \n",
       "[116] \"A Fast and Accurate Method for Genome-Wide Time-to-Event Data Analysis and Its Application to UK Biobank.\"                                                                                                                   \n",
       "[117] \"Evidence that genes involved in hedgehog signaling are associated with both bipolar disorder and high BMI.\"                                                                                                                  \n",
       "[118] \"Shared genetic architecture and casual relationship between leptin levels and type 2 diabetes: large-scale cross-trait meta-analysis and Mendelian randomization analysis.\"                                                  \n",
       "[119] \"Identification of type 2 diabetes loci in 433,540 East Asian individuals.\"                                                                                                                                                   \n",
       "[120] \"Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis.\"                                                                          \n",
       "[121] \"Exome Chip Analysis of 14,026 Koreans Reve-als Known and Newly Discovered Genetic Loci Associated with Type 2 Diabetes Mellitus.\"                                                                                            \n",
       "[122] \"Investigation of gene-gene interactions in cardiac traits and serum fatty acid levels in the LURIC Health Study.\"                                                                                                            \n",
       "[123] \"Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.\"                                                                                          \n",
       "[124] \"Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.\"                                                                                          \n",
       "[125] \"Large-scale genome-wide association study in a Japanese population identifies novel susceptibility loci across different diseases.\"                                                                                          \n",
       "[126] \"Multi-trait association studies discover pleiotropic loci between Alzheimer's disease and cardiometabolic traits.\"                                                                                                           \n",
       "[127] \"Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.\"                                                                                                                                \n",
       "[128] \"Genetic Predisposition to Coronary Artery Disease in Type 2 Diabetes Mellitus.\"                                                                                                                                              \n",
       "[129] \"Analysis of overlapping genetic association in type 1 and type 2 diabetes.\"                                                                                                                                                  \n",
       "[130] \"Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.\"                                                                                                                                \n",
       "[131] \"Genome-wide discovery of genetic loci that uncouple excess adiposity from its comorbidities.\"                                                                                                                                \n",
       "[132] \"A powerful method for pleiotropic analysis under composite null hypothesis identifies novel shared loci between Type 2 Diabetes and Prostate Cancer.\"                                                                        \n",
       "[133] \"Mixed-model association for biobank-scale datasets.\"                                                                                                                                                                         \n",
       "[134] \"Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.\"                                                                                                      \n",
       "[135] \"Causal relationship and shared genetic loci between psoriasis and type 2 diabetes through trans-disease meta-analysis.\"                                                                                                      \n",
       "[136] \"The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.\"                                                                                      \n",
       "[137] \"The First Genome-Wide Association Study for Type 2 Diabetes in Youth: The Progress in Diabetes Genetics in Youth (ProDiGY) Consortium.\"                                                                                      \n",
       "[138] \"Common genetic associations between age-related diseases.\"                                                                                                                                                                   \n",
       "[139] \"Genetic discovery and risk characterization in type 2 diabetes across diverse populations.\"                                                                                                                                  \n",
       "[140] \"Genetic discovery and risk characterization in type 2 diabetes across diverse populations.\"                                                                                                                                  \n",
       "[141] \"Genetic discovery and risk characterization in type 2 diabetes across diverse populations.\"                                                                                                                                  \n",
       "[142] \"Genetic discovery and risk characterization in type 2 diabetes across diverse populations.\"                                                                                                                                  \n",
       "[143] \"Genetic discovery and risk characterization in type 2 diabetes across diverse populations.\"                                                                                                                                  \n",
       "[144] \"Genetic discovery and risk characterization in type 2 diabetes across diverse populations.\"                                                                                                                                  \n",
       "[145] \"Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.\"                                                                                                                             \n",
       "[146] \"Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.\"                                                                                                                             \n",
       "[147] \"Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.\"                                                                                                                             \n",
       "[148] \"Distinct and shared genetic architectures of gestational diabetes mellitus and type 2 diabetes.\"                                                                                                                             \n",
       "[149] \"The impact of non-additive genetic associations on age-related complex diseases.\"                                                                                                                                            \n",
       "[150] \"The impact of non-additive genetic associations on age-related complex diseases.\"                                                                                                                                            \n",
       "[151] \"The impact of non-additive genetic associations on age-related complex diseases.\"                                                                                                                                            \n",
       "[152] \"The impact of non-additive genetic associations on age-related complex diseases.\"                                                                                                                                            \n",
       "[153] \"The impact of non-additive genetic associations on age-related complex diseases.\"                                                                                                                                            \n",
       "[154] \"Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.\"                                                                                                           \n",
       "[155] \"Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.\"                                                                                                           \n",
       "[156] \"Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.\"                                                                                                           \n",
       "[157] \"Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.\"                                                                                                           \n",
       "[158] \"Boosting the power of genome-wide association studies within and across ancestries by using polygenic scores.\"                                                                                                               \n",
       "[159] \"Recessive Genome-wide Meta-analysis Illuminates Genetic Architecture of Type 2 Diabetes.\"                                                                                                                                    \n",
       "[160] \"Computationally efficient whole-genome regression for quantitative and binary traits.\"                                                                                                                                       \n",
       "[161] \"Computationally efficient whole-genome regression for quantitative and binary traits.\"                                                                                                                                       \n",
       "[162] \"Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.\"                                                                                                                       \n",
       "[163] \"Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.\"                                                                                                                       \n",
       "[164] \"Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.\"                                                                                                                       \n",
       "[165] \"Identification of genetic effects underlying type 2 diabetes in South Asian and European populations.\"                                                                                                                       \n",
       "[166] \"Genome-wide association analysis of type 2 diabetes in the EPIC-InterAct study.\"                                                                                                                                             \n",
       "[167] \"Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.\"                                                                                                           \n",
       "[168] \"Genome-wide association analyses highlight etiological differences underlying newly defined subtypes of diabetes.\"                                                                                                           \n",
       "[169] \"A generalized linear mixed model association tool for biobank-scale data.\"                                                                                                                                                   \n",
       "[170] \"A generalized linear mixed model association tool for biobank-scale data.\"                                                                                                                                                   \n",
       "[171] \"A generalized linear mixed model association tool for biobank-scale data.\"                                                                                                                                                   \n",
       "[172] \"A cross-population atlas of genetic associations for 220 human phenotypes.\"                                                                                                                                                  \n",
       "[173] \"A cross-population atlas of genetic associations for 220 human phenotypes.\"                                                                                                                                                  \n",
       "[174] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[175] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[176] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[177] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[178] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[179] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[180] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[181] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[182] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[183] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[184] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[185] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[186] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[187] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[188] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[189] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[190] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[191] \"Genome-wide association study of cardiometabolic multimorbidity in the UK Biobank.\"                                                                                                                                          \n",
       "[192] \"A multivariate genome-wide association study of psycho-cardiometabolic multimorbidity.\"                                                                                                                                      \n",
       "[193] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[194] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[195] \"Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.\"                                                                                                  \n",
       "[196] \"Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.\"                                                                                                  \n",
       "[197] \"Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.\"                                                                                                  \n",
       "[198] \"Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.\"                                                                                                  \n",
       "[199] \"Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.\"                                                                                                  \n",
       "[200] \"Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation.\"                                                                                                  \n",
       "[201] \"The power of TOPMed imputation for the discovery of Latino-enriched rare variants associated with type 2 diabetes.\"                                                                                                          \n",
       "[202] \"Genotyping and population characteristics of the China Kadoorie Biobank.\"                                                                                                                                                    \n",
       "[203] \"Shared genetic architecture between neuroticism, coronary artery disease and cardiovascular risk factors.\"                                                                                                                   \n",
       "[204] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[205] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[206] \"Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.\"                                                                                                                   \n",
       "[207] \"Genetic Overlap Analysis Identifies a Shared Etiology between Migraine and Headache with Type 2 Diabetes.\"                                                                                                                   \n",
       "[208] \"CNest: A novel copy number association discovery method uncovers 862 new associations from 200,629 whole-exome sequence datasets in the UK Biobank.\"                                                                         \n",
       "[209] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[210] \"Body mass index and type 2 diabetes and breast cancer survival: a Mendelian randomization study.\"                                                                                                                            \n",
       "[211] \"Genetic evidence for a causal relationship between type 2 diabetes and peripheral artery disease in both Europeans and East Asians.\"                                                                                         \n",
       "[212] \"Phenome-wide analysis of Taiwan Biobank reveals novel glycemia-related loci and genetic risks for diabetes.\"                                                                                                                 \n",
       "[213] \"Evidence that the pituitary gland connects type 2 diabetes mellitus and schizophrenia based on large-scale trans-ethnic genetic analyses.\"                                                                                   \n",
       "[214] \"Integrating polygenic risk scores in the prediction of type 2 diabetes risk and subtypes in British Pakistanis and Bangladeshis: A population-based cohort study.\"                                                           \n",
       "[215] \"A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.\"                                                                  \n",
       "[216] \"A genome-wide cross-trait analysis identifies shared loci and causal relationships of type 2 diabetes and glycaemic traits with polycystic ovary syndrome.\"                                                                  \n",
       "[217] \"Identifying shared genetic loci between coronavirus disease 2019 and cardiovascular diseases based on cross-trait meta-analysis.\"                                                                                            \n",
       "[218] \"Identification of shared genetic architecture between non-alcoholic fatty liver disease and type 2 diabetes: A genome-wide analysis.\"                                                                                        \n",
       "[219] \"Susceptibility Loci for Type 2 Diabetes in the Ethnically Endogamous Indian Sindhi Population: A Pooled Blood Genome-Wide Association Study.\"                                                                                \n",
       "[220] \"Association of Polygenic Variants with Type 2 Diabetes Risk and Their Interaction with Lifestyles in Asians.\"                                                                                                                \n",
       "[221] \"Common and rare variants associated with cardiometabolic traits across 98,622 whole-genome sequences in the All of Us research program.\"                                                                                     \n",
       "[222] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[223] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[224] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[225] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[226] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[227] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[228] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           \n",
       "[229] \"Exome sequencing and analysis of 454,787 UK Biobank participants.\"                                                                                                                                                           "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "my_studies@publications$title"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "d0d1cfaa-c6d2-46ef-acc9-7d38fed6d192",
   "metadata": {},
   "outputs": [],
   "source": [
    "my_associations <- get_associations(study_id = my_studies@studies$study_id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "1333433c-2c55-43de-bda7-127b66d95a5a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "5470"
      ],
      "text/latex": [
       "5470"
      ],
      "text/markdown": [
       "5470"
      ],
      "text/plain": [
       "[1] 5470"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "gwasrapidd::n(my_associations)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "bffdca69-4175-4183-8614-b88b209bb2d9",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get association ids for which pvalue is less than 1e-6.\n",
    "dplyr::filter(my_associations@associations, pvalue < 1e-6) %>% # Filter by p-value\n",
    "  tidyr::drop_na(pvalue) %>%\n",
    "  dplyr::pull(association_id) -> association_ids # Extract column association_id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "199d5397-560f-4295-bd06-43f822c59804",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "4903"
      ],
      "text/latex": [
       "4903"
      ],
      "text/markdown": [
       "4903"
      ],
      "text/plain": [
       "[1] 4903"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Extract associations by association id\n",
    "my_associations2 <- my_associations[association_ids]\n",
    "gwasrapidd::n(my_associations2)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "c5b01d88-a683-4ead-99fd-fb0b56c94adc",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "An object of class \"associations\"\n",
       "Slot \"associations\":\n",
       "\u001b[90m# A tibble: 4,903 × 18\u001b[39m\n",
       "   association_id  pvalue pvalue_description pvalue_mantissa pvalue_exponent\n",
       "   \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m            \u001b[3m\u001b[90m<dbl>\u001b[39m\u001b[23m \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m                        \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m           \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m\n",
       "\u001b[90m 1\u001b[39m 11380          5  \u001b[90me\u001b[39m\u001b[31m- 8\u001b[39m \u001b[31mNA\u001b[39m                               5              -\u001b[31m8\u001b[39m\n",
       "\u001b[90m 2\u001b[39m 12442          1  \u001b[90me\u001b[39m\u001b[31m- 9\u001b[39m (non-obese)                      1              -\u001b[31m9\u001b[39m\n",
       "\u001b[90m 3\u001b[39m 12443          7  \u001b[90me\u001b[39m\u001b[31m- 7\u001b[39m (non-obese)                      7              -\u001b[31m7\u001b[39m\n",
       "\u001b[90m 4\u001b[39m 12174          2  \u001b[90me\u001b[39m\u001b[31m-17\u001b[39m (obese)                          2             -\u001b[31m17\u001b[39m\n",
       "\u001b[90m 5\u001b[39m 12177          5  \u001b[90me\u001b[39m\u001b[31m- 7\u001b[39m (obese)                          5              -\u001b[31m7\u001b[39m\n",
       "\u001b[90m 6\u001b[39m 12444          7  \u001b[90me\u001b[39m\u001b[31m- 7\u001b[39m (non-obese)                      7              -\u001b[31m7\u001b[39m\n",
       "\u001b[90m 7\u001b[39m 12452          6  \u001b[90me\u001b[39m\u001b[31m-16\u001b[39m (obese)                          6             -\u001b[31m16\u001b[39m\n",
       "\u001b[90m 8\u001b[39m 12453          9  \u001b[90me\u001b[39m\u001b[31m-30\u001b[39m (non-obese)                      9             -\u001b[31m30\u001b[39m\n",
       "\u001b[90m 9\u001b[39m 13346          7  \u001b[90me\u001b[39m\u001b[31m-11\u001b[39m (DGI+FUSION+WTCCC)               7             -\u001b[31m11\u001b[39m\n",
       "\u001b[90m10\u001b[39m 13286          6  \u001b[90me\u001b[39m\u001b[31m- 8\u001b[39m \u001b[31mNA\u001b[39m                               6              -\u001b[31m8\u001b[39m\n",
       "\u001b[90m# ℹ 4,893 more rows\u001b[39m\n",
       "\u001b[90m# ℹ 13 more variables: multiple_snp_haplotype <lgl>, snp_interaction <lgl>,\u001b[39m\n",
       "\u001b[90m#   snp_type <chr>, risk_frequency <dbl>, standard_error <dbl>, range <chr>,\u001b[39m\n",
       "\u001b[90m#   or_per_copy_number <dbl>, beta_number <dbl>, beta_unit <chr>,\u001b[39m\n",
       "\u001b[90m#   beta_direction <chr>, beta_description <chr>, last_mapping_date <dttm>,\u001b[39m\n",
       "\u001b[90m#   last_update_date <dttm>\u001b[39m\n",
       "\n",
       "Slot \"loci\":\n",
       "\u001b[90m# A tibble: 4,911 × 4\u001b[39m\n",
       "   association_id locus_id haplotype_snp_count description   \n",
       "   \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m             \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m               \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m         \n",
       "\u001b[90m 1\u001b[39m 11380                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 2\u001b[39m 12442                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 3\u001b[39m 12443                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 4\u001b[39m 12174                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 5\u001b[39m 12177                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 6\u001b[39m 12444                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 7\u001b[39m 12452                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 8\u001b[39m 12453                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m 9\u001b[39m 13346                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m10\u001b[39m 13286                 1                  \u001b[31mNA\u001b[39m Single variant\n",
       "\u001b[90m# ℹ 4,901 more rows\u001b[39m\n",
       "\n",
       "Slot \"risk_alleles\":\n",
       "\u001b[90m# A tibble: 4,917 × 7\u001b[39m\n",
       "   association_id locus_id variant_id risk_allele risk_frequency genome_wide\n",
       "   \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m             \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m      \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m                \u001b[3m\u001b[90m<dbl>\u001b[39m\u001b[23m \u001b[3m\u001b[90m<lgl>\u001b[39m\u001b[23m      \n",
       "\u001b[90m 1\u001b[39m 11380                 1 rs7903146  A                       \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m 2\u001b[39m 12442                 1 rs5219     \u001b[31mNA\u001b[39m                      \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m 3\u001b[39m 12443                 1 rs10811661 \u001b[31mNA\u001b[39m                      \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m 4\u001b[39m 12174                 1 rs8050136  \u001b[31mNA\u001b[39m                      \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m 5\u001b[39m 12177                 1 rs5219     \u001b[31mNA\u001b[39m                      \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m 6\u001b[39m 12444                 1 rs10946398 \u001b[31mNA\u001b[39m                      \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m 7\u001b[39m 12452                 1 rs7903146  \u001b[31mNA\u001b[39m                      \u001b[31mNA\u001b[39m FALSE      \n",
       "\u001b[90m 8\u001b[39m 12453                 1 rs7903146  \u001b[31mNA\u001b[39m                      \u001b[31mNA\u001b[39m FALSE      \n",
       "\u001b[90m 9\u001b[39m 13346                 1 rs5219     T                       \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m10\u001b[39m 13286                 1 rs9300039  C                       \u001b[31mNA\u001b[39m \u001b[31mNA\u001b[39m         \n",
       "\u001b[90m# ℹ 4,907 more rows\u001b[39m\n",
       "\u001b[90m# ℹ 1 more variable: limited_list <lgl>\u001b[39m\n",
       "\n",
       "Slot \"genes\":\n",
       "\u001b[90m# A tibble: 3,348 × 3\u001b[39m\n",
       "   association_id locus_id gene_name \n",
       "   \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m             \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m     \n",
       "\u001b[90m 1\u001b[39m 11380                 1 TCF7L2    \n",
       "\u001b[90m 2\u001b[39m 12442                 1 KCNJ11    \n",
       "\u001b[90m 3\u001b[39m 12443                 1 CDKN2B    \n",
       "\u001b[90m 4\u001b[39m 12174                 1 FTO       \n",
       "\u001b[90m 5\u001b[39m 12177                 1 KCNJ11    \n",
       "\u001b[90m 6\u001b[39m 12444                 1 CDKAL     \n",
       "\u001b[90m 7\u001b[39m 12452                 1 TCF7L2    \n",
       "\u001b[90m 8\u001b[39m 12453                 1 TCF7L2    \n",
       "\u001b[90m 9\u001b[39m 13346                 1 KCNJ11    \n",
       "\u001b[90m10\u001b[39m 13286                 1 intergenic\n",
       "\u001b[90m# ℹ 3,338 more rows\u001b[39m\n",
       "\n",
       "Slot \"ensembl_ids\":\n",
       "\u001b[90m# A tibble: 3,350 × 4\u001b[39m\n",
       "   association_id locus_id gene_name  ensembl_id     \n",
       "   \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m             \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m      \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m          \n",
       "\u001b[90m 1\u001b[39m 11380                 1 TCF7L2     ENSG00000148737\n",
       "\u001b[90m 2\u001b[39m 12442                 1 KCNJ11     ENSG00000187486\n",
       "\u001b[90m 3\u001b[39m 12443                 1 CDKN2B     ENSG00000147883\n",
       "\u001b[90m 4\u001b[39m 12174                 1 FTO        ENSG00000140718\n",
       "\u001b[90m 5\u001b[39m 12177                 1 KCNJ11     ENSG00000187486\n",
       "\u001b[90m 6\u001b[39m 12444                 1 CDKAL      \u001b[31mNA\u001b[39m             \n",
       "\u001b[90m 7\u001b[39m 12452                 1 TCF7L2     ENSG00000148737\n",
       "\u001b[90m 8\u001b[39m 12453                 1 TCF7L2     ENSG00000148737\n",
       "\u001b[90m 9\u001b[39m 13346                 1 KCNJ11     ENSG00000187486\n",
       "\u001b[90m10\u001b[39m 13286                 1 intergenic \u001b[31mNA\u001b[39m             \n",
       "\u001b[90m# ℹ 3,340 more rows\u001b[39m\n",
       "\n",
       "Slot \"entrez_ids\":\n",
       "\u001b[90m# A tibble: 3,348 × 4\u001b[39m\n",
       "   association_id locus_id gene_name  entrez_id\n",
       "   \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m             \u001b[3m\u001b[90m<int>\u001b[39m\u001b[23m \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m      \u001b[3m\u001b[90m<chr>\u001b[39m\u001b[23m    \n",
       "\u001b[90m 1\u001b[39m 11380                 1 TCF7L2     6934     \n",
       "\u001b[90m 2\u001b[39m 12442                 1 KCNJ11     3767     \n",
       "\u001b[90m 3\u001b[39m 12443                 1 CDKN2B     1030     \n",
       "\u001b[90m 4\u001b[39m 12174                 1 FTO        79068    \n",
       "\u001b[90m 5\u001b[39m 12177                 1 KCNJ11     3767     \n",
       "\u001b[90m 6\u001b[39m 12444                 1 CDKAL      \u001b[31mNA\u001b[39m       \n",
       "\u001b[90m 7\u001b[39m 12452                 1 TCF7L2     6934     \n",
       "\u001b[90m 8\u001b[39m 12453                 1 TCF7L2     6934     \n",
       "\u001b[90m 9\u001b[39m 13346                 1 KCNJ11     3767     \n",
       "\u001b[90m10\u001b[39m 13286                 1 intergenic \u001b[31mNA\u001b[39m       \n",
       "\u001b[90m# ℹ 3,338 more rows\u001b[39m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "my_associations2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "b3b221a9-5128-4132-b2b3-a4e982a768af",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A tibble: 4917 × 7</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>association_id</th><th scope=col>locus_id</th><th scope=col>variant_id</th><th scope=col>risk_allele</th><th scope=col>risk_frequency</th><th scope=col>genome_wide</th><th scope=col>limited_list</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;lgl&gt;</th><th scope=col>&lt;lgl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>11380</td><td>1</td><td>rs7903146 </td><td>A </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>12442</td><td>1</td><td>rs5219    </td><td>NA</td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>12443</td><td>1</td><td>rs10811661</td><td>NA</td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>12174</td><td>1</td><td>rs8050136 </td><td>NA</td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>12177</td><td>1</td><td>rs5219    </td><td>NA</td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>12444</td><td>1</td><td>rs10946398</td><td>NA</td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>12452</td><td>1</td><td>rs7903146 </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>12453</td><td>1</td><td>rs7903146 </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>13346</td><td>1</td><td>rs5219    </td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13286</td><td>1</td><td>rs9300039 </td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13339</td><td>1</td><td>rs4402960 </td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13342</td><td>1</td><td>rs8050136 </td><td>A </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13345</td><td>1</td><td>rs1111875 </td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13341</td><td>1</td><td>rs10811661</td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13340</td><td>1</td><td>rs7754840 </td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13287</td><td>1</td><td>rs7903146 </td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13344</td><td>1</td><td>rs13266634</td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13294</td><td>1</td><td>rs8050136 </td><td>A </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13347</td><td>1</td><td>rs4402960 </td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13348</td><td>1</td><td>rs13266634</td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13295</td><td>1</td><td>rs10946398</td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>13349</td><td>1</td><td>rs7901695 </td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>11785</td><td>1</td><td>rs5215    </td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>11233</td><td>1</td><td>rs13266634</td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>11320</td><td>1</td><td>rs7903146 </td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>5873 </td><td>1</td><td>rs864745  </td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>5874 </td><td>1</td><td>rs12779790</td><td>G </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>5875 </td><td>1</td><td>rs7961581 </td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>5876 </td><td>1</td><td>rs7578597 </td><td>T </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>11593</td><td>1</td><td>rs7020996 </td><td>C </td><td>NA</td><td>   NA</td><td>   NA</td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>100372749</td><td>1</td><td>rs9905906      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372755</td><td>1</td><td>rs11650852     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372761</td><td>1</td><td>rs28476533     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372767</td><td>1</td><td>rs8080678      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372773</td><td>1</td><td>rs76734228     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372779</td><td>1</td><td>rs73379947     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372785</td><td>1</td><td>rs4799327      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372791</td><td>1</td><td>rs962876       </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372797</td><td>1</td><td>rs11877530     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372802</td><td>1</td><td>rs2967605      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372808</td><td>1</td><td>rs17657660     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372814</td><td>1</td><td>rs2061771      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372819</td><td>1</td><td>rs651862       </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372824</td><td>1</td><td>rs6036804      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372830</td><td>1</td><td>rs2024578      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372836</td><td>1</td><td>rs2834623      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372842</td><td>1</td><td>rs9789875      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372848</td><td>1</td><td>rs73219073     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372854</td><td>1</td><td>rs76866981     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372066</td><td>1</td><td>rs4661310      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100372072</td><td>1</td><td>rs4920461      </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100478397</td><td>1</td><td>rs35612982     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100478404</td><td>1</td><td>rs61160304     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100478411</td><td>1</td><td>rs10811661     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>100478415</td><td>1</td><td>rs60808706     </td><td>NA</td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>129662696</td><td>1</td><td>chr6:26276061  </td><td>G </td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>129662701</td><td>1</td><td>chr9:22136441  </td><td>C </td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>129662708</td><td>1</td><td>chr10:113039134</td><td>A </td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>129662714</td><td>1</td><td>chr12:4275678  </td><td>G </td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "\t<tr><td>129662719</td><td>1</td><td>chr16:53773852 </td><td>G </td><td>NA</td><td>FALSE</td><td>FALSE</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A tibble: 4917 × 7\n",
       "\\begin{tabular}{lllllll}\n",
       " association\\_id & locus\\_id & variant\\_id & risk\\_allele & risk\\_frequency & genome\\_wide & limited\\_list\\\\\n",
       " <chr> & <int> & <chr> & <chr> & <dbl> & <lgl> & <lgl>\\\\\n",
       "\\hline\n",
       "\t 11380 & 1 & rs7903146  & A  & NA &    NA &    NA\\\\\n",
       "\t 12442 & 1 & rs5219     & NA & NA &    NA &    NA\\\\\n",
       "\t 12443 & 1 & rs10811661 & NA & NA &    NA &    NA\\\\\n",
       "\t 12174 & 1 & rs8050136  & NA & NA &    NA &    NA\\\\\n",
       "\t 12177 & 1 & rs5219     & NA & NA &    NA &    NA\\\\\n",
       "\t 12444 & 1 & rs10946398 & NA & NA &    NA &    NA\\\\\n",
       "\t 12452 & 1 & rs7903146  & NA & NA & FALSE & FALSE\\\\\n",
       "\t 12453 & 1 & rs7903146  & NA & NA & FALSE & FALSE\\\\\n",
       "\t 13346 & 1 & rs5219     & T  & NA &    NA &    NA\\\\\n",
       "\t 13286 & 1 & rs9300039  & C  & NA &    NA &    NA\\\\\n",
       "\t 13339 & 1 & rs4402960  & T  & NA &    NA &    NA\\\\\n",
       "\t 13342 & 1 & rs8050136  & A  & NA &    NA &    NA\\\\\n",
       "\t 13345 & 1 & rs1111875  & C  & NA &    NA &    NA\\\\\n",
       "\t 13341 & 1 & rs10811661 & T  & NA &    NA &    NA\\\\\n",
       "\t 13340 & 1 & rs7754840  & C  & NA &    NA &    NA\\\\\n",
       "\t 13287 & 1 & rs7903146  & T  & NA &    NA &    NA\\\\\n",
       "\t 13344 & 1 & rs13266634 & C  & NA &    NA &    NA\\\\\n",
       "\t 13294 & 1 & rs8050136  & A  & NA &    NA &    NA\\\\\n",
       "\t 13347 & 1 & rs4402960  & T  & NA &    NA &    NA\\\\\n",
       "\t 13348 & 1 & rs13266634 & C  & NA &    NA &    NA\\\\\n",
       "\t 13295 & 1 & rs10946398 & C  & NA &    NA &    NA\\\\\n",
       "\t 13349 & 1 & rs7901695  & C  & NA &    NA &    NA\\\\\n",
       "\t 11785 & 1 & rs5215     & C  & NA &    NA &    NA\\\\\n",
       "\t 11233 & 1 & rs13266634 & C  & NA &    NA &    NA\\\\\n",
       "\t 11320 & 1 & rs7903146  & T  & NA &    NA &    NA\\\\\n",
       "\t 5873  & 1 & rs864745   & T  & NA &    NA &    NA\\\\\n",
       "\t 5874  & 1 & rs12779790 & G  & NA &    NA &    NA\\\\\n",
       "\t 5875  & 1 & rs7961581  & C  & NA &    NA &    NA\\\\\n",
       "\t 5876  & 1 & rs7578597  & T  & NA &    NA &    NA\\\\\n",
       "\t 11593 & 1 & rs7020996  & C  & NA &    NA &    NA\\\\\n",
       "\t ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\t 100372749 & 1 & rs9905906       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372755 & 1 & rs11650852      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372761 & 1 & rs28476533      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372767 & 1 & rs8080678       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372773 & 1 & rs76734228      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372779 & 1 & rs73379947      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372785 & 1 & rs4799327       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372791 & 1 & rs962876        & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372797 & 1 & rs11877530      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372802 & 1 & rs2967605       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372808 & 1 & rs17657660      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372814 & 1 & rs2061771       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372819 & 1 & rs651862        & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372824 & 1 & rs6036804       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372830 & 1 & rs2024578       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372836 & 1 & rs2834623       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372842 & 1 & rs9789875       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372848 & 1 & rs73219073      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372854 & 1 & rs76866981      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372066 & 1 & rs4661310       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100372072 & 1 & rs4920461       & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100478397 & 1 & rs35612982      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100478404 & 1 & rs61160304      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100478411 & 1 & rs10811661      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 100478415 & 1 & rs60808706      & NA & NA & FALSE & FALSE\\\\\n",
       "\t 129662696 & 1 & chr6:26276061   & G  & NA & FALSE & FALSE\\\\\n",
       "\t 129662701 & 1 & chr9:22136441   & C  & NA & FALSE & FALSE\\\\\n",
       "\t 129662708 & 1 & chr10:113039134 & A  & NA & FALSE & FALSE\\\\\n",
       "\t 129662714 & 1 & chr12:4275678   & G  & NA & FALSE & FALSE\\\\\n",
       "\t 129662719 & 1 & chr16:53773852  & G  & NA & FALSE & FALSE\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A tibble: 4917 × 7\n",
       "\n",
       "| association_id &lt;chr&gt; | locus_id &lt;int&gt; | variant_id &lt;chr&gt; | risk_allele &lt;chr&gt; | risk_frequency &lt;dbl&gt; | genome_wide &lt;lgl&gt; | limited_list &lt;lgl&gt; |\n",
       "|---|---|---|---|---|---|---|\n",
       "| 11380 | 1 | rs7903146  | A  | NA |    NA |    NA |\n",
       "| 12442 | 1 | rs5219     | NA | NA |    NA |    NA |\n",
       "| 12443 | 1 | rs10811661 | NA | NA |    NA |    NA |\n",
       "| 12174 | 1 | rs8050136  | NA | NA |    NA |    NA |\n",
       "| 12177 | 1 | rs5219     | NA | NA |    NA |    NA |\n",
       "| 12444 | 1 | rs10946398 | NA | NA |    NA |    NA |\n",
       "| 12452 | 1 | rs7903146  | NA | NA | FALSE | FALSE |\n",
       "| 12453 | 1 | rs7903146  | NA | NA | FALSE | FALSE |\n",
       "| 13346 | 1 | rs5219     | T  | NA |    NA |    NA |\n",
       "| 13286 | 1 | rs9300039  | C  | NA |    NA |    NA |\n",
       "| 13339 | 1 | rs4402960  | T  | NA |    NA |    NA |\n",
       "| 13342 | 1 | rs8050136  | A  | NA |    NA |    NA |\n",
       "| 13345 | 1 | rs1111875  | C  | NA |    NA |    NA |\n",
       "| 13341 | 1 | rs10811661 | T  | NA |    NA |    NA |\n",
       "| 13340 | 1 | rs7754840  | C  | NA |    NA |    NA |\n",
       "| 13287 | 1 | rs7903146  | T  | NA |    NA |    NA |\n",
       "| 13344 | 1 | rs13266634 | C  | NA |    NA |    NA |\n",
       "| 13294 | 1 | rs8050136  | A  | NA |    NA |    NA |\n",
       "| 13347 | 1 | rs4402960  | T  | NA |    NA |    NA |\n",
       "| 13348 | 1 | rs13266634 | C  | NA |    NA |    NA |\n",
       "| 13295 | 1 | rs10946398 | C  | NA |    NA |    NA |\n",
       "| 13349 | 1 | rs7901695  | C  | NA |    NA |    NA |\n",
       "| 11785 | 1 | rs5215     | C  | NA |    NA |    NA |\n",
       "| 11233 | 1 | rs13266634 | C  | NA |    NA |    NA |\n",
       "| 11320 | 1 | rs7903146  | T  | NA |    NA |    NA |\n",
       "| 5873  | 1 | rs864745   | T  | NA |    NA |    NA |\n",
       "| 5874  | 1 | rs12779790 | G  | NA |    NA |    NA |\n",
       "| 5875  | 1 | rs7961581  | C  | NA |    NA |    NA |\n",
       "| 5876  | 1 | rs7578597  | T  | NA |    NA |    NA |\n",
       "| 11593 | 1 | rs7020996  | C  | NA |    NA |    NA |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| 100372749 | 1 | rs9905906       | NA | NA | FALSE | FALSE |\n",
       "| 100372755 | 1 | rs11650852      | NA | NA | FALSE | FALSE |\n",
       "| 100372761 | 1 | rs28476533      | NA | NA | FALSE | FALSE |\n",
       "| 100372767 | 1 | rs8080678       | NA | NA | FALSE | FALSE |\n",
       "| 100372773 | 1 | rs76734228      | NA | NA | FALSE | FALSE |\n",
       "| 100372779 | 1 | rs73379947      | NA | NA | FALSE | FALSE |\n",
       "| 100372785 | 1 | rs4799327       | NA | NA | FALSE | FALSE |\n",
       "| 100372791 | 1 | rs962876        | NA | NA | FALSE | FALSE |\n",
       "| 100372797 | 1 | rs11877530      | NA | NA | FALSE | FALSE |\n",
       "| 100372802 | 1 | rs2967605       | NA | NA | FALSE | FALSE |\n",
       "| 100372808 | 1 | rs17657660      | NA | NA | FALSE | FALSE |\n",
       "| 100372814 | 1 | rs2061771       | NA | NA | FALSE | FALSE |\n",
       "| 100372819 | 1 | rs651862        | NA | NA | FALSE | FALSE |\n",
       "| 100372824 | 1 | rs6036804       | NA | NA | FALSE | FALSE |\n",
       "| 100372830 | 1 | rs2024578       | NA | NA | FALSE | FALSE |\n",
       "| 100372836 | 1 | rs2834623       | NA | NA | FALSE | FALSE |\n",
       "| 100372842 | 1 | rs9789875       | NA | NA | FALSE | FALSE |\n",
       "| 100372848 | 1 | rs73219073      | NA | NA | FALSE | FALSE |\n",
       "| 100372854 | 1 | rs76866981      | NA | NA | FALSE | FALSE |\n",
       "| 100372066 | 1 | rs4661310       | NA | NA | FALSE | FALSE |\n",
       "| 100372072 | 1 | rs4920461       | NA | NA | FALSE | FALSE |\n",
       "| 100478397 | 1 | rs35612982      | NA | NA | FALSE | FALSE |\n",
       "| 100478404 | 1 | rs61160304      | NA | NA | FALSE | FALSE |\n",
       "| 100478411 | 1 | rs10811661      | NA | NA | FALSE | FALSE |\n",
       "| 100478415 | 1 | rs60808706      | NA | NA | FALSE | FALSE |\n",
       "| 129662696 | 1 | chr6:26276061   | G  | NA | FALSE | FALSE |\n",
       "| 129662701 | 1 | chr9:22136441   | C  | NA | FALSE | FALSE |\n",
       "| 129662708 | 1 | chr10:113039134 | A  | NA | FALSE | FALSE |\n",
       "| 129662714 | 1 | chr12:4275678   | G  | NA | FALSE | FALSE |\n",
       "| 129662719 | 1 | chr16:53773852  | G  | NA | FALSE | FALSE |\n",
       "\n"
      ],
      "text/plain": [
       "     association_id locus_id variant_id      risk_allele risk_frequency\n",
       "1    11380          1        rs7903146       A           NA            \n",
       "2    12442          1        rs5219          NA          NA            \n",
       "3    12443          1        rs10811661      NA          NA            \n",
       "4    12174          1        rs8050136       NA          NA            \n",
       "5    12177          1        rs5219          NA          NA            \n",
       "6    12444          1        rs10946398      NA          NA            \n",
       "7    12452          1        rs7903146       NA          NA            \n",
       "8    12453          1        rs7903146       NA          NA            \n",
       "9    13346          1        rs5219          T           NA            \n",
       "10   13286          1        rs9300039       C           NA            \n",
       "11   13339          1        rs4402960       T           NA            \n",
       "12   13342          1        rs8050136       A           NA            \n",
       "13   13345          1        rs1111875       C           NA            \n",
       "14   13341          1        rs10811661      T           NA            \n",
       "15   13340          1        rs7754840       C           NA            \n",
       "16   13287          1        rs7903146       T           NA            \n",
       "17   13344          1        rs13266634      C           NA            \n",
       "18   13294          1        rs8050136       A           NA            \n",
       "19   13347          1        rs4402960       T           NA            \n",
       "20   13348          1        rs13266634      C           NA            \n",
       "21   13295          1        rs10946398      C           NA            \n",
       "22   13349          1        rs7901695       C           NA            \n",
       "23   11785          1        rs5215          C           NA            \n",
       "24   11233          1        rs13266634      C           NA            \n",
       "25   11320          1        rs7903146       T           NA            \n",
       "26   5873           1        rs864745        T           NA            \n",
       "27   5874           1        rs12779790      G           NA            \n",
       "28   5875           1        rs7961581       C           NA            \n",
       "29   5876           1        rs7578597       T           NA            \n",
       "30   11593          1        rs7020996       C           NA            \n",
       "⋮    ⋮              ⋮        ⋮               ⋮           ⋮             \n",
       "4888 100372749      1        rs9905906       NA          NA            \n",
       "4889 100372755      1        rs11650852      NA          NA            \n",
       "4890 100372761      1        rs28476533      NA          NA            \n",
       "4891 100372767      1        rs8080678       NA          NA            \n",
       "4892 100372773      1        rs76734228      NA          NA            \n",
       "4893 100372779      1        rs73379947      NA          NA            \n",
       "4894 100372785      1        rs4799327       NA          NA            \n",
       "4895 100372791      1        rs962876        NA          NA            \n",
       "4896 100372797      1        rs11877530      NA          NA            \n",
       "4897 100372802      1        rs2967605       NA          NA            \n",
       "4898 100372808      1        rs17657660      NA          NA            \n",
       "4899 100372814      1        rs2061771       NA          NA            \n",
       "4900 100372819      1        rs651862        NA          NA            \n",
       "4901 100372824      1        rs6036804       NA          NA            \n",
       "4902 100372830      1        rs2024578       NA          NA            \n",
       "4903 100372836      1        rs2834623       NA          NA            \n",
       "4904 100372842      1        rs9789875       NA          NA            \n",
       "4905 100372848      1        rs73219073      NA          NA            \n",
       "4906 100372854      1        rs76866981      NA          NA            \n",
       "4907 100372066      1        rs4661310       NA          NA            \n",
       "4908 100372072      1        rs4920461       NA          NA            \n",
       "4909 100478397      1        rs35612982      NA          NA            \n",
       "4910 100478404      1        rs61160304      NA          NA            \n",
       "4911 100478411      1        rs10811661      NA          NA            \n",
       "4912 100478415      1        rs60808706      NA          NA            \n",
       "4913 129662696      1        chr6:26276061   G           NA            \n",
       "4914 129662701      1        chr9:22136441   C           NA            \n",
       "4915 129662708      1        chr10:113039134 A           NA            \n",
       "4916 129662714      1        chr12:4275678   G           NA            \n",
       "4917 129662719      1        chr16:53773852  G           NA            \n",
       "     genome_wide limited_list\n",
       "1       NA          NA       \n",
       "2       NA          NA       \n",
       "3       NA          NA       \n",
       "4       NA          NA       \n",
       "5       NA          NA       \n",
       "6       NA          NA       \n",
       "7    FALSE       FALSE       \n",
       "8    FALSE       FALSE       \n",
       "9       NA          NA       \n",
       "10      NA          NA       \n",
       "11      NA          NA       \n",
       "12      NA          NA       \n",
       "13      NA          NA       \n",
       "14      NA          NA       \n",
       "15      NA          NA       \n",
       "16      NA          NA       \n",
       "17      NA          NA       \n",
       "18      NA          NA       \n",
       "19      NA          NA       \n",
       "20      NA          NA       \n",
       "21      NA          NA       \n",
       "22      NA          NA       \n",
       "23      NA          NA       \n",
       "24      NA          NA       \n",
       "25      NA          NA       \n",
       "26      NA          NA       \n",
       "27      NA          NA       \n",
       "28      NA          NA       \n",
       "29      NA          NA       \n",
       "30      NA          NA       \n",
       "⋮    ⋮           ⋮           \n",
       "4888 FALSE       FALSE       \n",
       "4889 FALSE       FALSE       \n",
       "4890 FALSE       FALSE       \n",
       "4891 FALSE       FALSE       \n",
       "4892 FALSE       FALSE       \n",
       "4893 FALSE       FALSE       \n",
       "4894 FALSE       FALSE       \n",
       "4895 FALSE       FALSE       \n",
       "4896 FALSE       FALSE       \n",
       "4897 FALSE       FALSE       \n",
       "4898 FALSE       FALSE       \n",
       "4899 FALSE       FALSE       \n",
       "4900 FALSE       FALSE       \n",
       "4901 FALSE       FALSE       \n",
       "4902 FALSE       FALSE       \n",
       "4903 FALSE       FALSE       \n",
       "4904 FALSE       FALSE       \n",
       "4905 FALSE       FALSE       \n",
       "4906 FALSE       FALSE       \n",
       "4907 FALSE       FALSE       \n",
       "4908 FALSE       FALSE       \n",
       "4909 FALSE       FALSE       \n",
       "4910 FALSE       FALSE       \n",
       "4911 FALSE       FALSE       \n",
       "4912 FALSE       FALSE       \n",
       "4913 FALSE       FALSE       \n",
       "4914 FALSE       FALSE       \n",
       "4915 FALSE       FALSE       \n",
       "4916 FALSE       FALSE       \n",
       "4917 FALSE       FALSE       "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "my_associations2@risk_alleles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "8fe53b7a-4669-418d-839f-2a75c68015ea",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A tibble: 4903 × 18</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>association_id</th><th scope=col>pvalue</th><th scope=col>pvalue_description</th><th scope=col>pvalue_mantissa</th><th scope=col>pvalue_exponent</th><th scope=col>multiple_snp_haplotype</th><th scope=col>snp_interaction</th><th scope=col>snp_type</th><th scope=col>risk_frequency</th><th scope=col>standard_error</th><th scope=col>range</th><th scope=col>or_per_copy_number</th><th scope=col>beta_number</th><th scope=col>beta_unit</th><th scope=col>beta_direction</th><th scope=col>beta_description</th><th scope=col>last_mapping_date</th><th scope=col>last_update_date</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;lgl&gt;</th><th scope=col>&lt;lgl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dttm&gt;</th><th scope=col>&lt;dttm&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>11380</td><td>5e-08</td><td>NA                </td><td>5</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.41-2.08] </td><td>1.71</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:39:09</td><td>2024-02-16 18:39:09</td></tr>\n",
       "\t<tr><td>12442</td><td>1e-09</td><td>(non-obese)       </td><td>1</td><td> -9</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.16-1.34] </td><td>1.25</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:07:37</td><td>2024-02-16 18:07:37</td></tr>\n",
       "\t<tr><td>12443</td><td>7e-07</td><td>(non-obese)       </td><td>7</td><td> -7</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.15-1.38] </td><td>1.26</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:07:40</td><td>2024-02-16 18:07:40</td></tr>\n",
       "\t<tr><td>12174</td><td>2e-17</td><td>(obese)           </td><td>2</td><td>-17</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.23-1.39] </td><td>1.30</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:17:22</td><td>2024-02-16 18:17:22</td></tr>\n",
       "\t<tr><td>12177</td><td>5e-07</td><td>(obese)           </td><td>5</td><td> -7</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.11-1.27] </td><td>1.19</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:34:32</td><td>2024-02-16 18:34:32</td></tr>\n",
       "\t<tr><td>12444</td><td>7e-07</td><td>(non-obese)       </td><td>7</td><td> -7</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.11-1.26] </td><td>1.18</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:51:26</td><td>2024-02-16 18:51:26</td></tr>\n",
       "\t<tr><td>12452</td><td>6e-16</td><td>(obese)           </td><td>6</td><td>-16</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.23-1.40] </td><td>1.31</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 19:33:23</td><td>2024-02-16 19:33:23</td></tr>\n",
       "\t<tr><td>12453</td><td>9e-30</td><td>(non-obese)       </td><td>9</td><td>-30</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.39-1.59] </td><td>1.49</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 19:33:23</td><td>2024-02-16 19:33:23</td></tr>\n",
       "\t<tr><td>13346</td><td>7e-11</td><td>(DGI+FUSION+WTCCC)</td><td>7</td><td>-11</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.46</td><td>NA</td><td>[1.10-1.19] </td><td>1.14</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:11:10</td><td>2024-02-16 18:11:10</td></tr>\n",
       "\t<tr><td>13286</td><td>6e-08</td><td>NA                </td><td>6</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.89</td><td>NA</td><td>[1.28-1.71] </td><td>1.48</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:22:01</td><td>2024-02-16 18:22:01</td></tr>\n",
       "\t<tr><td>13339</td><td>9e-16</td><td>(DGI+FUSION+WTCCC)</td><td>9</td><td>-16</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.30</td><td>NA</td><td>[1.11-1.18] </td><td>1.14</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:27:51</td><td>2024-02-16 18:27:51</td></tr>\n",
       "\t<tr><td>13342</td><td>1e-12</td><td>(DGI+FUSION+WTCCC)</td><td>1</td><td>-12</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.38</td><td>NA</td><td>[1.12-1.22] </td><td>1.17</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:27:53</td><td>2024-02-16 18:27:53</td></tr>\n",
       "\t<tr><td>13345</td><td>6e-10</td><td>(DGI+FUSION+WTCCC)</td><td>6</td><td>-10</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.52</td><td>NA</td><td>[1.09-1.17] </td><td>1.13</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:27:57</td><td>2024-02-16 18:27:57</td></tr>\n",
       "\t<tr><td>13341</td><td>8e-15</td><td>(DGI+FUSION+WTCCC)</td><td>8</td><td>-15</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.85</td><td>NA</td><td>[1.14-1.25] </td><td>1.20</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:39:39</td><td>2024-02-16 18:39:39</td></tr>\n",
       "\t<tr><td>13340</td><td>4e-11</td><td>(DGI+FUSION+WTCCC)</td><td>4</td><td>-11</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.36</td><td>NA</td><td>[1.08-1.16] </td><td>1.12</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:49:01</td><td>2024-02-16 18:49:01</td></tr>\n",
       "\t<tr><td>13287</td><td>1e-08</td><td>NA                </td><td>1</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.18</td><td>NA</td><td>[1.21-1.49] </td><td>1.34</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:48:58</td><td>2024-02-16 18:48:58</td></tr>\n",
       "\t<tr><td>13344</td><td>5e-08</td><td>(DGI+FUSION+WTCCC)</td><td>5</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.61</td><td>NA</td><td>[1.07-1.16] </td><td>1.12</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:49:03</td><td>2024-02-16 18:49:03</td></tr>\n",
       "\t<tr><td>13294</td><td>7e-14</td><td>NA                </td><td>7</td><td>-14</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.40</td><td>NA</td><td>[1.18-1.32] </td><td>1.23</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:05:29</td><td>2024-02-16 18:05:29</td></tr>\n",
       "\t<tr><td>13347</td><td>9e-16</td><td>(DGI+FUSION+WTCCC)</td><td>9</td><td>-16</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.32</td><td>NA</td><td>[1.11-1.18] </td><td>1.14</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:11:12</td><td>2024-02-16 18:11:12</td></tr>\n",
       "\t<tr><td>13348</td><td>5e-08</td><td>(DGI+FUSION+WTCCC)</td><td>5</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.69</td><td>NA</td><td>[1.07-1.16] </td><td>1.12</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:11:14</td><td>2024-02-16 18:11:14</td></tr>\n",
       "\t<tr><td>13295</td><td>1e-08</td><td>NA                </td><td>1</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.32</td><td>NA</td><td>[1.10-1.22] </td><td>1.16</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:14:47</td><td>2024-02-16 18:14:47</td></tr>\n",
       "\t<tr><td>13349</td><td>1e-48</td><td>(DGI+FUSION+WTCCC)</td><td>1</td><td>-48</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.31-1.43] </td><td>1.37</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:17:25</td><td>2024-02-16 18:17:25</td></tr>\n",
       "\t<tr><td>11785</td><td>5e-11</td><td>(DGI+FUSION+WTCCC)</td><td>5</td><td>-11</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.10-1.19] </td><td>1.14</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:47:57</td><td>2024-02-16 18:47:57</td></tr>\n",
       "\t<tr><td>11233</td><td>6e-08</td><td>NA                </td><td>6</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.70</td><td>NA</td><td>[0.69-1.67] </td><td>1.18</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:08:00</td><td>2024-02-16 18:08:00</td></tr>\n",
       "\t<tr><td>11320</td><td>2e-34</td><td>NA                </td><td>2</td><td>-34</td><td>FALSE</td><td>FALSE</td><td>known</td><td>0.30</td><td>NA</td><td>[1.28, 2.02]</td><td>1.65</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:24:50</td><td>2024-02-16 18:24:50</td></tr>\n",
       "\t<tr><td>5873 </td><td>5e-14</td><td>NA                </td><td>5</td><td>-14</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.50</td><td>NA</td><td>[1.07-1.13] </td><td>1.10</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:11:07</td><td>2024-02-16 18:11:07</td></tr>\n",
       "\t<tr><td>5874 </td><td>1e-10</td><td>NA                </td><td>1</td><td>-10</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.18</td><td>NA</td><td>[1.07-1.14] </td><td>1.11</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:11:09</td><td>2024-02-16 18:11:09</td></tr>\n",
       "\t<tr><td>5875 </td><td>1e-09</td><td>NA                </td><td>1</td><td> -9</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.27</td><td>NA</td><td>[1.06-1.12] </td><td>1.09</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:11:11</td><td>2024-02-16 18:11:11</td></tr>\n",
       "\t<tr><td>5876 </td><td>1e-09</td><td>NA                </td><td>1</td><td> -9</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.90</td><td>NA</td><td>[1.10-1.20] </td><td>1.15</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:11:14</td><td>2024-02-16 18:11:14</td></tr>\n",
       "\t<tr><td>11593</td><td>2e-07</td><td>NA                </td><td>2</td><td> -7</td><td>FALSE</td><td>FALSE</td><td>known</td><td>  NA</td><td>NA</td><td>[1.15-1.38] </td><td>1.26</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-16 18:20:43</td><td>2024-02-16 18:20:43</td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>100372749</td><td>3e-21</td><td>NA</td><td>3</td><td>-21</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:16:06</td><td>2024-02-20 13:16:06</td></tr>\n",
       "\t<tr><td>100372755</td><td>2e-23</td><td>NA</td><td>2</td><td>-23</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 14:18:13</td><td>2024-02-20 14:18:13</td></tr>\n",
       "\t<tr><td>100372761</td><td>8e-21</td><td>NA</td><td>8</td><td>-21</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:15:43</td><td>2024-02-20 13:15:44</td></tr>\n",
       "\t<tr><td>100372767</td><td>1e-20</td><td>NA</td><td>1</td><td>-20</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 14:06:48</td><td>2024-02-20 14:06:48</td></tr>\n",
       "\t<tr><td>100372773</td><td>4e-19</td><td>NA</td><td>4</td><td>-19</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:15:47</td><td>2024-02-20 13:15:47</td></tr>\n",
       "\t<tr><td>100372779</td><td>4e-22</td><td>NA</td><td>4</td><td>-22</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:15:49</td><td>2024-02-20 13:15:49</td></tr>\n",
       "\t<tr><td>100372785</td><td>6e-23</td><td>NA</td><td>6</td><td>-23</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 14:08:20</td><td>2024-02-20 14:08:20</td></tr>\n",
       "\t<tr><td>100372791</td><td>3e-23</td><td>NA</td><td>3</td><td>-23</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:15:52</td><td>2024-02-20 13:15:52</td></tr>\n",
       "\t<tr><td>100372797</td><td>2e-21</td><td>NA</td><td>2</td><td>-21</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 14:14:38</td><td>2024-02-20 14:14:38</td></tr>\n",
       "\t<tr><td>100372802</td><td>2e-23</td><td>NA</td><td>2</td><td>-23</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:15:55</td><td>2024-02-20 13:15:55</td></tr>\n",
       "\t<tr><td>100372808</td><td>3e-17</td><td>NA</td><td>3</td><td>-17</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:15:58</td><td>2024-02-20 13:15:58</td></tr>\n",
       "\t<tr><td>100372814</td><td>3e-22</td><td>NA</td><td>3</td><td>-22</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:31:04</td><td>2024-02-20 13:31:04</td></tr>\n",
       "\t<tr><td>100372819</td><td>3e-17</td><td>NA</td><td>3</td><td>-17</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:16:00</td><td>2024-02-20 13:16:00</td></tr>\n",
       "\t<tr><td>100372824</td><td>7e-23</td><td>NA</td><td>7</td><td>-23</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:23:11</td><td>2024-02-20 13:23:11</td></tr>\n",
       "\t<tr><td>100372830</td><td>1e-22</td><td>NA</td><td>1</td><td>-22</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:16:02</td><td>2024-02-20 13:16:02</td></tr>\n",
       "\t<tr><td>100372836</td><td>5e-19</td><td>NA</td><td>5</td><td>-19</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:16:05</td><td>2024-02-20 13:16:05</td></tr>\n",
       "\t<tr><td>100372842</td><td>3e-23</td><td>NA</td><td>3</td><td>-23</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 14:21:00</td><td>2024-02-20 14:21:00</td></tr>\n",
       "\t<tr><td>100372848</td><td>5e-24</td><td>NA</td><td>5</td><td>-24</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:16:08</td><td>2024-02-20 13:16:08</td></tr>\n",
       "\t<tr><td>100372854</td><td>1e-09</td><td>NA</td><td>1</td><td> -9</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:31:07</td><td>2024-02-20 13:31:07</td></tr>\n",
       "\t<tr><td>100372066</td><td>1e-12</td><td>NA</td><td>1</td><td>-12</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 14:49:32</td><td>2024-02-20 14:49:32</td></tr>\n",
       "\t<tr><td>100372072</td><td>2e-21</td><td>NA</td><td>2</td><td>-21</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>   NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 14:49:34</td><td>2024-02-20 14:49:34</td></tr>\n",
       "\t<tr><td>100478397</td><td>9e-39</td><td>NA</td><td>9</td><td>-39</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>1.342</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:52:54</td><td>2024-02-20 13:52:54</td></tr>\n",
       "\t<tr><td>100478404</td><td>6e-26</td><td>NA</td><td>6</td><td>-26</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>1.492</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:53:00</td><td>2024-02-20 13:53:00</td></tr>\n",
       "\t<tr><td>100478411</td><td>6e-24</td><td>NA</td><td>6</td><td>-24</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>0.797</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:52:10</td><td>2024-02-20 13:52:10</td></tr>\n",
       "\t<tr><td>100478415</td><td>7e-25</td><td>NA</td><td>7</td><td>-25</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>  NA</td><td>NA</td><td>NA         </td><td>0.787</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-20 13:52:13</td><td>2024-02-20 13:52:13</td></tr>\n",
       "\t<tr><td>129662696</td><td>4e-08</td><td>NA</td><td>4</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.04</td><td>NA</td><td>[0.73-0.86]</td><td>0.790</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-28 16:32:47</td><td>2024-02-28 16:34:57</td></tr>\n",
       "\t<tr><td>129662701</td><td>3e-08</td><td>NA</td><td>3</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.25</td><td>NA</td><td>[1.07-1.16]</td><td>1.110</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-28 16:32:48</td><td>2024-02-28 16:34:57</td></tr>\n",
       "\t<tr><td>129662708</td><td>5e-20</td><td>NA</td><td>5</td><td>-20</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.25</td><td>NA</td><td>[1.15-1.24]</td><td>1.190</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-28 16:32:48</td><td>2024-02-28 16:34:57</td></tr>\n",
       "\t<tr><td>129662714</td><td>4e-08</td><td>NA</td><td>4</td><td> -8</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.01</td><td>NA</td><td>[0.57-0.77]</td><td>0.660</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-28 16:32:48</td><td>2024-02-28 16:34:57</td></tr>\n",
       "\t<tr><td>129662719</td><td>1e-11</td><td>NA</td><td>1</td><td>-11</td><td>FALSE</td><td>FALSE</td><td>novel</td><td>0.46</td><td>NA</td><td>[1.08-1.16]</td><td>1.120</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>2024-02-28 16:32:49</td><td>2024-02-28 16:34:57</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A tibble: 4903 × 18\n",
       "\\begin{tabular}{llllllllllllllllll}\n",
       " association\\_id & pvalue & pvalue\\_description & pvalue\\_mantissa & pvalue\\_exponent & multiple\\_snp\\_haplotype & snp\\_interaction & snp\\_type & risk\\_frequency & standard\\_error & range & or\\_per\\_copy\\_number & beta\\_number & beta\\_unit & beta\\_direction & beta\\_description & last\\_mapping\\_date & last\\_update\\_date\\\\\n",
       " <chr> & <dbl> & <chr> & <int> & <int> & <lgl> & <lgl> & <chr> & <dbl> & <dbl> & <chr> & <dbl> & <dbl> & <chr> & <chr> & <chr> & <dttm> & <dttm>\\\\\n",
       "\\hline\n",
       "\t 11380 & 5e-08 & NA                 & 5 &  -8 & FALSE & FALSE & known &   NA & NA & {[}1.41-2.08{]}  & 1.71 & NA & NA & NA & NA & 2024-02-16 18:39:09 & 2024-02-16 18:39:09\\\\\n",
       "\t 12442 & 1e-09 & (non-obese)        & 1 &  -9 & FALSE & FALSE & known &   NA & NA & {[}1.16-1.34{]}  & 1.25 & NA & NA & NA & NA & 2024-02-16 18:07:37 & 2024-02-16 18:07:37\\\\\n",
       "\t 12443 & 7e-07 & (non-obese)        & 7 &  -7 & FALSE & FALSE & known &   NA & NA & {[}1.15-1.38{]}  & 1.26 & NA & NA & NA & NA & 2024-02-16 18:07:40 & 2024-02-16 18:07:40\\\\\n",
       "\t 12174 & 2e-17 & (obese)            & 2 & -17 & FALSE & FALSE & known &   NA & NA & {[}1.23-1.39{]}  & 1.30 & NA & NA & NA & NA & 2024-02-16 18:17:22 & 2024-02-16 18:17:22\\\\\n",
       "\t 12177 & 5e-07 & (obese)            & 5 &  -7 & FALSE & FALSE & known &   NA & NA & {[}1.11-1.27{]}  & 1.19 & NA & NA & NA & NA & 2024-02-16 18:34:32 & 2024-02-16 18:34:32\\\\\n",
       "\t 12444 & 7e-07 & (non-obese)        & 7 &  -7 & FALSE & FALSE & known &   NA & NA & {[}1.11-1.26{]}  & 1.18 & NA & NA & NA & NA & 2024-02-16 18:51:26 & 2024-02-16 18:51:26\\\\\n",
       "\t 12452 & 6e-16 & (obese)            & 6 & -16 & FALSE & FALSE & known &   NA & NA & {[}1.23-1.40{]}  & 1.31 & NA & NA & NA & NA & 2024-02-16 19:33:23 & 2024-02-16 19:33:23\\\\\n",
       "\t 12453 & 9e-30 & (non-obese)        & 9 & -30 & FALSE & FALSE & known &   NA & NA & {[}1.39-1.59{]}  & 1.49 & NA & NA & NA & NA & 2024-02-16 19:33:23 & 2024-02-16 19:33:23\\\\\n",
       "\t 13346 & 7e-11 & (DGI+FUSION+WTCCC) & 7 & -11 & FALSE & FALSE & known & 0.46 & NA & {[}1.10-1.19{]}  & 1.14 & NA & NA & NA & NA & 2024-02-16 18:11:10 & 2024-02-16 18:11:10\\\\\n",
       "\t 13286 & 6e-08 & NA                 & 6 &  -8 & FALSE & FALSE & novel & 0.89 & NA & {[}1.28-1.71{]}  & 1.48 & NA & NA & NA & NA & 2024-02-16 18:22:01 & 2024-02-16 18:22:01\\\\\n",
       "\t 13339 & 9e-16 & (DGI+FUSION+WTCCC) & 9 & -16 & FALSE & FALSE & novel & 0.30 & NA & {[}1.11-1.18{]}  & 1.14 & NA & NA & NA & NA & 2024-02-16 18:27:51 & 2024-02-16 18:27:51\\\\\n",
       "\t 13342 & 1e-12 & (DGI+FUSION+WTCCC) & 1 & -12 & FALSE & FALSE & novel & 0.38 & NA & {[}1.12-1.22{]}  & 1.17 & NA & NA & NA & NA & 2024-02-16 18:27:53 & 2024-02-16 18:27:53\\\\\n",
       "\t 13345 & 6e-10 & (DGI+FUSION+WTCCC) & 6 & -10 & FALSE & FALSE & known & 0.52 & NA & {[}1.09-1.17{]}  & 1.13 & NA & NA & NA & NA & 2024-02-16 18:27:57 & 2024-02-16 18:27:57\\\\\n",
       "\t 13341 & 8e-15 & (DGI+FUSION+WTCCC) & 8 & -15 & FALSE & FALSE & novel & 0.85 & NA & {[}1.14-1.25{]}  & 1.20 & NA & NA & NA & NA & 2024-02-16 18:39:39 & 2024-02-16 18:39:39\\\\\n",
       "\t 13340 & 4e-11 & (DGI+FUSION+WTCCC) & 4 & -11 & FALSE & FALSE & novel & 0.36 & NA & {[}1.08-1.16{]}  & 1.12 & NA & NA & NA & NA & 2024-02-16 18:49:01 & 2024-02-16 18:49:01\\\\\n",
       "\t 13287 & 1e-08 & NA                 & 1 &  -8 & FALSE & FALSE & known & 0.18 & NA & {[}1.21-1.49{]}  & 1.34 & NA & NA & NA & NA & 2024-02-16 18:48:58 & 2024-02-16 18:48:58\\\\\n",
       "\t 13344 & 5e-08 & (DGI+FUSION+WTCCC) & 5 &  -8 & FALSE & FALSE & known & 0.61 & NA & {[}1.07-1.16{]}  & 1.12 & NA & NA & NA & NA & 2024-02-16 18:49:03 & 2024-02-16 18:49:03\\\\\n",
       "\t 13294 & 7e-14 & NA                 & 7 & -14 & FALSE & FALSE & known & 0.40 & NA & {[}1.18-1.32{]}  & 1.23 & NA & NA & NA & NA & 2024-02-16 18:05:29 & 2024-02-16 18:05:29\\\\\n",
       "\t 13347 & 9e-16 & (DGI+FUSION+WTCCC) & 9 & -16 & FALSE & FALSE & novel & 0.32 & NA & {[}1.11-1.18{]}  & 1.14 & NA & NA & NA & NA & 2024-02-16 18:11:12 & 2024-02-16 18:11:12\\\\\n",
       "\t 13348 & 5e-08 & (DGI+FUSION+WTCCC) & 5 &  -8 & FALSE & FALSE & known & 0.69 & NA & {[}1.07-1.16{]}  & 1.12 & NA & NA & NA & NA & 2024-02-16 18:11:14 & 2024-02-16 18:11:14\\\\\n",
       "\t 13295 & 1e-08 & NA                 & 1 &  -8 & FALSE & FALSE & known & 0.32 & NA & {[}1.10-1.22{]}  & 1.16 & NA & NA & NA & NA & 2024-02-16 18:14:47 & 2024-02-16 18:14:47\\\\\n",
       "\t 13349 & 1e-48 & (DGI+FUSION+WTCCC) & 1 & -48 & FALSE & FALSE & known &   NA & NA & {[}1.31-1.43{]}  & 1.37 & NA & NA & NA & NA & 2024-02-16 18:17:25 & 2024-02-16 18:17:25\\\\\n",
       "\t 11785 & 5e-11 & (DGI+FUSION+WTCCC) & 5 & -11 & FALSE & FALSE & known &   NA & NA & {[}1.10-1.19{]}  & 1.14 & NA & NA & NA & NA & 2024-02-16 18:47:57 & 2024-02-16 18:47:57\\\\\n",
       "\t 11233 & 6e-08 & NA                 & 6 &  -8 & FALSE & FALSE & novel & 0.70 & NA & {[}0.69-1.67{]}  & 1.18 & NA & NA & NA & NA & 2024-02-16 18:08:00 & 2024-02-16 18:08:00\\\\\n",
       "\t 11320 & 2e-34 & NA                 & 2 & -34 & FALSE & FALSE & known & 0.30 & NA & {[}1.28, 2.02{]} & 1.65 & NA & NA & NA & NA & 2024-02-16 18:24:50 & 2024-02-16 18:24:50\\\\\n",
       "\t 5873  & 5e-14 & NA                 & 5 & -14 & FALSE & FALSE & novel & 0.50 & NA & {[}1.07-1.13{]}  & 1.10 & NA & NA & NA & NA & 2024-02-16 18:11:07 & 2024-02-16 18:11:07\\\\\n",
       "\t 5874  & 1e-10 & NA                 & 1 & -10 & FALSE & FALSE & novel & 0.18 & NA & {[}1.07-1.14{]}  & 1.11 & NA & NA & NA & NA & 2024-02-16 18:11:09 & 2024-02-16 18:11:09\\\\\n",
       "\t 5875  & 1e-09 & NA                 & 1 &  -9 & FALSE & FALSE & novel & 0.27 & NA & {[}1.06-1.12{]}  & 1.09 & NA & NA & NA & NA & 2024-02-16 18:11:11 & 2024-02-16 18:11:11\\\\\n",
       "\t 5876  & 1e-09 & NA                 & 1 &  -9 & FALSE & FALSE & novel & 0.90 & NA & {[}1.10-1.20{]}  & 1.15 & NA & NA & NA & NA & 2024-02-16 18:11:14 & 2024-02-16 18:11:14\\\\\n",
       "\t 11593 & 2e-07 & NA                 & 2 &  -7 & FALSE & FALSE & known &   NA & NA & {[}1.15-1.38{]}  & 1.26 & NA & NA & NA & NA & 2024-02-16 18:20:43 & 2024-02-16 18:20:43\\\\\n",
       "\t ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\t 100372749 & 3e-21 & NA & 3 & -21 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:16:06 & 2024-02-20 13:16:06\\\\\n",
       "\t 100372755 & 2e-23 & NA & 2 & -23 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 14:18:13 & 2024-02-20 14:18:13\\\\\n",
       "\t 100372761 & 8e-21 & NA & 8 & -21 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:15:43 & 2024-02-20 13:15:44\\\\\n",
       "\t 100372767 & 1e-20 & NA & 1 & -20 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 14:06:48 & 2024-02-20 14:06:48\\\\\n",
       "\t 100372773 & 4e-19 & NA & 4 & -19 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:15:47 & 2024-02-20 13:15:47\\\\\n",
       "\t 100372779 & 4e-22 & NA & 4 & -22 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:15:49 & 2024-02-20 13:15:49\\\\\n",
       "\t 100372785 & 6e-23 & NA & 6 & -23 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 14:08:20 & 2024-02-20 14:08:20\\\\\n",
       "\t 100372791 & 3e-23 & NA & 3 & -23 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:15:52 & 2024-02-20 13:15:52\\\\\n",
       "\t 100372797 & 2e-21 & NA & 2 & -21 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 14:14:38 & 2024-02-20 14:14:38\\\\\n",
       "\t 100372802 & 2e-23 & NA & 2 & -23 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:15:55 & 2024-02-20 13:15:55\\\\\n",
       "\t 100372808 & 3e-17 & NA & 3 & -17 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:15:58 & 2024-02-20 13:15:58\\\\\n",
       "\t 100372814 & 3e-22 & NA & 3 & -22 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:31:04 & 2024-02-20 13:31:04\\\\\n",
       "\t 100372819 & 3e-17 & NA & 3 & -17 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:16:00 & 2024-02-20 13:16:00\\\\\n",
       "\t 100372824 & 7e-23 & NA & 7 & -23 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:23:11 & 2024-02-20 13:23:11\\\\\n",
       "\t 100372830 & 1e-22 & NA & 1 & -22 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:16:02 & 2024-02-20 13:16:02\\\\\n",
       "\t 100372836 & 5e-19 & NA & 5 & -19 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:16:05 & 2024-02-20 13:16:05\\\\\n",
       "\t 100372842 & 3e-23 & NA & 3 & -23 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 14:21:00 & 2024-02-20 14:21:00\\\\\n",
       "\t 100372848 & 5e-24 & NA & 5 & -24 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:16:08 & 2024-02-20 13:16:08\\\\\n",
       "\t 100372854 & 1e-09 & NA & 1 &  -9 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 13:31:07 & 2024-02-20 13:31:07\\\\\n",
       "\t 100372066 & 1e-12 & NA & 1 & -12 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 14:49:32 & 2024-02-20 14:49:32\\\\\n",
       "\t 100372072 & 2e-21 & NA & 2 & -21 & FALSE & FALSE & novel &   NA & NA & NA          &    NA & NA & NA & NA & NA & 2024-02-20 14:49:34 & 2024-02-20 14:49:34\\\\\n",
       "\t 100478397 & 9e-39 & NA & 9 & -39 & FALSE & FALSE & novel &   NA & NA & NA          & 1.342 & NA & NA & NA & NA & 2024-02-20 13:52:54 & 2024-02-20 13:52:54\\\\\n",
       "\t 100478404 & 6e-26 & NA & 6 & -26 & FALSE & FALSE & novel &   NA & NA & NA          & 1.492 & NA & NA & NA & NA & 2024-02-20 13:53:00 & 2024-02-20 13:53:00\\\\\n",
       "\t 100478411 & 6e-24 & NA & 6 & -24 & FALSE & FALSE & novel &   NA & NA & NA          & 0.797 & NA & NA & NA & NA & 2024-02-20 13:52:10 & 2024-02-20 13:52:10\\\\\n",
       "\t 100478415 & 7e-25 & NA & 7 & -25 & FALSE & FALSE & novel &   NA & NA & NA          & 0.787 & NA & NA & NA & NA & 2024-02-20 13:52:13 & 2024-02-20 13:52:13\\\\\n",
       "\t 129662696 & 4e-08 & NA & 4 &  -8 & FALSE & FALSE & novel & 0.04 & NA & {[}0.73-0.86{]} & 0.790 & NA & NA & NA & NA & 2024-02-28 16:32:47 & 2024-02-28 16:34:57\\\\\n",
       "\t 129662701 & 3e-08 & NA & 3 &  -8 & FALSE & FALSE & novel & 0.25 & NA & {[}1.07-1.16{]} & 1.110 & NA & NA & NA & NA & 2024-02-28 16:32:48 & 2024-02-28 16:34:57\\\\\n",
       "\t 129662708 & 5e-20 & NA & 5 & -20 & FALSE & FALSE & novel & 0.25 & NA & {[}1.15-1.24{]} & 1.190 & NA & NA & NA & NA & 2024-02-28 16:32:48 & 2024-02-28 16:34:57\\\\\n",
       "\t 129662714 & 4e-08 & NA & 4 &  -8 & FALSE & FALSE & novel & 0.01 & NA & {[}0.57-0.77{]} & 0.660 & NA & NA & NA & NA & 2024-02-28 16:32:48 & 2024-02-28 16:34:57\\\\\n",
       "\t 129662719 & 1e-11 & NA & 1 & -11 & FALSE & FALSE & novel & 0.46 & NA & {[}1.08-1.16{]} & 1.120 & NA & NA & NA & NA & 2024-02-28 16:32:49 & 2024-02-28 16:34:57\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A tibble: 4903 × 18\n",
       "\n",
       "| association_id &lt;chr&gt; | pvalue &lt;dbl&gt; | pvalue_description &lt;chr&gt; | pvalue_mantissa &lt;int&gt; | pvalue_exponent &lt;int&gt; | multiple_snp_haplotype &lt;lgl&gt; | snp_interaction &lt;lgl&gt; | snp_type &lt;chr&gt; | risk_frequency &lt;dbl&gt; | standard_error &lt;dbl&gt; | range &lt;chr&gt; | or_per_copy_number &lt;dbl&gt; | beta_number &lt;dbl&gt; | beta_unit &lt;chr&gt; | beta_direction &lt;chr&gt; | beta_description &lt;chr&gt; | last_mapping_date &lt;dttm&gt; | last_update_date &lt;dttm&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| 11380 | 5e-08 | NA                 | 5 |  -8 | FALSE | FALSE | known |   NA | NA | [1.41-2.08]  | 1.71 | NA | NA | NA | NA | 2024-02-16 18:39:09 | 2024-02-16 18:39:09 |\n",
       "| 12442 | 1e-09 | (non-obese)        | 1 |  -9 | FALSE | FALSE | known |   NA | NA | [1.16-1.34]  | 1.25 | NA | NA | NA | NA | 2024-02-16 18:07:37 | 2024-02-16 18:07:37 |\n",
       "| 12443 | 7e-07 | (non-obese)        | 7 |  -7 | FALSE | FALSE | known |   NA | NA | [1.15-1.38]  | 1.26 | NA | NA | NA | NA | 2024-02-16 18:07:40 | 2024-02-16 18:07:40 |\n",
       "| 12174 | 2e-17 | (obese)            | 2 | -17 | FALSE | FALSE | known |   NA | NA | [1.23-1.39]  | 1.30 | NA | NA | NA | NA | 2024-02-16 18:17:22 | 2024-02-16 18:17:22 |\n",
       "| 12177 | 5e-07 | (obese)            | 5 |  -7 | FALSE | FALSE | known |   NA | NA | [1.11-1.27]  | 1.19 | NA | NA | NA | NA | 2024-02-16 18:34:32 | 2024-02-16 18:34:32 |\n",
       "| 12444 | 7e-07 | (non-obese)        | 7 |  -7 | FALSE | FALSE | known |   NA | NA | [1.11-1.26]  | 1.18 | NA | NA | NA | NA | 2024-02-16 18:51:26 | 2024-02-16 18:51:26 |\n",
       "| 12452 | 6e-16 | (obese)            | 6 | -16 | FALSE | FALSE | known |   NA | NA | [1.23-1.40]  | 1.31 | NA | NA | NA | NA | 2024-02-16 19:33:23 | 2024-02-16 19:33:23 |\n",
       "| 12453 | 9e-30 | (non-obese)        | 9 | -30 | FALSE | FALSE | known |   NA | NA | [1.39-1.59]  | 1.49 | NA | NA | NA | NA | 2024-02-16 19:33:23 | 2024-02-16 19:33:23 |\n",
       "| 13346 | 7e-11 | (DGI+FUSION+WTCCC) | 7 | -11 | FALSE | FALSE | known | 0.46 | NA | [1.10-1.19]  | 1.14 | NA | NA | NA | NA | 2024-02-16 18:11:10 | 2024-02-16 18:11:10 |\n",
       "| 13286 | 6e-08 | NA                 | 6 |  -8 | FALSE | FALSE | novel | 0.89 | NA | [1.28-1.71]  | 1.48 | NA | NA | NA | NA | 2024-02-16 18:22:01 | 2024-02-16 18:22:01 |\n",
       "| 13339 | 9e-16 | (DGI+FUSION+WTCCC) | 9 | -16 | FALSE | FALSE | novel | 0.30 | NA | [1.11-1.18]  | 1.14 | NA | NA | NA | NA | 2024-02-16 18:27:51 | 2024-02-16 18:27:51 |\n",
       "| 13342 | 1e-12 | (DGI+FUSION+WTCCC) | 1 | -12 | FALSE | FALSE | novel | 0.38 | NA | [1.12-1.22]  | 1.17 | NA | NA | NA | NA | 2024-02-16 18:27:53 | 2024-02-16 18:27:53 |\n",
       "| 13345 | 6e-10 | (DGI+FUSION+WTCCC) | 6 | -10 | FALSE | FALSE | known | 0.52 | NA | [1.09-1.17]  | 1.13 | NA | NA | NA | NA | 2024-02-16 18:27:57 | 2024-02-16 18:27:57 |\n",
       "| 13341 | 8e-15 | (DGI+FUSION+WTCCC) | 8 | -15 | FALSE | FALSE | novel | 0.85 | NA | [1.14-1.25]  | 1.20 | NA | NA | NA | NA | 2024-02-16 18:39:39 | 2024-02-16 18:39:39 |\n",
       "| 13340 | 4e-11 | (DGI+FUSION+WTCCC) | 4 | -11 | FALSE | FALSE | novel | 0.36 | NA | [1.08-1.16]  | 1.12 | NA | NA | NA | NA | 2024-02-16 18:49:01 | 2024-02-16 18:49:01 |\n",
       "| 13287 | 1e-08 | NA                 | 1 |  -8 | FALSE | FALSE | known | 0.18 | NA | [1.21-1.49]  | 1.34 | NA | NA | NA | NA | 2024-02-16 18:48:58 | 2024-02-16 18:48:58 |\n",
       "| 13344 | 5e-08 | (DGI+FUSION+WTCCC) | 5 |  -8 | FALSE | FALSE | known | 0.61 | NA | [1.07-1.16]  | 1.12 | NA | NA | NA | NA | 2024-02-16 18:49:03 | 2024-02-16 18:49:03 |\n",
       "| 13294 | 7e-14 | NA                 | 7 | -14 | FALSE | FALSE | known | 0.40 | NA | [1.18-1.32]  | 1.23 | NA | NA | NA | NA | 2024-02-16 18:05:29 | 2024-02-16 18:05:29 |\n",
       "| 13347 | 9e-16 | (DGI+FUSION+WTCCC) | 9 | -16 | FALSE | FALSE | novel | 0.32 | NA | [1.11-1.18]  | 1.14 | NA | NA | NA | NA | 2024-02-16 18:11:12 | 2024-02-16 18:11:12 |\n",
       "| 13348 | 5e-08 | (DGI+FUSION+WTCCC) | 5 |  -8 | FALSE | FALSE | known | 0.69 | NA | [1.07-1.16]  | 1.12 | NA | NA | NA | NA | 2024-02-16 18:11:14 | 2024-02-16 18:11:14 |\n",
       "| 13295 | 1e-08 | NA                 | 1 |  -8 | FALSE | FALSE | known | 0.32 | NA | [1.10-1.22]  | 1.16 | NA | NA | NA | NA | 2024-02-16 18:14:47 | 2024-02-16 18:14:47 |\n",
       "| 13349 | 1e-48 | (DGI+FUSION+WTCCC) | 1 | -48 | FALSE | FALSE | known |   NA | NA | [1.31-1.43]  | 1.37 | NA | NA | NA | NA | 2024-02-16 18:17:25 | 2024-02-16 18:17:25 |\n",
       "| 11785 | 5e-11 | (DGI+FUSION+WTCCC) | 5 | -11 | FALSE | FALSE | known |   NA | NA | [1.10-1.19]  | 1.14 | NA | NA | NA | NA | 2024-02-16 18:47:57 | 2024-02-16 18:47:57 |\n",
       "| 11233 | 6e-08 | NA                 | 6 |  -8 | FALSE | FALSE | novel | 0.70 | NA | [0.69-1.67]  | 1.18 | NA | NA | NA | NA | 2024-02-16 18:08:00 | 2024-02-16 18:08:00 |\n",
       "| 11320 | 2e-34 | NA                 | 2 | -34 | FALSE | FALSE | known | 0.30 | NA | [1.28, 2.02] | 1.65 | NA | NA | NA | NA | 2024-02-16 18:24:50 | 2024-02-16 18:24:50 |\n",
       "| 5873  | 5e-14 | NA                 | 5 | -14 | FALSE | FALSE | novel | 0.50 | NA | [1.07-1.13]  | 1.10 | NA | NA | NA | NA | 2024-02-16 18:11:07 | 2024-02-16 18:11:07 |\n",
       "| 5874  | 1e-10 | NA                 | 1 | -10 | FALSE | FALSE | novel | 0.18 | NA | [1.07-1.14]  | 1.11 | NA | NA | NA | NA | 2024-02-16 18:11:09 | 2024-02-16 18:11:09 |\n",
       "| 5875  | 1e-09 | NA                 | 1 |  -9 | FALSE | FALSE | novel | 0.27 | NA | [1.06-1.12]  | 1.09 | NA | NA | NA | NA | 2024-02-16 18:11:11 | 2024-02-16 18:11:11 |\n",
       "| 5876  | 1e-09 | NA                 | 1 |  -9 | FALSE | FALSE | novel | 0.90 | NA | [1.10-1.20]  | 1.15 | NA | NA | NA | NA | 2024-02-16 18:11:14 | 2024-02-16 18:11:14 |\n",
       "| 11593 | 2e-07 | NA                 | 2 |  -7 | FALSE | FALSE | known |   NA | NA | [1.15-1.38]  | 1.26 | NA | NA | NA | NA | 2024-02-16 18:20:43 | 2024-02-16 18:20:43 |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| 100372749 | 3e-21 | NA | 3 | -21 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:16:06 | 2024-02-20 13:16:06 |\n",
       "| 100372755 | 2e-23 | NA | 2 | -23 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 14:18:13 | 2024-02-20 14:18:13 |\n",
       "| 100372761 | 8e-21 | NA | 8 | -21 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:15:43 | 2024-02-20 13:15:44 |\n",
       "| 100372767 | 1e-20 | NA | 1 | -20 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 14:06:48 | 2024-02-20 14:06:48 |\n",
       "| 100372773 | 4e-19 | NA | 4 | -19 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:15:47 | 2024-02-20 13:15:47 |\n",
       "| 100372779 | 4e-22 | NA | 4 | -22 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:15:49 | 2024-02-20 13:15:49 |\n",
       "| 100372785 | 6e-23 | NA | 6 | -23 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 14:08:20 | 2024-02-20 14:08:20 |\n",
       "| 100372791 | 3e-23 | NA | 3 | -23 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:15:52 | 2024-02-20 13:15:52 |\n",
       "| 100372797 | 2e-21 | NA | 2 | -21 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 14:14:38 | 2024-02-20 14:14:38 |\n",
       "| 100372802 | 2e-23 | NA | 2 | -23 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:15:55 | 2024-02-20 13:15:55 |\n",
       "| 100372808 | 3e-17 | NA | 3 | -17 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:15:58 | 2024-02-20 13:15:58 |\n",
       "| 100372814 | 3e-22 | NA | 3 | -22 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:31:04 | 2024-02-20 13:31:04 |\n",
       "| 100372819 | 3e-17 | NA | 3 | -17 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:16:00 | 2024-02-20 13:16:00 |\n",
       "| 100372824 | 7e-23 | NA | 7 | -23 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:23:11 | 2024-02-20 13:23:11 |\n",
       "| 100372830 | 1e-22 | NA | 1 | -22 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:16:02 | 2024-02-20 13:16:02 |\n",
       "| 100372836 | 5e-19 | NA | 5 | -19 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:16:05 | 2024-02-20 13:16:05 |\n",
       "| 100372842 | 3e-23 | NA | 3 | -23 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 14:21:00 | 2024-02-20 14:21:00 |\n",
       "| 100372848 | 5e-24 | NA | 5 | -24 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:16:08 | 2024-02-20 13:16:08 |\n",
       "| 100372854 | 1e-09 | NA | 1 |  -9 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 13:31:07 | 2024-02-20 13:31:07 |\n",
       "| 100372066 | 1e-12 | NA | 1 | -12 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 14:49:32 | 2024-02-20 14:49:32 |\n",
       "| 100372072 | 2e-21 | NA | 2 | -21 | FALSE | FALSE | novel |   NA | NA | NA          |    NA | NA | NA | NA | NA | 2024-02-20 14:49:34 | 2024-02-20 14:49:34 |\n",
       "| 100478397 | 9e-39 | NA | 9 | -39 | FALSE | FALSE | novel |   NA | NA | NA          | 1.342 | NA | NA | NA | NA | 2024-02-20 13:52:54 | 2024-02-20 13:52:54 |\n",
       "| 100478404 | 6e-26 | NA | 6 | -26 | FALSE | FALSE | novel |   NA | NA | NA          | 1.492 | NA | NA | NA | NA | 2024-02-20 13:53:00 | 2024-02-20 13:53:00 |\n",
       "| 100478411 | 6e-24 | NA | 6 | -24 | FALSE | FALSE | novel |   NA | NA | NA          | 0.797 | NA | NA | NA | NA | 2024-02-20 13:52:10 | 2024-02-20 13:52:10 |\n",
       "| 100478415 | 7e-25 | NA | 7 | -25 | FALSE | FALSE | novel |   NA | NA | NA          | 0.787 | NA | NA | NA | NA | 2024-02-20 13:52:13 | 2024-02-20 13:52:13 |\n",
       "| 129662696 | 4e-08 | NA | 4 |  -8 | FALSE | FALSE | novel | 0.04 | NA | [0.73-0.86] | 0.790 | NA | NA | NA | NA | 2024-02-28 16:32:47 | 2024-02-28 16:34:57 |\n",
       "| 129662701 | 3e-08 | NA | 3 |  -8 | FALSE | FALSE | novel | 0.25 | NA | [1.07-1.16] | 1.110 | NA | NA | NA | NA | 2024-02-28 16:32:48 | 2024-02-28 16:34:57 |\n",
       "| 129662708 | 5e-20 | NA | 5 | -20 | FALSE | FALSE | novel | 0.25 | NA | [1.15-1.24] | 1.190 | NA | NA | NA | NA | 2024-02-28 16:32:48 | 2024-02-28 16:34:57 |\n",
       "| 129662714 | 4e-08 | NA | 4 |  -8 | FALSE | FALSE | novel | 0.01 | NA | [0.57-0.77] | 0.660 | NA | NA | NA | NA | 2024-02-28 16:32:48 | 2024-02-28 16:34:57 |\n",
       "| 129662719 | 1e-11 | NA | 1 | -11 | FALSE | FALSE | novel | 0.46 | NA | [1.08-1.16] | 1.120 | NA | NA | NA | NA | 2024-02-28 16:32:49 | 2024-02-28 16:34:57 |\n",
       "\n"
      ],
      "text/plain": [
       "     association_id pvalue pvalue_description pvalue_mantissa pvalue_exponent\n",
       "1    11380          5e-08  NA                 5                -8            \n",
       "2    12442          1e-09  (non-obese)        1                -9            \n",
       "3    12443          7e-07  (non-obese)        7                -7            \n",
       "4    12174          2e-17  (obese)            2               -17            \n",
       "5    12177          5e-07  (obese)            5                -7            \n",
       "6    12444          7e-07  (non-obese)        7                -7            \n",
       "7    12452          6e-16  (obese)            6               -16            \n",
       "8    12453          9e-30  (non-obese)        9               -30            \n",
       "9    13346          7e-11  (DGI+FUSION+WTCCC) 7               -11            \n",
       "10   13286          6e-08  NA                 6                -8            \n",
       "11   13339          9e-16  (DGI+FUSION+WTCCC) 9               -16            \n",
       "12   13342          1e-12  (DGI+FUSION+WTCCC) 1               -12            \n",
       "13   13345          6e-10  (DGI+FUSION+WTCCC) 6               -10            \n",
       "14   13341          8e-15  (DGI+FUSION+WTCCC) 8               -15            \n",
       "15   13340          4e-11  (DGI+FUSION+WTCCC) 4               -11            \n",
       "16   13287          1e-08  NA                 1                -8            \n",
       "17   13344          5e-08  (DGI+FUSION+WTCCC) 5                -8            \n",
       "18   13294          7e-14  NA                 7               -14            \n",
       "19   13347          9e-16  (DGI+FUSION+WTCCC) 9               -16            \n",
       "20   13348          5e-08  (DGI+FUSION+WTCCC) 5                -8            \n",
       "21   13295          1e-08  NA                 1                -8            \n",
       "22   13349          1e-48  (DGI+FUSION+WTCCC) 1               -48            \n",
       "23   11785          5e-11  (DGI+FUSION+WTCCC) 5               -11            \n",
       "24   11233          6e-08  NA                 6                -8            \n",
       "25   11320          2e-34  NA                 2               -34            \n",
       "26   5873           5e-14  NA                 5               -14            \n",
       "27   5874           1e-10  NA                 1               -10            \n",
       "28   5875           1e-09  NA                 1                -9            \n",
       "29   5876           1e-09  NA                 1                -9            \n",
       "30   11593          2e-07  NA                 2                -7            \n",
       "⋮    ⋮              ⋮      ⋮                  ⋮               ⋮              \n",
       "4874 100372749      3e-21  NA                 3               -21            \n",
       "4875 100372755      2e-23  NA                 2               -23            \n",
       "4876 100372761      8e-21  NA                 8               -21            \n",
       "4877 100372767      1e-20  NA                 1               -20            \n",
       "4878 100372773      4e-19  NA                 4               -19            \n",
       "4879 100372779      4e-22  NA                 4               -22            \n",
       "4880 100372785      6e-23  NA                 6               -23            \n",
       "4881 100372791      3e-23  NA                 3               -23            \n",
       "4882 100372797      2e-21  NA                 2               -21            \n",
       "4883 100372802      2e-23  NA                 2               -23            \n",
       "4884 100372808      3e-17  NA                 3               -17            \n",
       "4885 100372814      3e-22  NA                 3               -22            \n",
       "4886 100372819      3e-17  NA                 3               -17            \n",
       "4887 100372824      7e-23  NA                 7               -23            \n",
       "4888 100372830      1e-22  NA                 1               -22            \n",
       "4889 100372836      5e-19  NA                 5               -19            \n",
       "4890 100372842      3e-23  NA                 3               -23            \n",
       "4891 100372848      5e-24  NA                 5               -24            \n",
       "4892 100372854      1e-09  NA                 1                -9            \n",
       "4893 100372066      1e-12  NA                 1               -12            \n",
       "4894 100372072      2e-21  NA                 2               -21            \n",
       "4895 100478397      9e-39  NA                 9               -39            \n",
       "4896 100478404      6e-26  NA                 6               -26            \n",
       "4897 100478411      6e-24  NA                 6               -24            \n",
       "4898 100478415      7e-25  NA                 7               -25            \n",
       "4899 129662696      4e-08  NA                 4                -8            \n",
       "4900 129662701      3e-08  NA                 3                -8            \n",
       "4901 129662708      5e-20  NA                 5               -20            \n",
       "4902 129662714      4e-08  NA                 4                -8            \n",
       "4903 129662719      1e-11  NA                 1               -11            \n",
       "     multiple_snp_haplotype snp_interaction snp_type risk_frequency\n",
       "1    FALSE                  FALSE           known      NA          \n",
       "2    FALSE                  FALSE           known      NA          \n",
       "3    FALSE                  FALSE           known      NA          \n",
       "4    FALSE                  FALSE           known      NA          \n",
       "5    FALSE                  FALSE           known      NA          \n",
       "6    FALSE                  FALSE           known      NA          \n",
       "7    FALSE                  FALSE           known      NA          \n",
       "8    FALSE                  FALSE           known      NA          \n",
       "9    FALSE                  FALSE           known    0.46          \n",
       "10   FALSE                  FALSE           novel    0.89          \n",
       "11   FALSE                  FALSE           novel    0.30          \n",
       "12   FALSE                  FALSE           novel    0.38          \n",
       "13   FALSE                  FALSE           known    0.52          \n",
       "14   FALSE                  FALSE           novel    0.85          \n",
       "15   FALSE                  FALSE           novel    0.36          \n",
       "16   FALSE                  FALSE           known    0.18          \n",
       "17   FALSE                  FALSE           known    0.61          \n",
       "18   FALSE                  FALSE           known    0.40          \n",
       "19   FALSE                  FALSE           novel    0.32          \n",
       "20   FALSE                  FALSE           known    0.69          \n",
       "21   FALSE                  FALSE           known    0.32          \n",
       "22   FALSE                  FALSE           known      NA          \n",
       "23   FALSE                  FALSE           known      NA          \n",
       "24   FALSE                  FALSE           novel    0.70          \n",
       "25   FALSE                  FALSE           known    0.30          \n",
       "26   FALSE                  FALSE           novel    0.50          \n",
       "27   FALSE                  FALSE           novel    0.18          \n",
       "28   FALSE                  FALSE           novel    0.27          \n",
       "29   FALSE                  FALSE           novel    0.90          \n",
       "30   FALSE                  FALSE           known      NA          \n",
       "⋮    ⋮                      ⋮               ⋮        ⋮             \n",
       "4874 FALSE                  FALSE           novel      NA          \n",
       "4875 FALSE                  FALSE           novel      NA          \n",
       "4876 FALSE                  FALSE           novel      NA          \n",
       "4877 FALSE                  FALSE           novel      NA          \n",
       "4878 FALSE                  FALSE           novel      NA          \n",
       "4879 FALSE                  FALSE           novel      NA          \n",
       "4880 FALSE                  FALSE           novel      NA          \n",
       "4881 FALSE                  FALSE           novel      NA          \n",
       "4882 FALSE                  FALSE           novel      NA          \n",
       "4883 FALSE                  FALSE           novel      NA          \n",
       "4884 FALSE                  FALSE           novel      NA          \n",
       "4885 FALSE                  FALSE           novel      NA          \n",
       "4886 FALSE                  FALSE           novel      NA          \n",
       "4887 FALSE                  FALSE           novel      NA          \n",
       "4888 FALSE                  FALSE           novel      NA          \n",
       "4889 FALSE                  FALSE           novel      NA          \n",
       "4890 FALSE                  FALSE           novel      NA          \n",
       "4891 FALSE                  FALSE           novel      NA          \n",
       "4892 FALSE                  FALSE           novel      NA          \n",
       "4893 FALSE                  FALSE           novel      NA          \n",
       "4894 FALSE                  FALSE           novel      NA          \n",
       "4895 FALSE                  FALSE           novel      NA          \n",
       "4896 FALSE                  FALSE           novel      NA          \n",
       "4897 FALSE                  FALSE           novel      NA          \n",
       "4898 FALSE                  FALSE           novel      NA          \n",
       "4899 FALSE                  FALSE           novel    0.04          \n",
       "4900 FALSE                  FALSE           novel    0.25          \n",
       "4901 FALSE                  FALSE           novel    0.25          \n",
       "4902 FALSE                  FALSE           novel    0.01          \n",
       "4903 FALSE                  FALSE           novel    0.46          \n",
       "     standard_error range        or_per_copy_number beta_number beta_unit\n",
       "1    NA             [1.41-2.08]  1.71               NA          NA       \n",
       "2    NA             [1.16-1.34]  1.25               NA          NA       \n",
       "3    NA             [1.15-1.38]  1.26               NA          NA       \n",
       "4    NA             [1.23-1.39]  1.30               NA          NA       \n",
       "5    NA             [1.11-1.27]  1.19               NA          NA       \n",
       "6    NA             [1.11-1.26]  1.18               NA          NA       \n",
       "7    NA             [1.23-1.40]  1.31               NA          NA       \n",
       "8    NA             [1.39-1.59]  1.49               NA          NA       \n",
       "9    NA             [1.10-1.19]  1.14               NA          NA       \n",
       "10   NA             [1.28-1.71]  1.48               NA          NA       \n",
       "11   NA             [1.11-1.18]  1.14               NA          NA       \n",
       "12   NA             [1.12-1.22]  1.17               NA          NA       \n",
       "13   NA             [1.09-1.17]  1.13               NA          NA       \n",
       "14   NA             [1.14-1.25]  1.20               NA          NA       \n",
       "15   NA             [1.08-1.16]  1.12               NA          NA       \n",
       "16   NA             [1.21-1.49]  1.34               NA          NA       \n",
       "17   NA             [1.07-1.16]  1.12               NA          NA       \n",
       "18   NA             [1.18-1.32]  1.23               NA          NA       \n",
       "19   NA             [1.11-1.18]  1.14               NA          NA       \n",
       "20   NA             [1.07-1.16]  1.12               NA          NA       \n",
       "21   NA             [1.10-1.22]  1.16               NA          NA       \n",
       "22   NA             [1.31-1.43]  1.37               NA          NA       \n",
       "23   NA             [1.10-1.19]  1.14               NA          NA       \n",
       "24   NA             [0.69-1.67]  1.18               NA          NA       \n",
       "25   NA             [1.28, 2.02] 1.65               NA          NA       \n",
       "26   NA             [1.07-1.13]  1.10               NA          NA       \n",
       "27   NA             [1.07-1.14]  1.11               NA          NA       \n",
       "28   NA             [1.06-1.12]  1.09               NA          NA       \n",
       "29   NA             [1.10-1.20]  1.15               NA          NA       \n",
       "30   NA             [1.15-1.38]  1.26               NA          NA       \n",
       "⋮    ⋮              ⋮            ⋮                  ⋮           ⋮        \n",
       "4874 NA             NA              NA              NA          NA       \n",
       "4875 NA             NA              NA              NA          NA       \n",
       "4876 NA             NA              NA              NA          NA       \n",
       "4877 NA             NA              NA              NA          NA       \n",
       "4878 NA             NA              NA              NA          NA       \n",
       "4879 NA             NA              NA              NA          NA       \n",
       "4880 NA             NA              NA              NA          NA       \n",
       "4881 NA             NA              NA              NA          NA       \n",
       "4882 NA             NA              NA              NA          NA       \n",
       "4883 NA             NA              NA              NA          NA       \n",
       "4884 NA             NA              NA              NA          NA       \n",
       "4885 NA             NA              NA              NA          NA       \n",
       "4886 NA             NA              NA              NA          NA       \n",
       "4887 NA             NA              NA              NA          NA       \n",
       "4888 NA             NA              NA              NA          NA       \n",
       "4889 NA             NA              NA              NA          NA       \n",
       "4890 NA             NA              NA              NA          NA       \n",
       "4891 NA             NA              NA              NA          NA       \n",
       "4892 NA             NA              NA              NA          NA       \n",
       "4893 NA             NA              NA              NA          NA       \n",
       "4894 NA             NA              NA              NA          NA       \n",
       "4895 NA             NA           1.342              NA          NA       \n",
       "4896 NA             NA           1.492              NA          NA       \n",
       "4897 NA             NA           0.797              NA          NA       \n",
       "4898 NA             NA           0.787              NA          NA       \n",
       "4899 NA             [0.73-0.86]  0.790              NA          NA       \n",
       "4900 NA             [1.07-1.16]  1.110              NA          NA       \n",
       "4901 NA             [1.15-1.24]  1.190              NA          NA       \n",
       "4902 NA             [0.57-0.77]  0.660              NA          NA       \n",
       "4903 NA             [1.08-1.16]  1.120              NA          NA       \n",
       "     beta_direction beta_description last_mapping_date   last_update_date   \n",
       "1    NA             NA               2024-02-16 18:39:09 2024-02-16 18:39:09\n",
       "2    NA             NA               2024-02-16 18:07:37 2024-02-16 18:07:37\n",
       "3    NA             NA               2024-02-16 18:07:40 2024-02-16 18:07:40\n",
       "4    NA             NA               2024-02-16 18:17:22 2024-02-16 18:17:22\n",
       "5    NA             NA               2024-02-16 18:34:32 2024-02-16 18:34:32\n",
       "6    NA             NA               2024-02-16 18:51:26 2024-02-16 18:51:26\n",
       "7    NA             NA               2024-02-16 19:33:23 2024-02-16 19:33:23\n",
       "8    NA             NA               2024-02-16 19:33:23 2024-02-16 19:33:23\n",
       "9    NA             NA               2024-02-16 18:11:10 2024-02-16 18:11:10\n",
       "10   NA             NA               2024-02-16 18:22:01 2024-02-16 18:22:01\n",
       "11   NA             NA               2024-02-16 18:27:51 2024-02-16 18:27:51\n",
       "12   NA             NA               2024-02-16 18:27:53 2024-02-16 18:27:53\n",
       "13   NA             NA               2024-02-16 18:27:57 2024-02-16 18:27:57\n",
       "14   NA             NA               2024-02-16 18:39:39 2024-02-16 18:39:39\n",
       "15   NA             NA               2024-02-16 18:49:01 2024-02-16 18:49:01\n",
       "16   NA             NA               2024-02-16 18:48:58 2024-02-16 18:48:58\n",
       "17   NA             NA               2024-02-16 18:49:03 2024-02-16 18:49:03\n",
       "18   NA             NA               2024-02-16 18:05:29 2024-02-16 18:05:29\n",
       "19   NA             NA               2024-02-16 18:11:12 2024-02-16 18:11:12\n",
       "20   NA             NA               2024-02-16 18:11:14 2024-02-16 18:11:14\n",
       "21   NA             NA               2024-02-16 18:14:47 2024-02-16 18:14:47\n",
       "22   NA             NA               2024-02-16 18:17:25 2024-02-16 18:17:25\n",
       "23   NA             NA               2024-02-16 18:47:57 2024-02-16 18:47:57\n",
       "24   NA             NA               2024-02-16 18:08:00 2024-02-16 18:08:00\n",
       "25   NA             NA               2024-02-16 18:24:50 2024-02-16 18:24:50\n",
       "26   NA             NA               2024-02-16 18:11:07 2024-02-16 18:11:07\n",
       "27   NA             NA               2024-02-16 18:11:09 2024-02-16 18:11:09\n",
       "28   NA             NA               2024-02-16 18:11:11 2024-02-16 18:11:11\n",
       "29   NA             NA               2024-02-16 18:11:14 2024-02-16 18:11:14\n",
       "30   NA             NA               2024-02-16 18:20:43 2024-02-16 18:20:43\n",
       "⋮    ⋮              ⋮                ⋮                   ⋮                  \n",
       "4874 NA             NA               2024-02-20 13:16:06 2024-02-20 13:16:06\n",
       "4875 NA             NA               2024-02-20 14:18:13 2024-02-20 14:18:13\n",
       "4876 NA             NA               2024-02-20 13:15:43 2024-02-20 13:15:44\n",
       "4877 NA             NA               2024-02-20 14:06:48 2024-02-20 14:06:48\n",
       "4878 NA             NA               2024-02-20 13:15:47 2024-02-20 13:15:47\n",
       "4879 NA             NA               2024-02-20 13:15:49 2024-02-20 13:15:49\n",
       "4880 NA             NA               2024-02-20 14:08:20 2024-02-20 14:08:20\n",
       "4881 NA             NA               2024-02-20 13:15:52 2024-02-20 13:15:52\n",
       "4882 NA             NA               2024-02-20 14:14:38 2024-02-20 14:14:38\n",
       "4883 NA             NA               2024-02-20 13:15:55 2024-02-20 13:15:55\n",
       "4884 NA             NA               2024-02-20 13:15:58 2024-02-20 13:15:58\n",
       "4885 NA             NA               2024-02-20 13:31:04 2024-02-20 13:31:04\n",
       "4886 NA             NA               2024-02-20 13:16:00 2024-02-20 13:16:00\n",
       "4887 NA             NA               2024-02-20 13:23:11 2024-02-20 13:23:11\n",
       "4888 NA             NA               2024-02-20 13:16:02 2024-02-20 13:16:02\n",
       "4889 NA             NA               2024-02-20 13:16:05 2024-02-20 13:16:05\n",
       "4890 NA             NA               2024-02-20 14:21:00 2024-02-20 14:21:00\n",
       "4891 NA             NA               2024-02-20 13:16:08 2024-02-20 13:16:08\n",
       "4892 NA             NA               2024-02-20 13:31:07 2024-02-20 13:31:07\n",
       "4893 NA             NA               2024-02-20 14:49:32 2024-02-20 14:49:32\n",
       "4894 NA             NA               2024-02-20 14:49:34 2024-02-20 14:49:34\n",
       "4895 NA             NA               2024-02-20 13:52:54 2024-02-20 13:52:54\n",
       "4896 NA             NA               2024-02-20 13:53:00 2024-02-20 13:53:00\n",
       "4897 NA             NA               2024-02-20 13:52:10 2024-02-20 13:52:10\n",
       "4898 NA             NA               2024-02-20 13:52:13 2024-02-20 13:52:13\n",
       "4899 NA             NA               2024-02-28 16:32:47 2024-02-28 16:34:57\n",
       "4900 NA             NA               2024-02-28 16:32:48 2024-02-28 16:34:57\n",
       "4901 NA             NA               2024-02-28 16:32:48 2024-02-28 16:34:57\n",
       "4902 NA             NA               2024-02-28 16:32:48 2024-02-28 16:34:57\n",
       "4903 NA             NA               2024-02-28 16:32:49 2024-02-28 16:34:57"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "my_associations2@associations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "b8d5d684-471d-4bf1-aa7f-1fbcfc52eb69",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A tibble: 4911 × 4</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>association_id</th><th scope=col>locus_id</th><th scope=col>haplotype_snp_count</th><th scope=col>description</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>11380</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>12442</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>12443</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>12174</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>12177</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>12444</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>12452</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>12453</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13346</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13286</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13339</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13342</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13345</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13341</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13340</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13287</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13344</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13294</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13347</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13348</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13295</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>13349</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>11785</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>11233</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>11320</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>5873 </td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>5874 </td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>5875 </td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>5876 </td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>11593</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>100372749</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372755</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372761</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372767</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372773</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372779</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372785</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372791</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372797</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372802</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372808</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372814</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372819</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372824</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372830</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372836</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372842</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372848</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372854</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372066</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100372072</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100478397</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100478404</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100478411</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>100478415</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>129662696</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>129662701</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>129662708</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>129662714</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "\t<tr><td>129662719</td><td>1</td><td>NA</td><td>Single variant</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A tibble: 4911 × 4\n",
       "\\begin{tabular}{llll}\n",
       " association\\_id & locus\\_id & haplotype\\_snp\\_count & description\\\\\n",
       " <chr> & <int> & <int> & <chr>\\\\\n",
       "\\hline\n",
       "\t 11380 & 1 & NA & Single variant\\\\\n",
       "\t 12442 & 1 & NA & Single variant\\\\\n",
       "\t 12443 & 1 & NA & Single variant\\\\\n",
       "\t 12174 & 1 & NA & Single variant\\\\\n",
       "\t 12177 & 1 & NA & Single variant\\\\\n",
       "\t 12444 & 1 & NA & Single variant\\\\\n",
       "\t 12452 & 1 & NA & Single variant\\\\\n",
       "\t 12453 & 1 & NA & Single variant\\\\\n",
       "\t 13346 & 1 & NA & Single variant\\\\\n",
       "\t 13286 & 1 & NA & Single variant\\\\\n",
       "\t 13339 & 1 & NA & Single variant\\\\\n",
       "\t 13342 & 1 & NA & Single variant\\\\\n",
       "\t 13345 & 1 & NA & Single variant\\\\\n",
       "\t 13341 & 1 & NA & Single variant\\\\\n",
       "\t 13340 & 1 & NA & Single variant\\\\\n",
       "\t 13287 & 1 & NA & Single variant\\\\\n",
       "\t 13344 & 1 & NA & Single variant\\\\\n",
       "\t 13294 & 1 & NA & Single variant\\\\\n",
       "\t 13347 & 1 & NA & Single variant\\\\\n",
       "\t 13348 & 1 & NA & Single variant\\\\\n",
       "\t 13295 & 1 & NA & Single variant\\\\\n",
       "\t 13349 & 1 & NA & Single variant\\\\\n",
       "\t 11785 & 1 & NA & Single variant\\\\\n",
       "\t 11233 & 1 & NA & Single variant\\\\\n",
       "\t 11320 & 1 & NA & Single variant\\\\\n",
       "\t 5873  & 1 & NA & Single variant\\\\\n",
       "\t 5874  & 1 & NA & Single variant\\\\\n",
       "\t 5875  & 1 & NA & Single variant\\\\\n",
       "\t 5876  & 1 & NA & Single variant\\\\\n",
       "\t 11593 & 1 & NA & Single variant\\\\\n",
       "\t ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\t 100372749 & 1 & NA & Single variant\\\\\n",
       "\t 100372755 & 1 & NA & Single variant\\\\\n",
       "\t 100372761 & 1 & NA & Single variant\\\\\n",
       "\t 100372767 & 1 & NA & Single variant\\\\\n",
       "\t 100372773 & 1 & NA & Single variant\\\\\n",
       "\t 100372779 & 1 & NA & Single variant\\\\\n",
       "\t 100372785 & 1 & NA & Single variant\\\\\n",
       "\t 100372791 & 1 & NA & Single variant\\\\\n",
       "\t 100372797 & 1 & NA & Single variant\\\\\n",
       "\t 100372802 & 1 & NA & Single variant\\\\\n",
       "\t 100372808 & 1 & NA & Single variant\\\\\n",
       "\t 100372814 & 1 & NA & Single variant\\\\\n",
       "\t 100372819 & 1 & NA & Single variant\\\\\n",
       "\t 100372824 & 1 & NA & Single variant\\\\\n",
       "\t 100372830 & 1 & NA & Single variant\\\\\n",
       "\t 100372836 & 1 & NA & Single variant\\\\\n",
       "\t 100372842 & 1 & NA & Single variant\\\\\n",
       "\t 100372848 & 1 & NA & Single variant\\\\\n",
       "\t 100372854 & 1 & NA & Single variant\\\\\n",
       "\t 100372066 & 1 & NA & Single variant\\\\\n",
       "\t 100372072 & 1 & NA & Single variant\\\\\n",
       "\t 100478397 & 1 & NA & Single variant\\\\\n",
       "\t 100478404 & 1 & NA & Single variant\\\\\n",
       "\t 100478411 & 1 & NA & Single variant\\\\\n",
       "\t 100478415 & 1 & NA & Single variant\\\\\n",
       "\t 129662696 & 1 & NA & Single variant\\\\\n",
       "\t 129662701 & 1 & NA & Single variant\\\\\n",
       "\t 129662708 & 1 & NA & Single variant\\\\\n",
       "\t 129662714 & 1 & NA & Single variant\\\\\n",
       "\t 129662719 & 1 & NA & Single variant\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A tibble: 4911 × 4\n",
       "\n",
       "| association_id &lt;chr&gt; | locus_id &lt;int&gt; | haplotype_snp_count &lt;int&gt; | description &lt;chr&gt; |\n",
       "|---|---|---|---|\n",
       "| 11380 | 1 | NA | Single variant |\n",
       "| 12442 | 1 | NA | Single variant |\n",
       "| 12443 | 1 | NA | Single variant |\n",
       "| 12174 | 1 | NA | Single variant |\n",
       "| 12177 | 1 | NA | Single variant |\n",
       "| 12444 | 1 | NA | Single variant |\n",
       "| 12452 | 1 | NA | Single variant |\n",
       "| 12453 | 1 | NA | Single variant |\n",
       "| 13346 | 1 | NA | Single variant |\n",
       "| 13286 | 1 | NA | Single variant |\n",
       "| 13339 | 1 | NA | Single variant |\n",
       "| 13342 | 1 | NA | Single variant |\n",
       "| 13345 | 1 | NA | Single variant |\n",
       "| 13341 | 1 | NA | Single variant |\n",
       "| 13340 | 1 | NA | Single variant |\n",
       "| 13287 | 1 | NA | Single variant |\n",
       "| 13344 | 1 | NA | Single variant |\n",
       "| 13294 | 1 | NA | Single variant |\n",
       "| 13347 | 1 | NA | Single variant |\n",
       "| 13348 | 1 | NA | Single variant |\n",
       "| 13295 | 1 | NA | Single variant |\n",
       "| 13349 | 1 | NA | Single variant |\n",
       "| 11785 | 1 | NA | Single variant |\n",
       "| 11233 | 1 | NA | Single variant |\n",
       "| 11320 | 1 | NA | Single variant |\n",
       "| 5873  | 1 | NA | Single variant |\n",
       "| 5874  | 1 | NA | Single variant |\n",
       "| 5875  | 1 | NA | Single variant |\n",
       "| 5876  | 1 | NA | Single variant |\n",
       "| 11593 | 1 | NA | Single variant |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| 100372749 | 1 | NA | Single variant |\n",
       "| 100372755 | 1 | NA | Single variant |\n",
       "| 100372761 | 1 | NA | Single variant |\n",
       "| 100372767 | 1 | NA | Single variant |\n",
       "| 100372773 | 1 | NA | Single variant |\n",
       "| 100372779 | 1 | NA | Single variant |\n",
       "| 100372785 | 1 | NA | Single variant |\n",
       "| 100372791 | 1 | NA | Single variant |\n",
       "| 100372797 | 1 | NA | Single variant |\n",
       "| 100372802 | 1 | NA | Single variant |\n",
       "| 100372808 | 1 | NA | Single variant |\n",
       "| 100372814 | 1 | NA | Single variant |\n",
       "| 100372819 | 1 | NA | Single variant |\n",
       "| 100372824 | 1 | NA | Single variant |\n",
       "| 100372830 | 1 | NA | Single variant |\n",
       "| 100372836 | 1 | NA | Single variant |\n",
       "| 100372842 | 1 | NA | Single variant |\n",
       "| 100372848 | 1 | NA | Single variant |\n",
       "| 100372854 | 1 | NA | Single variant |\n",
       "| 100372066 | 1 | NA | Single variant |\n",
       "| 100372072 | 1 | NA | Single variant |\n",
       "| 100478397 | 1 | NA | Single variant |\n",
       "| 100478404 | 1 | NA | Single variant |\n",
       "| 100478411 | 1 | NA | Single variant |\n",
       "| 100478415 | 1 | NA | Single variant |\n",
       "| 129662696 | 1 | NA | Single variant |\n",
       "| 129662701 | 1 | NA | Single variant |\n",
       "| 129662708 | 1 | NA | Single variant |\n",
       "| 129662714 | 1 | NA | Single variant |\n",
       "| 129662719 | 1 | NA | Single variant |\n",
       "\n"
      ],
      "text/plain": [
       "     association_id locus_id haplotype_snp_count description   \n",
       "1    11380          1        NA                  Single variant\n",
       "2    12442          1        NA                  Single variant\n",
       "3    12443          1        NA                  Single variant\n",
       "4    12174          1        NA                  Single variant\n",
       "5    12177          1        NA                  Single variant\n",
       "6    12444          1        NA                  Single variant\n",
       "7    12452          1        NA                  Single variant\n",
       "8    12453          1        NA                  Single variant\n",
       "9    13346          1        NA                  Single variant\n",
       "10   13286          1        NA                  Single variant\n",
       "11   13339          1        NA                  Single variant\n",
       "12   13342          1        NA                  Single variant\n",
       "13   13345          1        NA                  Single variant\n",
       "14   13341          1        NA                  Single variant\n",
       "15   13340          1        NA                  Single variant\n",
       "16   13287          1        NA                  Single variant\n",
       "17   13344          1        NA                  Single variant\n",
       "18   13294          1        NA                  Single variant\n",
       "19   13347          1        NA                  Single variant\n",
       "20   13348          1        NA                  Single variant\n",
       "21   13295          1        NA                  Single variant\n",
       "22   13349          1        NA                  Single variant\n",
       "23   11785          1        NA                  Single variant\n",
       "24   11233          1        NA                  Single variant\n",
       "25   11320          1        NA                  Single variant\n",
       "26   5873           1        NA                  Single variant\n",
       "27   5874           1        NA                  Single variant\n",
       "28   5875           1        NA                  Single variant\n",
       "29   5876           1        NA                  Single variant\n",
       "30   11593          1        NA                  Single variant\n",
       "⋮    ⋮              ⋮        ⋮                   ⋮             \n",
       "4882 100372749      1        NA                  Single variant\n",
       "4883 100372755      1        NA                  Single variant\n",
       "4884 100372761      1        NA                  Single variant\n",
       "4885 100372767      1        NA                  Single variant\n",
       "4886 100372773      1        NA                  Single variant\n",
       "4887 100372779      1        NA                  Single variant\n",
       "4888 100372785      1        NA                  Single variant\n",
       "4889 100372791      1        NA                  Single variant\n",
       "4890 100372797      1        NA                  Single variant\n",
       "4891 100372802      1        NA                  Single variant\n",
       "4892 100372808      1        NA                  Single variant\n",
       "4893 100372814      1        NA                  Single variant\n",
       "4894 100372819      1        NA                  Single variant\n",
       "4895 100372824      1        NA                  Single variant\n",
       "4896 100372830      1        NA                  Single variant\n",
       "4897 100372836      1        NA                  Single variant\n",
       "4898 100372842      1        NA                  Single variant\n",
       "4899 100372848      1        NA                  Single variant\n",
       "4900 100372854      1        NA                  Single variant\n",
       "4901 100372066      1        NA                  Single variant\n",
       "4902 100372072      1        NA                  Single variant\n",
       "4903 100478397      1        NA                  Single variant\n",
       "4904 100478404      1        NA                  Single variant\n",
       "4905 100478411      1        NA                  Single variant\n",
       "4906 100478415      1        NA                  Single variant\n",
       "4907 129662696      1        NA                  Single variant\n",
       "4908 129662701      1        NA                  Single variant\n",
       "4909 129662708      1        NA                  Single variant\n",
       "4910 129662714      1        NA                  Single variant\n",
       "4911 129662719      1        NA                  Single variant"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "my_associations2@loci"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "bacfe3c4-e5c6-4dba-b5c0-feb1eb8bf2b2",
   "metadata": {},
   "outputs": [],
   "source": [
    "genes_1 <-my_associations2@ensembl_ids\n",
    "\n",
    "write.table(my_associations2@ensembl_ids,\n",
    "          file = paste0(data_dir, \"GWASVariants_ensembl_ids.csv\"),\n",
    "          row.names = FALSE)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "f564abb8-8424-4978-836b-4103c954bc42",
   "metadata": {},
   "outputs": [],
   "source": [
    "genes <-my_associations2@entrez_ids\n",
    "\n",
    "write.table(my_associations2@ensembl_ids,\n",
    "          file = paste0(data_dir, \"GWASVariants_entrez_ids.csv\"),\n",
    "          row.names = FALSE)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "afc63907-1fff-4a6e-b34a-b7483fb11c5e",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A tibble: 4917 × 3</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>variant_id</th><th scope=col>risk_allele</th><th scope=col>risk_frequency</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>rs7903146 </td><td>A </td><td>NA</td></tr>\n",
       "\t<tr><td>rs5219    </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs10811661</td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs8050136 </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs5219    </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs10946398</td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs7903146 </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs7903146 </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs5219    </td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs9300039 </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs4402960 </td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs8050136 </td><td>A </td><td>NA</td></tr>\n",
       "\t<tr><td>rs1111875 </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs10811661</td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs7754840 </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs7903146 </td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs13266634</td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs8050136 </td><td>A </td><td>NA</td></tr>\n",
       "\t<tr><td>rs4402960 </td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs13266634</td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs10946398</td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs7901695 </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs5215    </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs13266634</td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs7903146 </td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs864745  </td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs12779790</td><td>G </td><td>NA</td></tr>\n",
       "\t<tr><td>rs7961581 </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>rs7578597 </td><td>T </td><td>NA</td></tr>\n",
       "\t<tr><td>rs7020996 </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>rs9905906      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs11650852     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs28476533     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs8080678      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs76734228     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs73379947     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs4799327      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs962876       </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs11877530     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs2967605      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs17657660     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs2061771      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs651862       </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs6036804      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs2024578      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs2834623      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs9789875      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs73219073     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs76866981     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs4661310      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs4920461      </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs35612982     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs61160304     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs10811661     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>rs60808706     </td><td>NA</td><td>NA</td></tr>\n",
       "\t<tr><td>chr6:26276061  </td><td>G </td><td>NA</td></tr>\n",
       "\t<tr><td>chr9:22136441  </td><td>C </td><td>NA</td></tr>\n",
       "\t<tr><td>chr10:113039134</td><td>A </td><td>NA</td></tr>\n",
       "\t<tr><td>chr12:4275678  </td><td>G </td><td>NA</td></tr>\n",
       "\t<tr><td>chr16:53773852 </td><td>G </td><td>NA</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A tibble: 4917 × 3\n",
       "\\begin{tabular}{lll}\n",
       " variant\\_id & risk\\_allele & risk\\_frequency\\\\\n",
       " <chr> & <chr> & <dbl>\\\\\n",
       "\\hline\n",
       "\t rs7903146  & A  & NA\\\\\n",
       "\t rs5219     & NA & NA\\\\\n",
       "\t rs10811661 & NA & NA\\\\\n",
       "\t rs8050136  & NA & NA\\\\\n",
       "\t rs5219     & NA & NA\\\\\n",
       "\t rs10946398 & NA & NA\\\\\n",
       "\t rs7903146  & NA & NA\\\\\n",
       "\t rs7903146  & NA & NA\\\\\n",
       "\t rs5219     & T  & NA\\\\\n",
       "\t rs9300039  & C  & NA\\\\\n",
       "\t rs4402960  & T  & NA\\\\\n",
       "\t rs8050136  & A  & NA\\\\\n",
       "\t rs1111875  & C  & NA\\\\\n",
       "\t rs10811661 & T  & NA\\\\\n",
       "\t rs7754840  & C  & NA\\\\\n",
       "\t rs7903146  & T  & NA\\\\\n",
       "\t rs13266634 & C  & NA\\\\\n",
       "\t rs8050136  & A  & NA\\\\\n",
       "\t rs4402960  & T  & NA\\\\\n",
       "\t rs13266634 & C  & NA\\\\\n",
       "\t rs10946398 & C  & NA\\\\\n",
       "\t rs7901695  & C  & NA\\\\\n",
       "\t rs5215     & C  & NA\\\\\n",
       "\t rs13266634 & C  & NA\\\\\n",
       "\t rs7903146  & T  & NA\\\\\n",
       "\t rs864745   & T  & NA\\\\\n",
       "\t rs12779790 & G  & NA\\\\\n",
       "\t rs7961581  & C  & NA\\\\\n",
       "\t rs7578597  & T  & NA\\\\\n",
       "\t rs7020996  & C  & NA\\\\\n",
       "\t ⋮ & ⋮ & ⋮\\\\\n",
       "\t rs9905906       & NA & NA\\\\\n",
       "\t rs11650852      & NA & NA\\\\\n",
       "\t rs28476533      & NA & NA\\\\\n",
       "\t rs8080678       & NA & NA\\\\\n",
       "\t rs76734228      & NA & NA\\\\\n",
       "\t rs73379947      & NA & NA\\\\\n",
       "\t rs4799327       & NA & NA\\\\\n",
       "\t rs962876        & NA & NA\\\\\n",
       "\t rs11877530      & NA & NA\\\\\n",
       "\t rs2967605       & NA & NA\\\\\n",
       "\t rs17657660      & NA & NA\\\\\n",
       "\t rs2061771       & NA & NA\\\\\n",
       "\t rs651862        & NA & NA\\\\\n",
       "\t rs6036804       & NA & NA\\\\\n",
       "\t rs2024578       & NA & NA\\\\\n",
       "\t rs2834623       & NA & NA\\\\\n",
       "\t rs9789875       & NA & NA\\\\\n",
       "\t rs73219073      & NA & NA\\\\\n",
       "\t rs76866981      & NA & NA\\\\\n",
       "\t rs4661310       & NA & NA\\\\\n",
       "\t rs4920461       & NA & NA\\\\\n",
       "\t rs35612982      & NA & NA\\\\\n",
       "\t rs61160304      & NA & NA\\\\\n",
       "\t rs10811661      & NA & NA\\\\\n",
       "\t rs60808706      & NA & NA\\\\\n",
       "\t chr6:26276061   & G  & NA\\\\\n",
       "\t chr9:22136441   & C  & NA\\\\\n",
       "\t chr10:113039134 & A  & NA\\\\\n",
       "\t chr12:4275678   & G  & NA\\\\\n",
       "\t chr16:53773852  & G  & NA\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A tibble: 4917 × 3\n",
       "\n",
       "| variant_id &lt;chr&gt; | risk_allele &lt;chr&gt; | risk_frequency &lt;dbl&gt; |\n",
       "|---|---|---|\n",
       "| rs7903146  | A  | NA |\n",
       "| rs5219     | NA | NA |\n",
       "| rs10811661 | NA | NA |\n",
       "| rs8050136  | NA | NA |\n",
       "| rs5219     | NA | NA |\n",
       "| rs10946398 | NA | NA |\n",
       "| rs7903146  | NA | NA |\n",
       "| rs7903146  | NA | NA |\n",
       "| rs5219     | T  | NA |\n",
       "| rs9300039  | C  | NA |\n",
       "| rs4402960  | T  | NA |\n",
       "| rs8050136  | A  | NA |\n",
       "| rs1111875  | C  | NA |\n",
       "| rs10811661 | T  | NA |\n",
       "| rs7754840  | C  | NA |\n",
       "| rs7903146  | T  | NA |\n",
       "| rs13266634 | C  | NA |\n",
       "| rs8050136  | A  | NA |\n",
       "| rs4402960  | T  | NA |\n",
       "| rs13266634 | C  | NA |\n",
       "| rs10946398 | C  | NA |\n",
       "| rs7901695  | C  | NA |\n",
       "| rs5215     | C  | NA |\n",
       "| rs13266634 | C  | NA |\n",
       "| rs7903146  | T  | NA |\n",
       "| rs864745   | T  | NA |\n",
       "| rs12779790 | G  | NA |\n",
       "| rs7961581  | C  | NA |\n",
       "| rs7578597  | T  | NA |\n",
       "| rs7020996  | C  | NA |\n",
       "| ⋮ | ⋮ | ⋮ |\n",
       "| rs9905906       | NA | NA |\n",
       "| rs11650852      | NA | NA |\n",
       "| rs28476533      | NA | NA |\n",
       "| rs8080678       | NA | NA |\n",
       "| rs76734228      | NA | NA |\n",
       "| rs73379947      | NA | NA |\n",
       "| rs4799327       | NA | NA |\n",
       "| rs962876        | NA | NA |\n",
       "| rs11877530      | NA | NA |\n",
       "| rs2967605       | NA | NA |\n",
       "| rs17657660      | NA | NA |\n",
       "| rs2061771       | NA | NA |\n",
       "| rs651862        | NA | NA |\n",
       "| rs6036804       | NA | NA |\n",
       "| rs2024578       | NA | NA |\n",
       "| rs2834623       | NA | NA |\n",
       "| rs9789875       | NA | NA |\n",
       "| rs73219073      | NA | NA |\n",
       "| rs76866981      | NA | NA |\n",
       "| rs4661310       | NA | NA |\n",
       "| rs4920461       | NA | NA |\n",
       "| rs35612982      | NA | NA |\n",
       "| rs61160304      | NA | NA |\n",
       "| rs10811661      | NA | NA |\n",
       "| rs60808706      | NA | NA |\n",
       "| chr6:26276061   | G  | NA |\n",
       "| chr9:22136441   | C  | NA |\n",
       "| chr10:113039134 | A  | NA |\n",
       "| chr12:4275678   | G  | NA |\n",
       "| chr16:53773852  | G  | NA |\n",
       "\n"
      ],
      "text/plain": [
       "     variant_id      risk_allele risk_frequency\n",
       "1    rs7903146       A           NA            \n",
       "2    rs5219          NA          NA            \n",
       "3    rs10811661      NA          NA            \n",
       "4    rs8050136       NA          NA            \n",
       "5    rs5219          NA          NA            \n",
       "6    rs10946398      NA          NA            \n",
       "7    rs7903146       NA          NA            \n",
       "8    rs7903146       NA          NA            \n",
       "9    rs5219          T           NA            \n",
       "10   rs9300039       C           NA            \n",
       "11   rs4402960       T           NA            \n",
       "12   rs8050136       A           NA            \n",
       "13   rs1111875       C           NA            \n",
       "14   rs10811661      T           NA            \n",
       "15   rs7754840       C           NA            \n",
       "16   rs7903146       T           NA            \n",
       "17   rs13266634      C           NA            \n",
       "18   rs8050136       A           NA            \n",
       "19   rs4402960       T           NA            \n",
       "20   rs13266634      C           NA            \n",
       "21   rs10946398      C           NA            \n",
       "22   rs7901695       C           NA            \n",
       "23   rs5215          C           NA            \n",
       "24   rs13266634      C           NA            \n",
       "25   rs7903146       T           NA            \n",
       "26   rs864745        T           NA            \n",
       "27   rs12779790      G           NA            \n",
       "28   rs7961581       C           NA            \n",
       "29   rs7578597       T           NA            \n",
       "30   rs7020996       C           NA            \n",
       "⋮    ⋮               ⋮           ⋮             \n",
       "4888 rs9905906       NA          NA            \n",
       "4889 rs11650852      NA          NA            \n",
       "4890 rs28476533      NA          NA            \n",
       "4891 rs8080678       NA          NA            \n",
       "4892 rs76734228      NA          NA            \n",
       "4893 rs73379947      NA          NA            \n",
       "4894 rs4799327       NA          NA            \n",
       "4895 rs962876        NA          NA            \n",
       "4896 rs11877530      NA          NA            \n",
       "4897 rs2967605       NA          NA            \n",
       "4898 rs17657660      NA          NA            \n",
       "4899 rs2061771       NA          NA            \n",
       "4900 rs651862        NA          NA            \n",
       "4901 rs6036804       NA          NA            \n",
       "4902 rs2024578       NA          NA            \n",
       "4903 rs2834623       NA          NA            \n",
       "4904 rs9789875       NA          NA            \n",
       "4905 rs73219073      NA          NA            \n",
       "4906 rs76866981      NA          NA            \n",
       "4907 rs4661310       NA          NA            \n",
       "4908 rs4920461       NA          NA            \n",
       "4909 rs35612982      NA          NA            \n",
       "4910 rs61160304      NA          NA            \n",
       "4911 rs10811661      NA          NA            \n",
       "4912 rs60808706      NA          NA            \n",
       "4913 chr6:26276061   G           NA            \n",
       "4914 chr9:22136441   C           NA            \n",
       "4915 chr10:113039134 A           NA            \n",
       "4916 chr12:4275678   G           NA            \n",
       "4917 chr16:53773852  G           NA            "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "my_associations2@risk_alleles[c('variant_id', 'risk_allele', 'risk_frequency')]\n",
    "\n",
    "write.table(my_associations2@risk_alleles[c('variant_id', 'risk_allele', 'risk_frequency')],\n",
    "          file = paste0(data_dir, \"GWASVariants.csv\"),\n",
    "          row.names = FALSE)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "a2236fde-a984-4bad-a6d2-5904d4b9fab1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<style>\n",
       ".list-inline {list-style: none; margin:0; padding: 0}\n",
       ".list-inline>li {display: inline-block}\n",
       ".list-inline>li:not(:last-child)::after {content: \"\\00b7\"; padding: 0 .5ex}\n",
       "</style>\n",
       "<ol class=list-inline><li>684</li><li>4</li></ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 684\n",
       "\\item 4\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 684\n",
       "2. 4\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1] 684   4"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#genes\n",
    "genes_df <- genes[!is.na(genes$gene_name) & !duplicated(genes$gene_name), ]\n",
    "dim(genes_df)\n",
    "\n",
    "write.table(genes_df,\n",
    "          file = paste0(data_dir, \"GWASVariants_unique_genes_.csv\"),sep=',',\n",
    "          row.names = FALSE)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d635b53d-7315-42da-87e5-9d3d7e2e384d",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "4.3.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
